
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>630A_scaffolds - 16 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="630A_scaffolds - 15 record(s) - 16 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-58a31c57-1520-43b7-adcc-b51faf9d5b62/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.0.1
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="630A_scaffolds.zip">Download all results</a></li>

<li><a href="630A_scaffolds.gbk">Download GenBank summary file</a></li>

<li><a href="bacteria-58a31c57-1520-43b7-adcc-b51faf9d5b62.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton other r1c1">
<a href="#r1c1">1.1</a>
</div>

<div class="regbutton RiPP redox-cofactor r1c2">
<a href="#r1c2">1.2</a>
</div>



<div class="regbutton other arylpolyene r2c1">
<a href="#r2c1">2.1</a>
</div>



<div class="regbutton other r3c1">
<a href="#r3c1">3.1</a>
</div>



<div class="regbutton other NI-siderophore r4c1">
<a href="#r4c1">4.1</a>
</div>



<div class="regbutton NRPS NRPS r12c1">
<a href="#r12c1">12.1</a>
</div>



<div class="regbutton other betalactone r13c1">
<a href="#r13c1">13.1</a>
</div>



<div class="regbutton hybrid r17c1">
<a href="#r17c1">17.1</a>
</div>



<div class="regbutton other r19c1">
<a href="#r19c1">19.1</a>
</div>



<div class="regbutton NRPS NAPAA r25c1">
<a href="#r25c1">25.1</a>
</div>



<div class="regbutton NRPS r27c1">
<a href="#r27c1">27.1</a>
</div>



<div class="regbutton RiPP RiPP-like r29c1">
<a href="#r29c1">29.1</a>
</div>



<div class="regbutton other NAGGN r31c1">
<a href="#r31c1">31.1</a>
</div>



<div class="regbutton NRPS NRPS r36c1">
<a href="#r36c1">36.1</a>
</div>



<div class="regbutton NRPS NRPS r38c1">
<a href="#r38c1">38.1</a>
</div>



<div class="regbutton NRPS NRPS r39c1">
<a href="#r39c1">39.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NODE_1_length_810466_cov_35.218110</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-1" data-filename="NODE_1_length_810466_cov_35.218110_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-1" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r1c1">
<td class="regbutton other r1c1">
<a href="#r1c1">Region&nbsp1.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
</td>
<td class="digits">138,570</td>
<td class="digits table-split-left">159,124</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002076/1" target="_blank">4,5-didehydroacaterin/acaterin</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r1c2">
<td class="regbutton RiPP redox-cofactor r1c2">
<a href="#r1c2">Region&nbsp1.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#redox-cofactor" target="_blank">redox-cofactor</a>
</td>
<td class="digits">725,905</td>
<td class="digits table-split-left">748,076</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001100/1" target="_blank">lankacidin C</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_2_length_455070_cov_34.765996</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-2" data-filename="NODE_2_length_455070_cov_34.765996_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-2" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton other arylpolyene r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#arylpolyene" target="_blank">arylpolyene</a>
</td>
<td class="digits">275,759</td>
<td class="digits table-split-left">319,364</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000837/1" target="_blank">APE Vf</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_3_length_277995_cov_33.414370</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-3" data-filename="NODE_3_length_277995_cov_33.414370_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-3" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r3c1">
<td class="regbutton other r3c1">
<a href="#r3c1">Region&nbsp3.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#other" target="_blank">other</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#acyl_amino_acids" target="_blank">acyl_amino_acids</a>
</td>
<td class="digits">164,892</td>
<td class="digits table-split-left">231,523</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000924/1" target="_blank">pyrrolnitrin</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_4_length_271877_cov_33.374798</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-4" data-filename="NODE_4_length_271877_cov_33.374798_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-4" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r4c1">
<td class="regbutton other NI-siderophore r4c1">
<a href="#r4c1">Region&nbsp4.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ni-siderophore" target="_blank">NI-siderophore</a>
</td>
<td class="digits">56,236</td>
<td class="digits table-split-left">68,158</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_12_length_193785_cov_33.241611</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-12" data-filename="NODE_12_length_193785_cov_33.241611_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-12" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r12c1">
<td class="regbutton NRPS NRPS r12c1">
<a href="#r12c1">Region&nbsp12.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">399</td>
<td class="digits table-split-left">53,382</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000413/1" target="_blank">Pf-5 pyoverdine</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 17%, #ffffff00 17%)">17%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_13_length_174134_cov_32.681214</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-13" data-filename="NODE_13_length_174134_cov_32.681214_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-13" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r13c1">
<td class="regbutton other betalactone r13c1">
<a href="#r13c1">Region&nbsp13.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#betalactone" target="_blank">betalactone</a>
</td>
<td class="digits">73,548</td>
<td class="digits table-split-left">96,798</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001095/1" target="_blank">fengycin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_17_length_150305_cov_32.725291</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-17" data-filename="NODE_17_length_150305_cov_32.725291_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-17" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r17c1">
<td class="regbutton hybrid r17c1">
<a href="#r17c1">Region&nbsp17.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#transat-pks" target="_blank">transAT-PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">108,661</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001112/1" target="_blank">rhizoxin A</a></td>
<td>NRP+Polyketide:Trans-AT type I polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_19_length_113190_cov_33.580932</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-19" data-filename="NODE_19_length_113190_cov_33.580932_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-19" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r19c1">
<td class="regbutton other r19c1">
<a href="#r19c1">Region&nbsp19.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#phosphonate" target="_blank">phosphonate</a>
</td>
<td class="digits">70,219</td>
<td class="digits table-split-left">89,675</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000859/1" target="_blank">ectoine</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_25_length_82593_cov_31.697574</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-25" data-filename="NODE_25_length_82593_cov_31.697574_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-25" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r25c1">
<td class="regbutton NRPS NAPAA r25c1">
<a href="#r25c1">Region&nbsp25.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">21,024</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_27_length_76213_cov_33.724217</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-27" data-filename="NODE_27_length_76213_cov_33.724217_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-27" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r27c1">
<td class="regbutton NRPS r27c1">
<a href="#r27c1">Region&nbsp27.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">69,682</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000413/1" target="_blank">Pf-5 pyoverdine</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 21%, #ffffff00 21%)">21%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_29_length_74090_cov_33.425885</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-29" data-filename="NODE_29_length_74090_cov_33.425885_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-29" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r29c1">
<td class="regbutton RiPP RiPP-like r29c1">
<a href="#r29c1">Region&nbsp29.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
</td>
<td class="digits">16,216</td>
<td class="digits table-split-left">28,423</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_31_length_58202_cov_34.183880</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-31" data-filename="NODE_31_length_58202_cov_34.183880_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-31" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r31c1">
<td class="regbutton other NAGGN r31c1">
<a href="#r31c1">Region&nbsp31.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#naggn" target="_blank">NAGGN</a>
</td>
<td class="digits">20,612</td>
<td class="digits table-split-left">35,337</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_36_length_46894_cov_34.610291</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-36" data-filename="NODE_36_length_46894_cov_34.610291_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-36" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r36c1">
<td class="regbutton NRPS NRPS r36c1">
<a href="#r36c1">Region&nbsp36.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">35,294</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001842/1" target="_blank">gacamide A</a></td>
<td>NRP:Lipopeptide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 85%, #ffffff00 85%)">85%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_38_length_43995_cov_34.355731</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-38" data-filename="NODE_38_length_43995_cov_34.355731_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-38" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r38c1">
<td class="regbutton NRPS NRPS r38c1">
<a href="#r38c1">Region&nbsp38.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">41,734</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001980/1" target="_blank">lokisin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 35%, #ffffff00 35%)">35%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_39_length_43244_cov_33.315727</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-39" data-filename="NODE_39_length_43244_cov_33.315727_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-39" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r39c1">
<td class="regbutton NRPS NRPS r39c1">
<a href="#r39c1">Region&nbsp39.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">43,244</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002503/1" target="_blank">chitinimide B/chitinimide F/chitinimide D/chitinimide A/chitinimide C/chitinimide E/chitinimide G</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 41%, #ffffff00 41%)">41%</td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r1c1">
<td class="regbutton other r1c1">
<a href="#r1c1">Region&nbsp1.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
</td>
<td class="digits">138,570</td>
<td class="digits table-split-left">159,124</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002076/1" target="_blank">4,5-didehydroacaterin/acaterin</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r1c2">
<td class="regbutton RiPP redox-cofactor r1c2">
<a href="#r1c2">Region&nbsp1.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#redox-cofactor" target="_blank">redox-cofactor</a>
</td>
<td class="digits">725,905</td>
<td class="digits table-split-left">748,076</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001100/1" target="_blank">lankacidin C</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton other arylpolyene r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#arylpolyene" target="_blank">arylpolyene</a>
</td>
<td class="digits">275,759</td>
<td class="digits table-split-left">319,364</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000837/1" target="_blank">APE Vf</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>


<tr class="linked-row even" data-anchor="#r3c1">
<td class="regbutton other r3c1">
<a href="#r3c1">Region&nbsp3.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#other" target="_blank">other</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#acyl_amino_acids" target="_blank">acyl_amino_acids</a>
</td>
<td class="digits">164,892</td>
<td class="digits table-split-left">231,523</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000924/1" target="_blank">pyrrolnitrin</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r4c1">
<td class="regbutton other NI-siderophore r4c1">
<a href="#r4c1">Region&nbsp4.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ni-siderophore" target="_blank">NI-siderophore</a>
</td>
<td class="digits">56,236</td>
<td class="digits table-split-left">68,158</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r12c1">
<td class="regbutton NRPS NRPS r12c1">
<a href="#r12c1">Region&nbsp12.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">399</td>
<td class="digits table-split-left">53,382</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000413/1" target="_blank">Pf-5 pyoverdine</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 17%, #ffffff00 17%)">17%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r13c1">
<td class="regbutton other betalactone r13c1">
<a href="#r13c1">Region&nbsp13.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#betalactone" target="_blank">betalactone</a>
</td>
<td class="digits">73,548</td>
<td class="digits table-split-left">96,798</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001095/1" target="_blank">fengycin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>

</tr>


<tr class="linked-row even" data-anchor="#r17c1">
<td class="regbutton hybrid r17c1">
<a href="#r17c1">Region&nbsp17.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#transat-pks" target="_blank">transAT-PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">108,661</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001112/1" target="_blank">rhizoxin A</a></td>
<td>NRP+Polyketide:Trans-AT type I polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r19c1">
<td class="regbutton other r19c1">
<a href="#r19c1">Region&nbsp19.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#phosphonate" target="_blank">phosphonate</a>
</td>
<td class="digits">70,219</td>
<td class="digits table-split-left">89,675</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000859/1" target="_blank">ectoine</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>

</tr>


<tr class="linked-row even" data-anchor="#r25c1">
<td class="regbutton NRPS NAPAA r25c1">
<a href="#r25c1">Region&nbsp25.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">21,024</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r27c1">
<td class="regbutton NRPS r27c1">
<a href="#r27c1">Region&nbsp27.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">69,682</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000413/1" target="_blank">Pf-5 pyoverdine</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 21%, #ffffff00 21%)">21%</td>

</tr>


<tr class="linked-row even" data-anchor="#r29c1">
<td class="regbutton RiPP RiPP-like r29c1">
<a href="#r29c1">Region&nbsp29.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
</td>
<td class="digits">16,216</td>
<td class="digits table-split-left">28,423</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r31c1">
<td class="regbutton other NAGGN r31c1">
<a href="#r31c1">Region&nbsp31.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#naggn" target="_blank">NAGGN</a>
</td>
<td class="digits">20,612</td>
<td class="digits table-split-left">35,337</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r36c1">
<td class="regbutton NRPS NRPS r36c1">
<a href="#r36c1">Region&nbsp36.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">35,294</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001842/1" target="_blank">gacamide A</a></td>
<td>NRP:Lipopeptide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 85%, #ffffff00 85%)">85%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r38c1">
<td class="regbutton NRPS NRPS r38c1">
<a href="#r38c1">Region&nbsp38.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">41,734</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001980/1" target="_blank">lokisin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 35%, #ffffff00 35%)">35%</td>

</tr>


<tr class="linked-row even" data-anchor="#r39c1">
<td class="regbutton NRPS NRPS r39c1">
<a href="#r39c1">Region&nbsp39.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">43,244</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002503/1" target="_blank">chitinimide B/chitinimide F/chitinimide D/chitinimide A/chitinimide C/chitinimide E/chitinimide G</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 41%, #ffffff00 41%)">41%</td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NODE_5_length_267620_cov_34.518758</strong></dd>

<dd><strong>NODE_6_length_251511_cov_34.390731</strong></dd>

<dd><strong>NODE_7_length_246380_cov_34.651563</strong></dd>

<dd><strong>NODE_8_length_224336_cov_33.682448</strong></dd>

<dd><strong>NODE_9_length_222538_cov_32.233529</strong></dd>

<dd><strong>NODE_10_length_209722_cov_32.990341</strong></dd>

<dd><strong>NODE_11_length_209673_cov_33.441411</strong></dd>

<dd><strong>NODE_14_length_168442_cov_33.572185</strong></dd>

<dd><strong>NODE_15_length_166412_cov_35.062837</strong></dd>

<dd><strong>NODE_16_length_151225_cov_32.636820</strong></dd>

<dd><strong>NODE_18_length_130074_cov_32.181827</strong></dd>

<dd><strong>NODE_20_length_106008_cov_33.545147</strong></dd>

<dd><strong>NODE_21_length_103761_cov_33.639588</strong></dd>

<dd><strong>NODE_22_length_103589_cov_32.357485</strong></dd>

<dd><strong>NODE_23_length_91827_cov_34.324425</strong></dd>

<dd><strong>NODE_24_length_84942_cov_33.451270</strong></dd>

<dd><strong>NODE_26_length_82454_cov_33.545383</strong></dd>

<dd><strong>NODE_28_length_74529_cov_34.657793</strong></dd>

<dd><strong>NODE_30_length_64006_cov_33.239031</strong></dd>

<dd><strong>NODE_32_length_54536_cov_35.031749</strong></dd>

<dd><strong>NODE_33_length_53595_cov_30.938432</strong></dd>

<dd><strong>NODE_34_length_50453_cov_34.091432</strong></dd>

<dd><strong>NODE_35_length_47298_cov_35.451663</strong></dd>

<dd><strong>NODE_37_length_45703_cov_32.268422</strong></dd>

<dd><strong>NODE_40_length_41022_cov_33.576798</strong></dd>

<dd><strong>NODE_41_length_40450_cov_34.050033</strong></dd>

<dd><strong>NODE_42_length_38110_cov_34.180265</strong></dd>

<dd><strong>NODE_43_length_37485_cov_33.633635</strong></dd>

<dd><strong>NODE_44_length_36319_cov_39.004773</strong></dd>

<dd><strong>NODE_45_length_31262_cov_33.449158</strong></dd>

<dd><strong>NODE_46_length_28159_cov_31.422940</strong></dd>

<dd><strong>NODE_47_length_24743_cov_32.506568</strong></dd>

<dd><strong>NODE_48_length_24638_cov_32.023493</strong></dd>

<dd><strong>NODE_49_length_24598_cov_34.591697</strong></dd>

<dd><strong>NODE_50_length_22898_cov_32.911485</strong></dd>

<dd><strong>NODE_51_length_22896_cov_32.488628</strong></dd>

<dd><strong>NODE_52_length_19473_cov_32.334760</strong></dd>

<dd><strong>NODE_53_length_18682_cov_32.926364</strong></dd>

<dd><strong>NODE_54_length_17968_cov_35.754737</strong></dd>

<dd><strong>NODE_55_length_13926_cov_33.188895</strong></dd>

<dd><strong>NODE_56_length_10760_cov_35.141252</strong></dd>

<dd><strong>NODE_57_length_7909_cov_32.136491</strong></dd>

<dd><strong>NODE_58_length_5670_cov_29.560880</strong></dd>

<dd><strong>NODE_59_length_5334_cov_261.547461</strong></dd>

<dd><strong>NODE_60_length_2951_cov_28.758177</strong></dd>

<dd><strong>NODE_61_length_2114_cov_102.424153</strong></dd>

<dd><strong>NODE_62_length_1894_cov_56.772152</strong></dd>

<dd><strong>NODE_63_length_1245_cov_147.509418</strong></dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>







<div class="page" id='r1c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_1_length_810466_cov_35.218110 - Region 1 - butyrolactone,hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-166"></div> <span class="help-tooltip" id="region_svg-help-166">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_1_length_810466_cov_35.218110.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c1-svg" data-filename="NODE_1_length_810466_cov_35.218110_r1c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 138,570 - 159,124 nt. (total: 20,555 nt)

<a class="cluster-rules-header" id="r1c1-rules-header" href="#r1c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c1-rules">hserlactone: Autoind_synth<br>butyrolactone: AfsA</div>

<div class="region-svg-container">
<div id='r1c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-167"></div> <span class="help-tooltip" id="focus-panel-help-167">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c1-gene-table-details" data-name="r1c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c1-MIBiG-cluster-compare" data-name="r1c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c1-clusterblast" data-name="r1c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c1-knownclusterblast" data-name="r1c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c1-subclusterblast" data-name="r1c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c1-pfam-details" data-name="r1c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c1-tigrfam-details" data-name="r1c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-5"></div> <span class="help-tooltip" id="gt-help-tooltip-help-5">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-1"></div> <span class="help-tooltip" id="MIBiG-help-1">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c1"></div>
<div id="comparison-MIBiG-r1c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">hserlactone</span></th>

<th><span class="cc-heat-product">butyrolactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002076: 6-3180">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002076/index.html#r1c1" target="_new">BGC0002076</a></td>





<td class="cc-heat-cell" title="0.87 (id:0.67, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.87), rgb(0, 0, 0, 0.87) 87%, white 87%)"></td>






<td class="cc-heat-cell" title="0.87 (id:0.67, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.87), rgb(0, 0, 0, 0.87) 87%, white 87%)"></td>


<td>1.34</td>
<td>Other</td>

<td>4,5-didehydroacaterin, acaterin</td>


<td>Pseudomonas jessenii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000866: 467-5165">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000866/index.html#r1c1" target="_new">BGC0000866</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.47</td>
<td>Other</td>

<td>polyhydroxyalkanoate</td>


<td>Burkholderia sp. DSM 9242</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197: 0-12679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.40</td>
<td>Polyketide</td>

<td>frankiamicin</td>


<td>Frankia sp. EAN1pec</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002069: 0-10624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002069/index.html#r1c1" target="_new">BGC0002069</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.47, order:0.48, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.47, order:0.48, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.39</td>
<td>Other (Aminocoumarin)</td>

<td>styrolide A, styrolide B</td>


<td>Pseudomonas fluorescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888: 0-4342">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.38</td>
<td>Other</td>

<td>bacilysin</td>


<td>Bacillus sp. CS93</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000671: 95-168464">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000671/index.html#r1c1" target="_new">BGC0000671</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.35</td>
<td>Terpene</td>

<td>momilactone</td>


<td>Oryza sativa Japonica Group</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001184: 0-7298">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001184/index.html#r1c1" target="_new">BGC0001184</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>






<td class="cc-heat-cell" title="0.45 (id:0.44, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.33</td>
<td>Other</td>

<td>bacilysin</td>


<td>Bacillus velezensis FZB42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000204: 4242-15189">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000204/index.html#r1c1" target="_new">BGC0000204</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>






<td class="cc-heat-cell" title="0.45 (id:0.43, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.33</td>
<td>Polyketide</td>

<td>benastatin A, benastatin B, BE-43767, TAN1532B</td>


<td>Streptomyces sp. A2991200</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000217: 2-8089">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000217/index.html#r1c1" target="_new">BGC0000217</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.51)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>






<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.51)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.33</td>
<td>Polyketide</td>

<td>daunorubicin</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002510: 0-11781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002510/index.html#r1c1" target="_new">BGC0002510</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>






<td class="cc-heat-cell" title="0.42 (id:0.45, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>youssoufene A1, youssoufene B1, youssoufene B2, youssoufene B3, youssoufene B4</td>


<td>Streptomyces sp.</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002076: 6-3180">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002076/index.html#r1c1" target="_new">BGC0002076</a></td>


<td class="cc-heat-cell" title="0.87 (id:0.67, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.87), rgb(0, 0, 0, 0.87) 87%, white 87%)"></td>
<td>0.87</td>
<td>Other</td>
<td>4,5-didehydroacaterin, acaterin</td>
<td>Pseudomonas jessenii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000866: 467-5165">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000866/index.html#r1c1" target="_new">BGC0000866</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>polyhydroxyalkanoate</td>
<td>Burkholderia sp. DSM 9242</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001586: 0-19073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001586/index.html#r1c1" target="_new">BGC0001586</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Saccharide</td>
<td>EPS273</td>
<td>Pseudomonas stutzeri</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888: 0-4342">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Other</td>
<td>bacilysin</td>
<td>Bacillus sp. CS93</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521: 1747-26349">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>auriculamide</td>
<td>Herpetosiphon aurantiacus DSM 785</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000783: 0-13186">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000783/index.html#r1c1" target="_new">BGC0000783</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Saccharide</td>
<td>O-antigen</td>
<td>Xanthomonas oryzae pv. oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002069: 0-10624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002069/index.html#r1c1" target="_new">BGC0002069</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.47, order:0.48, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other (Aminocoumarin)</td>
<td>styrolide A, styrolide B</td>
<td>Pseudomonas fluorescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001588: 0-24399">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001588/index.html#r1c1" target="_new">BGC0001588</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Saccharide</td>
<td>exopolysaccharides</td>
<td>Burkholderia pseudomallei 1026b</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197: 0-12679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Polyketide</td>
<td>frankiamicin</td>
<td>Frankia sp. EAN1pec</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000807: 0-36089">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000807/index.html#r1c1" target="_new">BGC0000807</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.47, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Saccharide</td>
<td>phosphonoglycans</td>
<td>Stackebrandtia nassauensis DSM 44728</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-2"></div> <span class="help-tooltip" id="cb-general-help-2">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c1_1.svg">
NZ_AZRW02000008 (129359-151259): Pseudomonas sp. QTF5 scaffold8, whole genome... (52% of genes show similarity), TfuA-related
</option>

<option value = "svg/clusterblast_r1c1_2.svg">
NZ_CP038207 (1587406-1607961): Pseudomonas sp. S150 chromosome, complete genome (33% of genes show similarity), butyrolactone,hserlactone
</option>

<option value = "svg/clusterblast_r1c1_3.svg">
NZ_CP038438 (1045190-1065745): Pseudomonas fluorescens strain LBUM677 chromos... (33% of genes show similarity), butyrolactone,hserlactone
</option>

<option value = "svg/clusterblast_r1c1_4.svg">
NZ_OBDS01000017 (22122-42677): Pseudomonas sp. LAMO17WK12:I5, whole genome sh... (33% of genes show similarity), butyrolactone,hserlactone
</option>

<option value = "svg/clusterblast_r1c1_5.svg">
NZ_MOBJ01000021 (22928-43480): Pseudomonas brassicacearum strain 48H11 48H11 ... (27% of genes show similarity), butyrolactone,hserlactone
</option>

<option value = "svg/clusterblast_r1c1_6.svg">
NZ_FYDR01000009 (123076-143631): Pseudomonas sp. Irchel s3a10 isolate Pseudom... (27% of genes show similarity), butyrolactone,hserlactone
</option>

<option value = "svg/clusterblast_r1c1_7.svg">
NZ_VIWL01000017 (257330-277885): Pseudomonas koreensis strain 3286 Ga0365445 ... (27% of genes show similarity), butyrolactone,hserlactone
</option>

<option value = "svg/clusterblast_r1c1_8.svg">
NZ_QTTH01000008 (182913-203468): Pseudomonas sp. 443 Ga0255686 108, whole gen... (27% of genes show similarity), butyrolactone,hserlactone
</option>

<option value = "svg/clusterblast_r1c1_9.svg">
NZ_AZVL01000002 (198580-219135): Pseudomonas sp. URIL14HWK12:I6 H043DRAFT sca... (27% of genes show similarity), butyrolactone,hserlactone
</option>

<option value = "svg/clusterblast_r1c1_10.svg">
NZ_VDDB01000004 (150910-171465): Pseudomonas jessenii strain EC-S101 contig 0... (27% of genes show similarity), butyrolactone,hserlactone
</option>

</select>
<button id= "clusterblast-r1c1-download">Download graphic</button>
</div>
<div id="clusterblast-r1c1-svg"></div>

</div>
</div>

<div class="body-details-section r1c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-3"></div> <span class="help-tooltip" id="cb-known-help-3">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r1c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r1c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r1c1_1.svg">
4,5-didehydroacaterin/acaterin
</option>

</select>
<button id="knownclusterblast-r1c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r1c1-svg"></div>

</div>
</div>

<div class="body-details-section r1c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-4"></div> <span class="help-tooltip" id="cb-sub-help-4">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-6"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-6">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-pfam-details-svg" data-filename="NODE_1_length_810466_cov_35.218110_r1c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-7"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-7">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-tigrfam-details-svg" data-filename="NODE_1_length_810466_cov_35.218110_r1c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r1c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_1_length_810466_cov_35.218110 - Region 2 - redox-cofactor <div class="help-container"> <div class="help-icon" data-id="region_svg-help-168"></div> <span class="help-tooltip" id="region_svg-help-168">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_1_length_810466_cov_35.218110.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c2-svg" data-filename="NODE_1_length_810466_cov_35.218110_r1c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 725,905 - 748,076 nt. (total: 22,172 nt)

<a class="cluster-rules-header" id="r1c2-rules-header" href="#r1c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c2-rules">redox-cofactor: (cds(PF04055 and (TIGR04085 or SPASM)) and (TIGR02111 or TIGR03966) and minimum(1, [Bottromycin_Methyltransferase_RRE, Cyanobactin_D_RRE, Cytolysin_C_RRE, Goadsporin_CD_RRE, Heterocycloanthracin_C_RRE, Lanthipeptide_LanB_RRE, Lasso_Fused_RRE, Mycofactocin_RRE, NHLP_CD_RRE, Other_Sactipeptide_rSAM_RRE, Pantocin_Microcin_RRE, Plantazolicin_C_RRE, PlpY_RRE, PqqD_RRE, Proteusin_Epimerase_RRE, Proteusin_Methyltransferase_RRE, Quinohemoprotein_rSAM_RRE, Ranthipeptide_rSAM_RRE, Stand_Alone_Lasso_RRE, Streptide_RRE, Subtilosin_rSAM_RRE, Thiaglutamate_B_RRE, Thiaglutamate_G_RRE, Thiaglutamate_I_RRE, Thiopeptide_F_RRE, Thuricin_rSAM_RRE, Thurincin_rSAM_RRE, Trifolitoxin_RRE]))</div>

<div class="region-svg-container">
<div id='r1c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-169"></div> <span class="help-tooltip" id="focus-panel-help-169">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c2-gene-table-details" data-name="r1c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c2-MIBiG-cluster-compare" data-name="r1c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c2-clusterblast" data-name="r1c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c2-knownclusterblast" data-name="r1c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c2-subclusterblast" data-name="r1c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c2-tfbs-finder" data-name="r1c2-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r1c2-pfam-details" data-name="r1c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c2-tigrfam-details" data-name="r1c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-15"></div> <span class="help-tooltip" id="gt-help-tooltip-help-15">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-8"></div> <span class="help-tooltip" id="MIBiG-help-8">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c2"></div>
<div id="comparison-MIBiG-r1c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">redox-cofactor</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000919: 746-6062">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000919/index.html#r1c1" target="_new">BGC0000919</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Other</td>

<td>phaseolotoxin</td>


<td>Pseudomonas syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000906: 424-16949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000906/index.html#r1c1" target="_new">BGC0000906</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>heme D1</td>


<td>Heliophilum fasciatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylobacter marinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>ectoine</td>


<td>Sporosarcina pasteurii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000258: 2-24341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000258/index.html#r1c1" target="_new">BGC0000258</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>prodigiosin</td>


<td>Serratia sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000259: 167-24834">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000259/index.html#r1c1" target="_new">BGC0000259</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>prodigiosin</td>


<td>Serratia marcescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000663: 0-13757">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000663/index.html#r1c1" target="_new">BGC0000663</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Terpene</td>

<td>hopene</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002633: 0-11574">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002633/index.html#r1c1" target="_new">BGC0002633</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Other</td>

<td>schizokinen</td>


<td>Leptolyngbya sp. PCC 7376</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000260: 853-35060">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000260/index.html#r1c1" target="_new">BGC0000260</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>prodigiosin</td>


<td>Hahella chejuensis KCTC 2396</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002675: 0-20254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002675/index.html#r1c1" target="_new">BGC0002675</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>di-pyrrolyl-dipyrromethene prodigiosin</td>


<td>Pseudoalteromonas rubra</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000919: 746-6062">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000919/index.html#r1c1" target="_new">BGC0000919</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>phaseolotoxin</td>
<td>Pseudomonas syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000906: 424-16949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000906/index.html#r1c1" target="_new">BGC0000906</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>heme D1</td>
<td>Heliophilum fasciatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylobacter marinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>ectoine</td>
<td>Sporosarcina pasteurii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000258: 2-24341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000258/index.html#r1c1" target="_new">BGC0000258</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>prodigiosin</td>
<td>Serratia sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000259: 167-24834">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000259/index.html#r1c1" target="_new">BGC0000259</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>prodigiosin</td>
<td>Serratia marcescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000663: 0-13757">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000663/index.html#r1c1" target="_new">BGC0000663</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Terpene</td>
<td>hopene</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002633: 0-11574">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002633/index.html#r1c1" target="_new">BGC0002633</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Other</td>
<td>schizokinen</td>
<td>Leptolyngbya sp. PCC 7376</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000260: 853-35060">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000260/index.html#r1c1" target="_new">BGC0000260</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Polyketide</td>
<td>prodigiosin</td>
<td>Hahella chejuensis KCTC 2396</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002675: 0-20254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002675/index.html#r1c1" target="_new">BGC0002675</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Polyketide</td>
<td>di-pyrrolyl-dipyrromethene prodigiosin</td>
<td>Pseudoalteromonas rubra</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-9"></div> <span class="help-tooltip" id="cb-general-help-9">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c2_1.svg">
NZ_LT222319 (521801-564397): Pseudomonas cerasi isolate Sour cherry (Prunus c... (37% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r1c2_2.svg">
NC_007005 (5504337-5546191): Pseudomonas syringae pv. syringae B728a, complet... (30% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r1c2_3.svg">
NZ_AP020337 (5391819-5433703): Pseudomonas sp. KUIN-1 (30% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r1c2_4.svg">
NZ_LT629769 (4147937-4189287): Pseudomonas syringae strain 31R1 chromosome I (36% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r1c2_5.svg">
NZ_KB316543 (28005-72046): Pseudomonas syringae BRIP34881 scaffold13, whole g... (31% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r1c2_6.svg">
NZ_FNJH01000007 (136429-180470): Pseudomonas congelans strain DSM 14939, whol... (38% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r1c2_7.svg">
NZ_CP005970 (3160121-3201723): Pseudomonas syringae UMAF0158 chromosome, comp... (37% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r1c2_8.svg">
NZ_CP028490 (4967599-5011640): Pseudomonas syringae pv. atrofaciens strain LM... (32% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r1c2_9.svg">
NZ_RBOI01000041 (87050-131091): Pseudomonas syringae pv. aptata strain G733 P... (38% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r1c2_10.svg">
NZ_CP006256 (5365585-5405357): Pseudomonas syringae pv. syringae HS191 chromo... (39% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r1c2-download">Download graphic</button>
</div>
<div id="clusterblast-r1c2-svg"></div>

</div>
</div>

<div class="body-details-section r1c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-10"></div> <span class="help-tooltip" id="cb-known-help-10">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r1c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r1c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r1c2_1.svg">
lankacidin C
</option>

</select>
<button id="knownclusterblast-r1c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r1c2-svg"></div>

</div>
</div>

<div class="body-details-section r1c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-11"></div> <span class="help-tooltip" id="cb-sub-help-11">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c2-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-14"></div> <span class="help-tooltip" id="tfbs-body-help-14">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 20.83 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r1c2-725956-ZuR"></div>

<div class="tfbs-hit-detail">
CsoR
<div class="collapser collapser-target-CsoR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Copper-responsive repressor<br>
<span>Score</span>: 25.46 out of 32.38
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r1c2-733197-CsoR"></div>

</div>


</div>
</div>



<div class="body-details-section r1c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-16"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-16">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-pfam-details-svg" data-filename="NODE_1_length_810466_cov_35.218110_r1c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-17"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-17">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-tigrfam-details-svg" data-filename="NODE_1_length_810466_cov_35.218110_r1c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r1c2-RREfinder" data-name="r1c2-RREfinder">
RREFinder
</div>



<div class="sidepanel-details-header r1c2-tfbs-finder" data-name="r1c2-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r1c2-RREfinder">
<div class="more-details">
<div class="heading">
<span>RRE predictions</span>
<div class="help-container"> <div class="help-icon" data-id="rre-side-help-12"></div> <span class="help-tooltip" id="rre-side-help-12">RREfinder results sidepanel.</span></div>
</div>
<dl class = 'prediction-text'>

<span><strong>redox-cofactor protocluster</strong> (725904...748076)</span>
<dt></dt>




<strong>ctg1_677</strong> <div class="collapser collapser-target-ctg1_677 collapser-level-cds"> </div><div class="collapser-content">

<dd>RRE type: PqqD_RRE</dd>
<dd>Location: 1 - 90</dd>
<dd>Score: 115.4</dd>

</div><br>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r1c2-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-13"></div> <span class="help-tooltip" id="tfbs-side-help-13">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CsoR" target="_blank">CsoR</a></td>
<td>Copper-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r2c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_2_length_455070_cov_34.765996 - Region 1 - arylpolyene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-170"></div> <span class="help-tooltip" id="region_svg-help-170">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_2_length_455070_cov_34.765996.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c1-svg" data-filename="NODE_2_length_455070_cov_34.765996_r2c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 275,759 - 319,364 nt. (total: 43,606 nt)

<a class="cluster-rules-header" id="r2c1-rules-header" href="#r2c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c1-rules">arylpolyene: (APE_KS1 or APE_KS2)</div>

<div class="region-svg-container">
<div id='r2c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-171"></div> <span class="help-tooltip" id="focus-panel-help-171">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c1-gene-table-details" data-name="r2c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c1-MIBiG-cluster-compare" data-name="r2c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c1-clusterblast" data-name="r2c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c1-knownclusterblast" data-name="r2c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c1-subclusterblast" data-name="r2c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r2c1-tfbs-finder" data-name="r2c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r2c1-pfam-details" data-name="r2c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c1-tigrfam-details" data-name="r2c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-24"></div> <span class="help-tooltip" id="gt-help-tooltip-help-24">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-18"></div> <span class="help-tooltip" id="MIBiG-help-18">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c1"></div>
<div id="comparison-MIBiG-r2c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">arylpolyene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002341: 602-1868">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002341/index.html#r1c1" target="_new">BGC0002341</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Other</td>

<td>3,5-dibromo-p-anisic acid</td>


<td>Planctomycetales bacterium 10988</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000836: 0-15022">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000836/index.html#r1c1" target="_new">BGC0000836</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.49, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Other</td>

<td>APE Ec</td>


<td>Escherichia coli CFT073</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002008: 0-15251">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002008/index.html#r1c1" target="_new">BGC0002008</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.50, order:0.44, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Other</td>

<td>aryl polyenes</td>


<td>Xenorhabdus doucetiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002493: 1205-6076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002493/index.html#r1c1" target="_new">BGC0002493</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>parabactin</td>


<td>Paracoccus denitrificans PD1222</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723: 128-10733">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>oviedomycin</td>


<td>Streptomyces antibioticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231: 189-5010">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>griseusin A, griseusin B</td>


<td>Streptomyces griseus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002510: 0-11781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002510/index.html#r1c1" target="_new">BGC0002510</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>youssoufene A1, youssoufene B1, youssoufene B2, youssoufene B3, youssoufene B4</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234: 152-6978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>jadomycin</td>


<td>Streptomyces venezuelae ATCC 10712</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000196: 953-9991">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000196/index.html#r1c1" target="_new">BGC0000196</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>AQ-256, AQ-270a, AQ-284a</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001897: 3-16597">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001897/index.html#r1c1" target="_new">BGC0001897</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>cepacin A</td>


<td>Burkholderia ambifaria IOP40-10</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002341: 602-1868">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002341/index.html#r1c1" target="_new">BGC0002341</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>3,5-dibromo-p-anisic acid</td>
<td>Planctomycetales bacterium 10988</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000836: 0-15022">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000836/index.html#r1c1" target="_new">BGC0000836</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.49, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Other</td>
<td>APE Ec</td>
<td>Escherichia coli CFT073</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002008: 0-15251">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002008/index.html#r1c1" target="_new">BGC0002008</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.50, order:0.44, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Other</td>
<td>aryl polyenes</td>
<td>Xenorhabdus doucetiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002493: 1205-6076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002493/index.html#r1c1" target="_new">BGC0002493</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>parabactin</td>
<td>Paracoccus denitrificans PD1222</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723: 128-10733">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>oviedomycin</td>
<td>Streptomyces antibioticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231: 189-5010">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>griseusin A, griseusin B</td>
<td>Streptomyces griseus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002510: 0-11781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002510/index.html#r1c1" target="_new">BGC0002510</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>youssoufene A1, youssoufene B1, youssoufene B2, youssoufene B3, youssoufene B4</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234: 152-6978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>jadomycin</td>
<td>Streptomyces venezuelae ATCC 10712</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000196: 953-9991">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000196/index.html#r1c1" target="_new">BGC0000196</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>AQ-256, AQ-270a, AQ-284a</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001897: 3-16597">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001897/index.html#r1c1" target="_new">BGC0001897</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>cepacin A</td>
<td>Burkholderia ambifaria IOP40-10</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-19"></div> <span class="help-tooltip" id="cb-general-help-19">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c1_1.svg">
NZ_MOBR01000002 (91747-135353): Pseudomonas frederiksbergensis strain 38F7 38... (91% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_2.svg">
NZ_MOBQ01000019 (124529-168135): Pseudomonas frederiksbergensis strain 37A10 ... (86% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_3.svg">
NZ_FYDL01000003 (130604-177955): Pseudomonas sp. Irchel s3h17 isolate Pseudom... (76% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_4.svg">
NZ_CP011020 (1337668-1381280): Pseudomonas chlororaphis strain UFB2 chromosom... (84% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_5.svg">
NZ_JACAPU010000018 (147925-191530): Pseudomonas gingeri strain F1001 run0228 ... (84% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_6.svg">
NZ_CM001558 (5795831-5839442): Pseudomonas fluorescens Q2-87 chromosome, whol... (82% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_7.svg">
NZ_CABVJA010000014 (56807-100413): Pseudomonas fluorescens isolate PS914, who... (85% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_8.svg">
NZ_FYDO01000002 (234373-277978): Pseudomonas sp. Irchel s3f7 isolate Pseudomo... (84% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_9.svg">
NZ_LT629777 (1099006-1142611): Pseudomonas asplenii strain ATCC 23835 chromos... (83% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_10.svg">
NZ_LT629972 (6544206-6587811): Pseudomonas fuscovaginae strain LMG 2158 chrom... (83% of genes show similarity), arylpolyene
</option>

</select>
<button id= "clusterblast-r2c1-download">Download graphic</button>
</div>
<div id="clusterblast-r2c1-svg"></div>

</div>
</div>

<div class="body-details-section r2c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-20"></div> <span class="help-tooltip" id="cb-known-help-20">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c1_1.svg">
APE Vf
</option>

<option value = "svg/knownclusterblast_r2c1_2.svg">
aryl polyenes
</option>

<option value = "svg/knownclusterblast_r2c1_3.svg">
APE Ec
</option>

</select>
<button id="knownclusterblast-r2c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c1-svg"></div>

</div>
</div>

<div class="body-details-section r2c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-21"></div> <span class="help-tooltip" id="cb-sub-help-21">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r2c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-23"></div> <span class="help-tooltip" id="tfbs-body-help-23">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 17.80 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r2c1-299656-AbrC3"></div>

<div class="tfbs-hit-detail">
CatR
<div class="collapser collapser-target-CatR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: H2O2-responsive repressor<br>
<span>Score</span>: 20.67 out of 25.31
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r2c1-305010-CatR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.90 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c1-314255-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r2c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-25"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-25">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-pfam-details-svg" data-filename="NODE_2_length_455070_cov_34.765996_r2c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-26"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-26">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-tigrfam-details-svg" data-filename="NODE_2_length_455070_cov_34.765996_r2c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r2c1-tfbs-finder" data-name="r2c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r2c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-22"></div> <span class="help-tooltip" id="tfbs-side-help-22">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AbrC3" target="_blank">AbrC3</a></td>
<td>Antibiotic production activator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CatR" target="_blank">CatR</a></td>
<td>H2O2-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r3c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_3_length_277995_cov_33.414370 - Region 1 - acyl_amino_acids,other <div class="help-container"> <div class="help-icon" data-id="region_svg-help-172"></div> <span class="help-tooltip" id="region_svg-help-172">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_3_length_277995_cov_33.414370.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c1-svg" data-filename="NODE_3_length_277995_cov_33.414370_r3c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 164,892 - 231,523 nt. (total: 66,632 nt)

<a class="cluster-rules-header" id="r3c1-rules-header" href="#r3c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c1-rules">other: (LmbU or Neocarzinostat or bcpB or frbD or mitE or vlmB or prnB or CaiA or bacilysin or orf2_PTase)<br>acyl_amino_acids: NasY</div>

<div class="region-svg-container">
<div id='r3c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-173"></div> <span class="help-tooltip" id="focus-panel-help-173">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c1-gene-table-details" data-name="r3c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c1-MIBiG-cluster-compare" data-name="r3c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c1-clusterblast" data-name="r3c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c1-knownclusterblast" data-name="r3c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c1-subclusterblast" data-name="r3c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r3c1-tfbs-finder" data-name="r3c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r3c1-pfam-details" data-name="r3c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r3c1-tigrfam-details" data-name="r3c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r3c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-33"></div> <span class="help-tooltip" id="gt-help-tooltip-help-33">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-27"></div> <span class="help-tooltip" id="MIBiG-help-27">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c1"></div>
<div id="comparison-MIBiG-r3c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">other</span></th>

<th><span class="cc-heat-product">acyl_amino_acids</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000924: 136-5698">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000924/index.html#r1c1" target="_new">BGC0000924</a></td>





<td class="cc-heat-cell" title="0.91 (id:0.74, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.91), rgb(0, 0, 0, 0.91) 91%, white 91%)"></td>






<td class="cc-heat-cell" title="0.91 (id:0.74, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.91), rgb(0, 0, 0, 0.91) 91%, white 91%)"></td>


<td>1.48</td>
<td>Other</td>

<td>pyrrolnitrin</td>


<td>Pseudomonas chlororaphis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002482: 14032-28331">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002482/index.html#r1c1" target="_new">BGC0002482</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.56</td>
<td>Other</td>

<td>tjipanazole D, tjipanazole I, tjipanazole L, tjipanazole K, tjipanazole M</td>


<td>Symphyonema bifilamentata 97.28</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001782: 1050-39207">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001782/index.html#r1c1" target="_new">BGC0001782</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.46, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.46, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.39</td>
<td>Other</td>

<td>spiroindimicin A, spiroindimicin B, spiroindimicin C, spiroindimicin D, indimicin A, indimicin B, indimicin C, indimicin D, indimicin E, lynamicin A, lynamicin D, lynamicin F, lynamicin G</td>


<td>Streptomyces sp. SCSIO 03032</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001387: 150-29437">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001387/index.html#r1c1" target="_new">BGC0001387</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.38</td>
<td>Other</td>

<td>nucleocidin</td>


<td>Streptomyces calvus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001512: 0-19326">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001512/index.html#r1c1" target="_new">BGC0001512</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.37</td>
<td>Polyketide</td>

<td>BE-24566B</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000725: 0-16652">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000725/index.html#r1c1" target="_new">BGC0000725</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.37</td>
<td>Saccharide</td>

<td>alginate</td>


<td>Pseudomonas aeruginosa PAO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001501: 0-42974">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001501/index.html#r1c1" target="_new">BGC0001501</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.35</td>
<td>Alkaloid</td>

<td>ambiguine P</td>


<td>Fischerella sp. TAU</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000668: 125-36568">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000668/index.html#r1c1" target="_new">BGC0000668</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.35</td>
<td>Terpene, Alkaloid</td>

<td>12-epi-hapalindole C isonitrile, 12-epi-hapalindole E, 12-epi-fischerindole U isonitrile, fischerindole L, 12-epi-fischerindole I isonitrile, welwitindolinone A isonitrile, welwitindolinone B isothiocyanate, welwitindolinone C isothiocyanate, N-methylwelwitindolinone C isothiocyanate, N-methylwelwitinsolinone C isonitrile, 3-epi-welwitindolinone B isothiocyanate, 3-(Z-2&#39;-isocyanoethenyl)-indole</td>


<td>Hapalosiphon welwitschii UTEX B 1830</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001126: 256-41538">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001126/index.html#r1c1" target="_new">BGC0001126</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.35</td>
<td>Terpene, Alkaloid</td>

<td>12-epi-hapalindole J isonitrile, ambiguine A isonitrile, ambiguine B isonitrile, ambiguine C isonitrile, ambiguine D isonitrile, ambiguine E isonitrile, ambiguine K isonitrile, ambiguine L isonitrile, ambiguine I isonitrile, ambiguine J isonitrile</td>


<td>Fischerella ambigua UTEX 1903</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001612: 0-41276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001612/index.html#r1c1" target="_new">BGC0001612</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.35</td>
<td>Alkaloid</td>

<td>ambiguine H isonitrile</td>


<td>Fischerella ambigua UTEX 1903</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000924: 136-5698">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000924/index.html#r1c1" target="_new">BGC0000924</a></td>


<td class="cc-heat-cell" title="0.91 (id:0.74, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.91), rgb(0, 0, 0, 0.91) 91%, white 91%)"></td>
<td>0.91</td>
<td>Other</td>
<td>pyrrolnitrin</td>
<td>Pseudomonas chlororaphis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000725: 0-16652">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000725/index.html#r1c1" target="_new">BGC0000725</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Saccharide</td>
<td>alginate</td>
<td>Pseudomonas aeruginosa PAO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002482: 14032-28331">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002482/index.html#r1c1" target="_new">BGC0002482</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Other</td>
<td>tjipanazole D, tjipanazole I, tjipanazole L, tjipanazole K, tjipanazole M</td>
<td>Symphyonema bifilamentata 97.28</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000742: 0-2815">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000742/index.html#r1c1" target="_new">BGC0000742</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000743: 0-2815">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000743/index.html#r1c1" target="_new">BGC0000743</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000741: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000741/index.html#r1c1" target="_new">BGC0000741</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000744: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000744/index.html#r1c1" target="_new">BGC0000744</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000745: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000745/index.html#r1c1" target="_new">BGC0000745</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000777: 356-11114">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000777/index.html#r1c1" target="_new">BGC0000777</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.48, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Saccharide</td>
<td>lipopolysaccharide</td>
<td>Escherichia coli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001687: 0-40242">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001687/index.html#r1c1" target="_new">BGC0001687</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>N-tetradecanoyl tyrosine</td>
<td>uncultured bacterium CSL12</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-28"></div> <span class="help-tooltip" id="cb-general-help-28">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c1_1.svg">
NZ_JHYW01000013 (74348-135479): Pseudomonas mosselii DSM 17497 Q380DRAFT scaf... (22% of genes show similarity), acyl_amino_acids
</option>

<option value = "svg/clusterblast_r3c1_2.svg">
NZ_CP052859 (2250430-2291516): Pseudomonas sp. ADAK18 chromosome, complete ge... (32% of genes show similarity), other
</option>

<option value = "svg/clusterblast_r3c1_3.svg">
NZ_CP013184 (4282855-4323941): Pseudomonas protegens strain H78, complete genome (29% of genes show similarity), other
</option>

<option value = "svg/clusterblast_r3c1_4.svg">
NZ_FMWW01000001 (190278-231364): Pseudomonas sp. NFPP12, whole genome shotgun... (29% of genes show similarity), other
</option>

<option value = "svg/clusterblast_r3c1_5.svg">
NZ_MOAT01000017 (1025531-1066617): Pseudomonas protegens strain 1C5 1C5 Joine... (29% of genes show similarity), other
</option>

<option value = "svg/clusterblast_r3c1_6.svg">
NZ_VIWR01000006 (189462-230548): Pseudomonas protegens strain 3295 Ga0365449 ... (29% of genes show similarity), other
</option>

<option value = "svg/clusterblast_r3c1_7.svg">
NZ_RJUX01000001 (3344022-3385108): Pseudomonas protegens strain JUb28 Ga03047... (29% of genes show similarity), other
</option>

<option value = "svg/clusterblast_r3c1_8.svg">
NC_021237 (4066889-4107975): Pseudomonas protegens CHA0, complete genome (28% of genes show similarity), other
</option>

<option value = "svg/clusterblast_r3c1_9.svg">
NC_004129 (4138686-4179772): Pseudomonas protegens Pf-5, complete sequence (29% of genes show similarity), other
</option>

<option value = "svg/clusterblast_r3c1_10.svg">
NZ_FOEY01000001 (1078487-1119573): Pseudomonas sp. NFPP19, whole genome shotg... (29% of genes show similarity), other
</option>

</select>
<button id= "clusterblast-r3c1-download">Download graphic</button>
</div>
<div id="clusterblast-r3c1-svg"></div>

</div>
</div>

<div class="body-details-section r3c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-29"></div> <span class="help-tooltip" id="cb-known-help-29">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r3c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r3c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r3c1_1.svg">
pyrrolnitrin
</option>

</select>
<button id="knownclusterblast-r3c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r3c1-svg"></div>

</div>
</div>

<div class="body-details-section r3c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-30"></div> <span class="help-tooltip" id="cb-sub-help-30">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r3c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-32"></div> <span class="help-tooltip" id="tfbs-body-help-32">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (3):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
DasR
<div class="collapser collapser-target-DasR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: N-acetylglucosamine dependent repressor<br>
<span>Score</span>: 20.87 out of 27.04
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r3c1-166861-DasR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 19.08 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r3c1-194628-ZuR"></div>

<div class="tfbs-hit-detail">
HypR
<div class="collapser collapser-target-HypR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: L-hydroxyproline utilization repressor<br>
<span>Score</span>: 18.94 out of 22.40
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r3c1-231474-HypR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r3c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-34"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-34">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c1-pfam-details-svg" data-filename="NODE_3_length_277995_cov_33.414370_r3c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r3c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-35"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-35">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c1-tigrfam-details-svg" data-filename="NODE_3_length_277995_cov_33.414370_r3c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r3c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r3c1-tfbs-finder" data-name="r3c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r3c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-31"></div> <span class="help-tooltip" id="tfbs-side-help-31">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/DasR" target="_blank">DasR</a></td>
<td>N-acetylglucosamine dependent repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/HypR" target="_blank">HypR</a></td>
<td>L-hydroxyproline utilization repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r4c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_4_length_271877_cov_33.374798 - Region 1 - NI-siderophore <div class="help-container"> <div class="help-icon" data-id="region_svg-help-174"></div> <span class="help-tooltip" id="region_svg-help-174">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_4_length_271877_cov_33.374798.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c1-svg" data-filename="NODE_4_length_271877_cov_33.374798_r4c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 56,236 - 68,158 nt. (total: 11,923 nt)

<a class="cluster-rules-header" id="r4c1-rules-header" href="#r4c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c1-rules">NI-siderophore: IucA_IucC</div>

<div class="region-svg-container">
<div id='r4c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-175"></div> <span class="help-tooltip" id="focus-panel-help-175">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c1-gene-table-details" data-name="r4c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c1-MIBiG-cluster-compare" data-name="r4c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c1-clusterblast" data-name="r4c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c1-knownclusterblast" data-name="r4c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c1-subclusterblast" data-name="r4c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c1-pfam-details" data-name="r4c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r4c1-tigrfam-details" data-name="r4c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r4c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-40"></div> <span class="help-tooltip" id="gt-help-tooltip-help-40">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-36"></div> <span class="help-tooltip" id="MIBiG-help-36">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c1"></div>
<div id="comparison-MIBiG-r4c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NI-siderophore</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002469: 0-6373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002469/index.html#r1c1" target="_new">BGC0002469</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Other</td>

<td>woodybactin A, woodybactin B, woodybactin C, woodybactin D</td>


<td>Shewanella woodyi ATCC 51908</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002681: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002681/index.html#r1c1" target="_new">BGC0002681</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Other</td>

<td>rhizobactin</td>


<td>Francisella tularensis subsp. tularensis SCHU S4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000295: 569-12865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000295/index.html#r1c1" target="_new">BGC0000295</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>acinetoferrin</td>


<td>Acinetobacter haemolyticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001498: 1442-13533">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001498/index.html#r1c1" target="_new">BGC0001498</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Other</td>

<td>aerobactin</td>


<td>Xenorhabdus szentirmaii DSM 16338</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000939: 1834-15938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000939/index.html#r1c1" target="_new">BGC0000939</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Other</td>

<td>aerobactin</td>


<td>Grimontia hollisae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002682: 139-12158">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002682/index.html#r1c1" target="_new">BGC0002682</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Other</td>

<td>aerobactin</td>


<td>Vibrio mimicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713: 164-10986">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Other</td>

<td>ochrobactin</td>


<td>Ochrobactrum anthropi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499: 2267-11578">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Other</td>

<td>aerobactin</td>


<td>Pantoea ananatis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002471: 0-15224">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002471/index.html#r1c1" target="_new">BGC0002471</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.47, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Other</td>

<td>baumannoferrin A, baumannoferrin B</td>


<td>Acinetobacter baumannii AYE</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002683: 0-14011">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002683/index.html#r1c1" target="_new">BGC0002683</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.41, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Other</td>

<td>schizokinen</td>


<td>Nostoc sp. PCC 7120 = FACHB-418</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002469: 0-6373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002469/index.html#r1c1" target="_new">BGC0002469</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Other</td>
<td>woodybactin A, woodybactin B, woodybactin C, woodybactin D</td>
<td>Shewanella woodyi ATCC 51908</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002681: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002681/index.html#r1c1" target="_new">BGC0002681</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Other</td>
<td>rhizobactin</td>
<td>Francisella tularensis subsp. tularensis SCHU S4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000295: 569-12865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000295/index.html#r1c1" target="_new">BGC0000295</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>acinetoferrin</td>
<td>Acinetobacter haemolyticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001498: 1442-13533">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001498/index.html#r1c1" target="_new">BGC0001498</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Other</td>
<td>aerobactin</td>
<td>Xenorhabdus szentirmaii DSM 16338</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000939: 1834-15938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000939/index.html#r1c1" target="_new">BGC0000939</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Other</td>
<td>aerobactin</td>
<td>Grimontia hollisae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002682: 139-12158">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002682/index.html#r1c1" target="_new">BGC0002682</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Other</td>
<td>aerobactin</td>
<td>Vibrio mimicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713: 164-10986">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Other</td>
<td>ochrobactin</td>
<td>Ochrobactrum anthropi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499: 2267-11578">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Other</td>
<td>aerobactin</td>
<td>Pantoea ananatis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002471: 0-15224">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002471/index.html#r1c1" target="_new">BGC0002471</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.47, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Other</td>
<td>baumannoferrin A, baumannoferrin B</td>
<td>Acinetobacter baumannii AYE</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002683: 0-14011">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002683/index.html#r1c1" target="_new">BGC0002683</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.41, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Other</td>
<td>schizokinen</td>
<td>Nostoc sp. PCC 7120 = FACHB-418</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-37"></div> <span class="help-tooltip" id="cb-general-help-37">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c1_1.svg">
NZ_MOBM01000039 (641633-653556): Pseudomonas frederiksbergensis strain 36C6 3... (100% of genes show similarity), siderophore
</option>

<option value = "svg/clusterblast_r4c1_2.svg">
NZ_KB900111 (30983-42906): Pseudomonas sp. 35MFCvi1.1 F571DRAFT scaffold00004... (100% of genes show similarity), siderophore
</option>

<option value = "svg/clusterblast_r4c1_3.svg">
NZ_CP011567 (4924131-4936054): Pseudomonas sp. GR 6-02 chromosome, complete g... (100% of genes show similarity), siderophore
</option>

<option value = "svg/clusterblast_r4c1_4.svg">
NZ_CP023466 (1565305-1577228): Pseudomonas frederiksbergensis strain KNU-15 c... (100% of genes show similarity), siderophore
</option>

<option value = "svg/clusterblast_r4c1_5.svg">
NZ_FYEB01000001 (544169-556092): Pseudomonas sp. Irchel s3b2 isolate Pseudomo... (100% of genes show similarity), siderophore
</option>

<option value = "svg/clusterblast_r4c1_6.svg">
NZ_QKYZ01000015 (106260-118183): Pseudomonas sp. URMO17WK12:I6 F475DRAFT scaf... (100% of genes show similarity), siderophore
</option>

<option value = "svg/clusterblast_r4c1_7.svg">
NZ_KB906328 (72644-84567): Pseudomonas mandelii 36MFCvi1.1 F626DRAFT scaffold... (100% of genes show similarity), siderophore
</option>

<option value = "svg/clusterblast_r4c1_8.svg">
NZ_CP041753 (5121605-5133528): Pseudomonas sp. ATCC 43928 chromosome, complet... (100% of genes show similarity), siderophore
</option>

<option value = "svg/clusterblast_r4c1_9.svg">
NZ_JHVT01000007 (388732-400655): Pseudomonas umsongensis UNC430CL58Col N519DR... (100% of genes show similarity), siderophore
</option>

<option value = "svg/clusterblast_r4c1_10.svg">
NZ_QTUI01000002 (89644-101567): Pseudomonas sp. GV088 Ga0215790 102, whole ge... (100% of genes show similarity), siderophore
</option>

</select>
<button id= "clusterblast-r4c1-download">Download graphic</button>
</div>
<div id="clusterblast-r4c1-svg"></div>

</div>
</div>

<div class="body-details-section r4c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-38"></div> <span class="help-tooltip" id="cb-known-help-38">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r4c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-39"></div> <span class="help-tooltip" id="cb-sub-help-39">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-41"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-41">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-pfam-details-svg" data-filename="NODE_4_length_271877_cov_33.374798_r4c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r4c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-42"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-42">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-tigrfam-details-svg" data-filename="NODE_4_length_271877_cov_33.374798_r4c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r4c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>




<div class="page" id='r12c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_12_length_193785_cov_33.241611 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-176"></div> <span class="help-tooltip" id="region_svg-help-176">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_12_length_193785_cov_33.241611.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r12c1-svg" data-filename="NODE_12_length_193785_cov_33.241611_r12c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 399 - 53,382 nt. (total: 52,984 nt)

<a class="cluster-rules-header" id="r12c1-rules-header" href="#r12c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r12c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r12c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-177"></div> <span class="help-tooltip" id="focus-panel-help-177">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r12c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r12c1-gene-table-details" data-name="r12c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r12c1-nrps_pks" data-name="r12c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r12c1-MIBiG-cluster-compare" data-name="r12c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r12c1-clusterblast" data-name="r12c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r12c1-knownclusterblast" data-name="r12c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r12c1-subclusterblast" data-name="r12c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r12c1-tfbs-finder" data-name="r12c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r12c1-nrps-pks-bubbles" data-name="r12c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r12c1-pfam-details" data-name="r12c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r12c1-tigrfam-details" data-name="r12c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r12c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-53"></div> <span class="help-tooltip" id="gt-help-tooltip-help-53">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r12c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r12c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r12c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r12c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r12c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-43"></div> <span class="help-tooltip" id="nrps-domain-help-43">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r12c1-details-svg" data-filename="NODE_12_length_193785_cov_33.241611_r12c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r12c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r12c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-44"></div> <span class="help-tooltip" id="MIBiG-help-44">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r12c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r12c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r12c1"></div>
<div id="comparison-MIBiG-r12c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000913: 1-11028">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000913/index.html#r1c1" target="_new">BGC0000913</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Other</td>

<td>MK-8</td>


<td>Enterobacter cloacae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>thermoactinoamide A</td>


<td>Thermoactinomyces sp. AS95</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000895: 2-11180">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000895/index.html#r1c1" target="_new">BGC0000895</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Other</td>

<td>citrulline</td>


<td>Lactobacillus plantarum subsp. plantarum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928: 108-8402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Other</td>

<td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>


<td>Streptomyces sp. MA37</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002017: 0-12514">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002017/index.html#r1c1" target="_new">BGC0002017</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>potensibactin</td>


<td>Nocardiopsis potens DSM 45234</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001441: 0-27983">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001441/index.html#r1c1" target="_new">BGC0001441</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.52, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Other</td>

<td>belactosin A, belactosin C</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001671: 5106-76272">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001671/index.html#r1c1" target="_new">BGC0001671</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>monobactam</td>


<td>Agrobacterium tumefaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002494: 273-12179">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002494/index.html#r1c1" target="_new">BGC0002494</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.91)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>vibriobactin</td>


<td>Vibrio cholerae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r12c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000913: 1-11028">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000913/index.html#r1c1" target="_new">BGC0000913</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>MK-8</td>
<td>Enterobacter cloacae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>thermoactinoamide A</td>
<td>Thermoactinomyces sp. AS95</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000895: 2-11180">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000895/index.html#r1c1" target="_new">BGC0000895</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>citrulline</td>
<td>Lactobacillus plantarum subsp. plantarum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928: 108-8402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>
<td>Streptomyces sp. MA37</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002017: 0-12514">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002017/index.html#r1c1" target="_new">BGC0002017</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>potensibactin</td>
<td>Nocardiopsis potens DSM 45234</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001441: 0-27983">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001441/index.html#r1c1" target="_new">BGC0001441</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.52, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Other</td>
<td>belactosin A, belactosin C</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001671: 5106-76272">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001671/index.html#r1c1" target="_new">BGC0001671</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>monobactam</td>
<td>Agrobacterium tumefaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002494: 273-12179">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002494/index.html#r1c1" target="_new">BGC0002494</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.91)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>vibriobactin</td>
<td>Vibrio cholerae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r12c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-45"></div> <span class="help-tooltip" id="cb-general-help-45">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r12c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r12c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r12c1_1.svg">
NZ_RHGD01000001 (5061692-5113264): Pseudomonas mandelii strain B2/38-Q-III Sc... (100% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r12c1_2.svg">
NZ_VIUJ01000010 (12917-65901): Pseudomonas sp. SJZ077 Ga0314243 110, whole ge... (97% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r12c1_3.svg">
NZ_CP005960 (1667929-1720913): Pseudomonas mandelii JR-1 chromosome, complete... (97% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r12c1_4.svg">
NZ_LT629796 (140829-193810): Pseudomonas mandelii strain LMG 21607 chromosome I (97% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r12c1_5.svg">
NZ_JSAL01000024 (14877-67876): Pseudomonas fluorescens strain H16 contig 24, ... (100% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r12c1_6.svg">
NZ_CM001561 (4256442-4309441): Pseudomonas fluorescens R124 chromosome, whole... (100% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r12c1_7.svg">
NZ_FYDZ01000006 (15325-68324): Pseudomonas sp. Irchel s3f10 isolate Pseudomon... (97% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r12c1_8.svg">
NZ_QAYF01000010 (153617-206616): Pseudomonas sp. GV091 Ga0189757 110, whole g... (100% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r12c1_9.svg">
NZ_QJRT01000022 (14962-67961): Pseudomonas jessenii strain LBp-160603 NODE 22... (97% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r12c1_10.svg">
NZ_JABWQN010000005 (269889-322888): Pseudomonas sp. ZA 5.3 contig5, whole gen... (100% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r12c1-download">Download graphic</button>
</div>
<div id="clusterblast-r12c1-svg"></div>

</div>
</div>

<div class="body-details-section r12c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-46"></div> <span class="help-tooltip" id="cb-known-help-46">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r12c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r12c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r12c1_1.svg">
Pf-5 pyoverdine
</option>

<option value = "svg/knownclusterblast_r12c1_2.svg">
azotobactin D
</option>

<option value = "svg/knownclusterblast_r12c1_3.svg">
delftibactin A/delftibactin B
</option>

<option value = "svg/knownclusterblast_r12c1_4.svg">
variobactin A/variobactin B
</option>

<option value = "svg/knownclusterblast_r12c1_5.svg">
crochelin A
</option>

<option value = "svg/knownclusterblast_r12c1_6.svg">
taiwachelin
</option>

<option value = "svg/knownclusterblast_r12c1_7.svg">
serobactin C/serobactin B/serobactin A
</option>

<option value = "svg/knownclusterblast_r12c1_8.svg">
pacifibactin
</option>

<option value = "svg/knownclusterblast_r12c1_9.svg">
amphibactin E/amphibactin H/amphibactin I
</option>

<option value = "svg/knownclusterblast_r12c1_10.svg">
potashchelin A/potashchelin B/potashchelin C/potashchelin D
</option>

</select>
<button id="knownclusterblast-r12c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r12c1-svg"></div>

</div>
</div>

<div class="body-details-section r12c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-47"></div> <span class="help-tooltip" id="cb-sub-help-47">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r12c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-51"></div> <span class="help-tooltip" id="tfbs-body-help-51">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 22.98 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r12c1-33655-ZuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 23.46 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r12c1-41593-ZuR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NrdR
<div class="collapser collapser-target-NrdR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Represses ribonucleotide reductase encoding genes<br>
<span>Score</span>: 21.66 out of 27.18
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r12c1-2376-NrdR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r12c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-52"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-52">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r12c1-domain-bubble-svg" data-filename="NODE_12_length_193785_cov_33.241611_r12c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r12c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (398 - 53382): single NRPS
</option>


</select>

<div class="details-svg" id="r12c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r12c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r12c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-54"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-54">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r12c1-pfam-details-svg" data-filename="NODE_12_length_193785_cov_33.241611_r12c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r12c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r12c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-55"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-55">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r12c1-tigrfam-details-svg" data-filename="NODE_12_length_193785_cov_33.241611_r12c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r12c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r12c1-nrps_pks_products" data-name="r12c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r12c1-nrps_pks_monomers" data-name="r12c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r12c1-tfbs-finder" data-name="r12c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r12c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-48"></div> <span class="help-tooltip" id="nrps-structures-help-48">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 398 - 53382:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r12c1-1" class="smiles-canvas" data-smiles="NC(CCC(=O)O)C(=O)NC(Cc1ccc(O)cc1)C(=O)NC(CCN)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Glu - D-Tyr - Dab)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=glu,d-tyr,dab" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=glu*,d-tyr*,dab*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r12c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CCC(=O)O)C(=O)NC(Cc1ccc(O)cc1)C(=O)NC(CCN)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r12c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-49"></div> <span class="help-tooltip" id="nrps-monomers-help-49">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">ctg12_23</span></strong>: pk - Glu - Tyr - Dab
<div class="collapser collapser-target-ctg12_23 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=glu,tyr,dab" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=glu*,tyr*,dab*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>CAL_domain (41..477)</strong>: pk
<div class="collapser collapser-target-ctg12_23 collapser-level-candidate"> </div><div class="collapser-content">

minowa:  NH2
<div class="collapser collapser-target-ctg12_23-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>NH2: 159.4</dt>

<dd></dd><dt>Acetyl-CoA: 48.3</dt>

<dd></dd><dt>AHBA: 19.9</dt>

<dd></dd><dt>fatty_acid: 12.7</dt>

<dd></dd><dt>shikimic_acid: 2.2</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1131..1528)</strong>: Glu
<div class="collapser collapser-target-ctg12_23 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Glu
<div class="collapser collapser-target-ctg12_23-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>glu,gln (Glu, Gln)</dd>
<dt>Single AA prediction:</dt>
<dd>glu (Glu)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVWHFGRINK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Glu <span class="serif">DVWHFGRINK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (2190..2594)</strong>: Tyr
<div class="collapser collapser-target-ctg12_23 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Tyr
<div class="collapser collapser-target-ctg12_23-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAEGIGAVMK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Tyr <span class="serif">DAEGIGAVMK</span> (97% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (3718..4124)</strong>: Dab
<div class="collapser collapser-target-ctg12_23 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Dab
<div class="collapser collapser-target-ctg12_23-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DIWELTADDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Dab <span class="serif">DIWELTADDK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r12c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-50"></div> <span class="help-tooltip" id="tfbs-side-help-50">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrdR" target="_blank">NrdR</a></td>
<td>Represses ribonucleotide reductase encoding genes</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>




<div class="page" id='r13c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_13_length_174134_cov_32.681214 - Region 1 - betalactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-178"></div> <span class="help-tooltip" id="region_svg-help-178">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_13_length_174134_cov_32.681214.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r13c1-svg" data-filename="NODE_13_length_174134_cov_32.681214_r13c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 73,548 - 96,798 nt. (total: 23,251 nt)

<a class="cluster-rules-header" id="r13c1-rules-header" href="#r13c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r13c1-rules">betalactone: (HMGL-like and AMP-binding)</div>

<div class="region-svg-container">
<div id='r13c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-179"></div> <span class="help-tooltip" id="focus-panel-help-179">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r13c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r13c1-gene-table-details" data-name="r13c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r13c1-MIBiG-cluster-compare" data-name="r13c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r13c1-clusterblast" data-name="r13c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r13c1-knownclusterblast" data-name="r13c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r13c1-subclusterblast" data-name="r13c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r13c1-tfbs-finder" data-name="r13c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r13c1-pfam-details" data-name="r13c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r13c1-tigrfam-details" data-name="r13c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r13c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-62"></div> <span class="help-tooltip" id="gt-help-tooltip-help-62">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r13c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r13c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r13c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r13c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r13c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-56"></div> <span class="help-tooltip" id="MIBiG-help-56">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r13c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r13c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r13c1"></div>
<div id="comparison-MIBiG-r13c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">betalactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001137: 0-18476">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001137/index.html#r1c1" target="_new">BGC0001137</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Alkaloid</td>

<td>marinacarboline A, marinacarboline B, marinacarboline C, marinacarboline D</td>


<td>Marinactinospora thermotolerans</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002405: 17312-42861">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002405/index.html#r1c1" target="_new">BGC0002405</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Saccharide</td>

<td>mid-chain acyl sugars</td>


<td>Solanum lycopersicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000131: 0-6280">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000131/index.html#r1c1" target="_new">BGC0000131</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>pyrrolomycin</td>


<td>Streptomyces sp. UC 11065</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000706: 350-14906">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000706/index.html#r1c1" target="_new">BGC0000706</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Saccharide</td>

<td>kanamycin</td>


<td>Streptomyces kanamyceticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778: 0-16475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>showdomycin</td>


<td>Streptomyces showdoensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002284: 0-17900">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002284/index.html#r1c1" target="_new">BGC0002284</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.50, order:0.36, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Other</td>

<td>corynecin III, corynecin I, corynecin II</td>


<td>Rhodococcus sp. H-CA8f</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>RiPP</td>

<td>SLI-2138</td>


<td>Streptomyces lividans TK24</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>RiPP</td>

<td>SCO-2138</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000913: 1-11028">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000913/index.html#r1c1" target="_new">BGC0000913</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Other</td>

<td>MK-8</td>


<td>Enterobacter cloacae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000258: 2-24341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000258/index.html#r1c1" target="_new">BGC0000258</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>prodigiosin</td>


<td>Serratia sp.</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r13c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001137: 0-18476">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001137/index.html#r1c1" target="_new">BGC0001137</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Alkaloid</td>
<td>marinacarboline A, marinacarboline B, marinacarboline C, marinacarboline D</td>
<td>Marinactinospora thermotolerans</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002405: 17312-42861">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002405/index.html#r1c1" target="_new">BGC0002405</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Saccharide</td>
<td>mid-chain acyl sugars</td>
<td>Solanum lycopersicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000131: 0-6280">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000131/index.html#r1c1" target="_new">BGC0000131</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>pyrrolomycin</td>
<td>Streptomyces sp. UC 11065</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000706: 350-14906">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000706/index.html#r1c1" target="_new">BGC0000706</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Saccharide</td>
<td>kanamycin</td>
<td>Streptomyces kanamyceticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778: 0-16475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>showdomycin</td>
<td>Streptomyces showdoensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002284: 0-17900">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002284/index.html#r1c1" target="_new">BGC0002284</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.50, order:0.36, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Other</td>
<td>corynecin III, corynecin I, corynecin II</td>
<td>Rhodococcus sp. H-CA8f</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>RiPP</td>
<td>SLI-2138</td>
<td>Streptomyces lividans TK24</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>RiPP</td>
<td>SCO-2138</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000913: 1-11028">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000913/index.html#r1c1" target="_new">BGC0000913</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>MK-8</td>
<td>Enterobacter cloacae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000258: 2-24341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000258/index.html#r1c1" target="_new">BGC0000258</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>prodigiosin</td>
<td>Serratia sp.</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r13c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-57"></div> <span class="help-tooltip" id="cb-general-help-57">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r13c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r13c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r13c1_1.svg">
NZ_QAOV01000014 (22285-45574): Pseudomonas sp. GV085 Ga0189755 114, whole gen... (94% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r13c1_2.svg">
NZ_JSAK01000004 (86157-109408): Pseudomonas fluorescens strain C2 contig 4, w... (94% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r13c1_3.svg">
NZ_FYED01000002 (337787-361065): Pseudomonas sp. Irchel 3A7 isolate Pseudomon... (100% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r13c1_4.svg">
NZ_JABFMP010000013 (49687-114322): Pseudomonas sp. C1C7 NODE 13 length 158710... (37% of genes show similarity), NRPS,betalactone
</option>

<option value = "svg/clusterblast_r13c1_5.svg">
NZ_CP052862 (6904429-6927682): Pseudomonas sp. ADAK2 chromosome, complete genome (88% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r13c1_6.svg">
NZ_SOCV01000010 (96716-122838): Pseudomonas mandelii strain YF10-2(1) 51 Ga02... (80% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r13c1_7.svg">
NZ_LMGK01000020 (148440-177077): Pseudomonas sp. Root562 contig 3, whole geno... (94% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r13c1_8.svg">
NZ_CP029693 (1941797-1965079): Pseudomonas putida strain JBC17 chromosome, co... (88% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r13c1_9.svg">
NZ_MOBR01000011 (12186-35442): Pseudomonas frederiksbergensis strain 38F7 38F... (88% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r13c1_10.svg">
NZ_LT629762 (448540-472277): Pseudomonas prosekii strain LMG 26867 chromosome I (88% of genes show similarity), betalactone
</option>

</select>
<button id= "clusterblast-r13c1-download">Download graphic</button>
</div>
<div id="clusterblast-r13c1-svg"></div>

</div>
</div>

<div class="body-details-section r13c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-58"></div> <span class="help-tooltip" id="cb-known-help-58">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r13c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r13c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r13c1_1.svg">
fengycin
</option>

<option value = "svg/knownclusterblast_r13c1_2.svg">
plipastatin
</option>

<option value = "svg/knownclusterblast_r13c1_3.svg">
mycosubtilin
</option>

<option value = "svg/knownclusterblast_r13c1_4.svg">
corynecin III/corynecin I/corynecin II
</option>

</select>
<button id="knownclusterblast-r13c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r13c1-svg"></div>

</div>
</div>

<div class="body-details-section r13c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-59"></div> <span class="help-tooltip" id="cb-sub-help-59">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r13c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-61"></div> <span class="help-tooltip" id="tfbs-body-help-61">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
DasR
<div class="collapser collapser-target-DasR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: N-acetylglucosamine dependent repressor<br>
<span>Score</span>: 22.38 out of 27.04
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r13c1-77809-DasR"></div>

<div class="tfbs-hit-detail">
DasR
<div class="collapser collapser-target-DasR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: N-acetylglucosamine dependent repressor<br>
<span>Score</span>: 21.37 out of 27.04
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r13c1-86929-DasR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 20.81 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r13c1-74016-NrtR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r13c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-63"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-63">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r13c1-pfam-details-svg" data-filename="NODE_13_length_174134_cov_32.681214_r13c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r13c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r13c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-64"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-64">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r13c1-tigrfam-details-svg" data-filename="NODE_13_length_174134_cov_32.681214_r13c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r13c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r13c1-tfbs-finder" data-name="r13c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r13c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-60"></div> <span class="help-tooltip" id="tfbs-side-help-60">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/DasR" target="_blank">DasR</a></td>
<td>N-acetylglucosamine dependent repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r17c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_17_length_150305_cov_32.725291 - Region 1 - NRPS-like,transAT-PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-180"></div> <span class="help-tooltip" id="region_svg-help-180">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_17_length_150305_cov_32.725291.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r17c1-svg" data-filename="NODE_17_length_150305_cov_32.725291_r17c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 108,661 nt. (total: 108,661 nt)

<a class="cluster-rules-header" id="r17c1-rules-header" href="#r17c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r17c1-rules">transAT-PKS: (cds(PKS_AT and not (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS)) and (cds(ATd and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS)) or cds(tra_KS and PP-binding)))<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r17c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-181"></div> <span class="help-tooltip" id="focus-panel-help-181">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r17c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r17c1-gene-table-details" data-name="r17c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r17c1-nrps_pks" data-name="r17c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r17c1-MIBiG-cluster-compare" data-name="r17c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r17c1-clusterblast" data-name="r17c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r17c1-knownclusterblast" data-name="r17c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r17c1-subclusterblast" data-name="r17c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r17c1-tfbs-finder" data-name="r17c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r17c1-nrps-pks-bubbles" data-name="r17c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r17c1-pfam-details" data-name="r17c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r17c1-tigrfam-details" data-name="r17c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r17c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-75"></div> <span class="help-tooltip" id="gt-help-tooltip-help-75">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r17c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r17c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r17c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r17c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r17c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-65"></div> <span class="help-tooltip" id="nrps-domain-help-65">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r17c1-details-svg" data-filename="NODE_17_length_150305_cov_32.725291_r17c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r17c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r17c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-66"></div> <span class="help-tooltip" id="MIBiG-help-66">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r17c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r17c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r17c1"></div>
<div id="comparison-MIBiG-r17c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">transAT-PKS</span></th>

<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001112: 0-81989">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001112/index.html#r1c1" target="_new">BGC0001112</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.59, order:0.63, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.47 (id:0.58, order:0.42, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.50</td>
<td>NRP, Polyketide</td>

<td>rhizoxin A</td>


<td>Paraburkholderia rhizoxinica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000180: 0-26866">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000180/index.html#r1c1" target="_new">BGC0000180</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.42</td>
<td>Polyketide</td>

<td>legioliulin</td>


<td>Legionella parisiensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002056: 102-52034">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002056/index.html#r1c1" target="_new">BGC0002056</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.45, order:0.46, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.44, order:0.51, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.38</td>
<td>NRP, Polyketide</td>

<td>peloruside A</td>


<td>sponge metagenome</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001840: 1369-21011">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001840/index.html#r1c1" target="_new">BGC0001840</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.35</td>
<td>NRP, Polyketide</td>

<td>detoxin S1</td>


<td>Streptomyces sp. NRRL S-325</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001235: 53-50000">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001235/index.html#r1c1" target="_new">BGC0001235</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.45 (id:0.42, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.33</td>
<td>NRP, Polyketide</td>

<td>antalid</td>


<td>Polyangium spumosum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002606: 0-21662">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002606/index.html#r1c1" target="_new">BGC0002606</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.31</td>
<td>NRP, Polyketide</td>

<td>ochratoxin A</td>


<td>Aspergillus niger</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001795: 10-87641">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001795/index.html#r1c1" target="_new">BGC0001795</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.46, order:0.48, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.36 (id:0.45, order:0.39, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.31</td>
<td>Polyketide</td>

<td>swinholide A</td>


<td>Nostoc sp. UHCC 0450</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000393: 0-47918">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000393/index.html#r1c1" target="_new">BGC0000393</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.31</td>
<td>NRP, Polyketide</td>

<td>myxoprincomide-c506</td>


<td>Myxococcus xanthus DK 1622</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000353: 60-28528">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000353/index.html#r1c1" target="_new">BGC0000353</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.31</td>
<td>NRP</td>

<td>tacrolimus</td>


<td>Streptomyces sp. MA6548</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000067: 14949-58348">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000067/index.html#r1c1" target="_new">BGC0000067</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.43, order:0.50, components:0.51)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.43 (id:0.42, order:0.57, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>geldanamycin</td>


<td>Streptomyces hygroscopicus subsp. duamyceticus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r17c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001112: 0-81989">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001112/index.html#r1c1" target="_new">BGC0001112</a></td>


<td class="cc-heat-cell" title="0.74 (id:0.59, order:0.76, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
<td>0.74</td>
<td>NRP, Polyketide</td>
<td>rhizoxin A</td>
<td>Paraburkholderia rhizoxinica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000131: 0-6280">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000131/index.html#r1c1" target="_new">BGC0000131</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>pyrrolomycin</td>
<td>Streptomyces sp. UC 11065</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000260: 853-35060">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000260/index.html#r1c1" target="_new">BGC0000260</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>prodigiosin</td>
<td>Hahella chejuensis KCTC 2396</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000891: 0-11263">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000891/index.html#r1c1" target="_new">BGC0000891</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Other (Aminocoumarin)</td>
<td>pentabromopseudilin</td>
<td>Pseudoalteromonas luteoviolacea 2ta16</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000890: 0-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000890/index.html#r1c1" target="_new">BGC0000890</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Other (Aminocoumarin)</td>
<td>pentabromopseudilin</td>
<td>Pseudoalteromonas phenolica O-BC30</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001465: 0-12220">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001465/index.html#r1c1" target="_new">BGC0001465</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Other</td>
<td>bromophene, bistribromopyrrole, pentabromopseudilin</td>
<td>Marinomonas mediterranea MMB-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002381: 1047-33939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002381/index.html#r1c1" target="_new">BGC0002381</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Alkaloid, Polyketide</td>
<td>tambjamine BE-18591</td>
<td>Streptomyces albus subsp. albus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000706: 350-14906">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000706/index.html#r1c1" target="_new">BGC0000706</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Saccharide</td>
<td>kanamycin</td>
<td>Streptomyces kanamyceticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000220: 512-21855">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000220/index.html#r1c1" target="_new">BGC0000220</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>enterocin</td>
<td>Streptomyces maritimus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001605: 0-9118">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001605/index.html#r1c1" target="_new">BGC0001605</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Terpene</td>
<td>gibberellin</td>
<td>Sinorhizobium fredii NGR234</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r17c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-67"></div> <span class="help-tooltip" id="cb-general-help-67">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r17c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r17c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r17c1_1.svg">
NZ_QJRM01000005 (333948-452859): Pseudomonas protegens strain MB-090714 NODE ... (29% of genes show similarity), NRPS-like,transAT-PKS,transAT-PKS-like
</option>

<option value = "svg/clusterblast_r17c1_2.svg">
NZ_QJRN01000003 (301938-420792): Pseudomonas protegens strain MB-090624 NODE ... (29% of genes show similarity), NRPS-like,transAT-PKS,transAT-PKS-like
</option>

<option value = "svg/clusterblast_r17c1_3.svg">
NC_004129 (3347744-3466615): Pseudomonas protegens Pf-5, complete sequence (29% of genes show similarity), NRPS-like,transAT-PKS,transAT-PKS-like
</option>

<option value = "svg/clusterblast_r17c1_4.svg">
NZ_AP014627 (3266542-3385308): Pseudomonas sp. Os17 (27% of genes show similarity), NRPS-like,transAT-PKS,transAT-PKS-like
</option>

<option value = "svg/clusterblast_r17c1_5.svg">
NC_014722 (1556868-1677792): Paraburkholderia rhizoxinica HKI 454, complete g... (25% of genes show similarity), NRPS,transAT-PKS,transAT-PKS-like
</option>

<option value = "svg/clusterblast_r17c1_6.svg">
NZ_CP022775 (454829-575667): Ralstonia solanacearum strain T12 plasmid unname... (26% of genes show similarity), NRPS-like,siderophore,transAT-PKS,transAT-PKS-like
</option>

<option value = "svg/clusterblast_r17c1_7.svg">
NZ_CDMA01000001 (&lt;42760-113990): Ralstonia solanacearum strain B50, whole gen... (11% of genes show similarity), transAT-PKS-like
</option>

<option value = "svg/clusterblast_r17c1_8.svg">
NZ_FMWJ01000001 (270292-328689): Photorhabdus luminescens strain ATCC 29999, ... (18% of genes show similarity), transAT-PKS
</option>

<option value = "svg/clusterblast_r17c1_9.svg">
NZ_CP026091 (1350398-1427095): Ralstonia solanacearum strain IBSBF 2570 plasm... (20% of genes show similarity), transAT-PKS
</option>

<option value = "svg/clusterblast_r17c1_10.svg">
NC_016111 (3784856-3887790): Streptomyces cattleya NRRL 8057 = DSM 46488, com... (14% of genes show similarity), NRPS,T1PKS,transAT-PKS
</option>

</select>
<button id= "clusterblast-r17c1-download">Download graphic</button>
</div>
<div id="clusterblast-r17c1-svg"></div>

</div>
</div>

<div class="body-details-section r17c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-68"></div> <span class="help-tooltip" id="cb-known-help-68">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r17c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r17c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r17c1_1.svg">
rhizoxin A
</option>

<option value = "svg/knownclusterblast_r17c1_2.svg">
aurantinin B/aurantinin C/aurantinin D
</option>

</select>
<button id="knownclusterblast-r17c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r17c1-svg"></div>

</div>
</div>

<div class="body-details-section r17c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-69"></div> <span class="help-tooltip" id="cb-sub-help-69">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r17c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-73"></div> <span class="help-tooltip" id="tfbs-body-help-73">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 24.27 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r17c1-4375-ZuR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 21.23 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r17c1-89431-LexA"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r17c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-74"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-74">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r17c1-domain-bubble-svg" data-filename="NODE_17_length_150305_cov_32.725291_r17c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r17c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (0 - 108661): neighbouring transAT-PKS-NRPS-like
</option>



<option value="CC2">
Candidate 2 (0 - 88632): single transAT-PKS
</option>



<option value="CC3">
Candidate 3 (48690 - 108661): single NRPS-like
</option>


</select>

<div class="details-svg" id="r17c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r17c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r17c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-76"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-76">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r17c1-pfam-details-svg" data-filename="NODE_17_length_150305_cov_32.725291_r17c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r17c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r17c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-77"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-77">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r17c1-tigrfam-details-svg" data-filename="NODE_17_length_150305_cov_32.725291_r17c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r17c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r17c1-nrps_pks_products" data-name="r17c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r17c1-nrps_pks_monomers" data-name="r17c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r17c1-tfbs-finder" data-name="r17c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r17c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-70"></div> <span class="help-tooltip" id="nrps-structures-help-70">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 0 - 108661:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r17c1-1" class="smiles-canvas" data-smiles="NC(CO)C(=O)CC(=O)C=CC=CC=CCC(O)C=CCC(O)C=CC=CCC(O)C=CCC(=O)C=CCC(=O)CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Ser - mal - ccmal - ccmal - ccmal) + (ohmal - ccmal - ohmal) + (ccmal - ccmal) + (ohmal - ccmal - mal) + (ccmal - mal) + (mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,ccmal,ccmal,ccmal&amp;nrps2=ohmal,ccmal,ohmal&amp;nrps3=ccmal,ccmal&amp;nrps4=ohmal,ccmal&amp;nrps5=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,ccmal*,ccmal*,ccmal*&amp;nrps2=ohmal*,ccmal*,ohmal*&amp;nrps3=ccmal*,ccmal*&amp;nrps4=ohmal*,ccmal*&amp;nrps5=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r17c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CO)C(=O)CC(=O)C=CC=CC=CCC(O)C=CCC(O)C=CC=CCC(O)C=CCC(=O)C=CCC(=O)CC(=O)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 2, location 0 - 88632:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r17c1-2" class="smiles-canvas" data-smiles="NC(CO)C(=O)CC(=O)C=CC=CC=CCC(O)C=CCC(O)C=CC=CCC(O)C=CCC(=O)C=CCC(=O)CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Ser - mal - ccmal - ccmal - ccmal) + (ohmal - ccmal - ohmal) + (ccmal - ccmal) + (ohmal - ccmal - mal) + (ccmal - mal) + (mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,ccmal,ccmal,ccmal&amp;nrps2=ohmal,ccmal,ohmal&amp;nrps3=ccmal,ccmal&amp;nrps4=ohmal,ccmal&amp;nrps5=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,ccmal*,ccmal*,ccmal*&amp;nrps2=ohmal*,ccmal*,ohmal*&amp;nrps3=ccmal*,ccmal*&amp;nrps4=ohmal*,ccmal*&amp;nrps5=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r17c1-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CO)C(=O)CC(=O)C=CC=CC=CCC(O)C=CCC(O)C=CC=CCC(O)C=CCC(=O)C=CCC(=O)CC(=O)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 48690 - 108661:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r17c1-3" class="smiles-canvas" data-smiles="NC(CO)C(=O)CC(=O)C=CC=CC=CCC(O)C=CCC(O)C=CC=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Ser - mal - ccmal - ccmal - ccmal) + (ohmal - ccmal - ohmal) + (ccmal - ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,ccmal,ccmal,ccmal&amp;nrps2=ohmal,ccmal,ohmal&amp;nrps3=ccmal,ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,ccmal*,ccmal*,ccmal*&amp;nrps2=ohmal*,ccmal*,ohmal*&amp;nrps3=ccmal*,ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r17c1-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CO)C(=O)CC(=O)C=CC=CC=CCC(O)C=CCC(O)C=CC=CC(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r17c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-71"></div> <span class="help-tooltip" id="nrps-monomers-help-71">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">ctg17_5</span></strong>: X
<div class="collapser collapser-target-ctg17_5 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (14..425)</strong>: X
<div class="collapser collapser-target-ctg17_5 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly, Ala, Val, Leu, Ile, Abu, Ival
<div class="collapser collapser-target-ctg17_5-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">GTLFIGGMVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>clAnt <span class="serif">GAFFIGAGVK</span> (44% 8 match)</dd>
<dd>Leu or Ile <span class="serif">DALFIGGTFK</span> (32% 8 match)</dd>
<dd>Ile <span class="serif">DGLFIGIPVK</span> (26% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">ctg17_10</span></strong>: mal - mal
<div class="collapser collapser-target-ctg17_10 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>PKS_AT (49..348)</strong>: mal
<div class="collapser collapser-target-ctg17_10 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  (unknown)
<div class="collapser collapser-target-ctg17_10-ATSignature collapser-level-none"> </div><div class="collapser-content">
No matches above 50%
</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-ctg17_10-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 52.4</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 25.6</dt>

<dd></dd><dt>inactive: 23.3</dt>

<dd></dd><dt>Isobutyryl-CoA: 23.3</dt>

<dd></dd><dt>Methylmalonyl-CoA: 22.5</dt>

<dd></dd><dt>Benzoyl-CoA: 15.1</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>fatty_acid: 0.0</dt>

<dd></dd><dt>Propionyl-CoA: 0.0</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 0.0</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>



<dd><strong>PKS_AT (416..693)</strong>: mal
<div class="collapser collapser-target-ctg17_10 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-ctg17_10-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>inactive: 62.5%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-ctg17_10-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 107.3</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 45.0</dt>

<dd></dd><dt>inactive: 35.3</dt>

<dd></dd><dt>Methylmalonyl-CoA: 28.7</dt>

<dd></dd><dt>Propionyl-CoA: 23.4</dt>

<dd></dd><dt>fatty_acid: 14.2</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 13.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 11.4</dt>

<dd></dd><dt>CHC-CoA: 10.1</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

<dd></dd><dt>Benzoyl-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">ctg17_17</span></strong>: Ser
<div class="collapser collapser-target-ctg17_17 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (50..448)</strong>: Ser
<div class="collapser collapser-target-ctg17_17 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ser
<div class="collapser collapser-target-ctg17_17-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>ser (Ser)</dd>
<dt>Single AA prediction:</dt>
<dd>ser (Ser)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVWHFSLVDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ser <span class="serif">DVWHFSLVDK</span> (94% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










































</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r17c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-72"></div> <span class="help-tooltip" id="tfbs-side-help-72">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>




<div class="page" id='r19c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_19_length_113190_cov_33.580932 - Region 1 - ectoine,phosphonate <div class="help-container"> <div class="help-icon" data-id="region_svg-help-182"></div> <span class="help-tooltip" id="region_svg-help-182">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_19_length_113190_cov_33.580932.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r19c1-svg" data-filename="NODE_19_length_113190_cov_33.580932_r19c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 70,219 - 89,675 nt. (total: 19,457 nt)

<a class="cluster-rules-header" id="r19c1-rules-header" href="#r19c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r19c1-rules">ectoine: ectoine_synt<br>phosphonate: (phosphonates and minimum(1, [2-Hacid_dh_C, Aldedh, Aminotran_1_2, Aminotran_5, AurF, CTP_transf_like, DUF4992, Fe-ADH, HMGL-like, Metallophos_2, NTP_transf_3, NTP_transf_5, PALP, PEP_mutase, TPP_enzyme_C, TPP_enzyme_N, phosphonates]) and not (phosphonatesHydrolase or phosphonatesMeIsocitrateLyase))</div>

<div class="region-svg-container">
<div id='r19c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-183"></div> <span class="help-tooltip" id="focus-panel-help-183">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r19c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r19c1-gene-table-details" data-name="r19c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r19c1-MIBiG-cluster-compare" data-name="r19c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r19c1-clusterblast" data-name="r19c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r19c1-knownclusterblast" data-name="r19c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r19c1-subclusterblast" data-name="r19c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r19c1-tfbs-finder" data-name="r19c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r19c1-pfam-details" data-name="r19c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r19c1-tigrfam-details" data-name="r19c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r19c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-84"></div> <span class="help-tooltip" id="gt-help-tooltip-help-84">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r19c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r19c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r19c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r19c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r19c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-78"></div> <span class="help-tooltip" id="MIBiG-help-78">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r19c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r19c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r19c1"></div>
<div id="comparison-MIBiG-r19c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">ectoine</span></th>

<th><span class="cc-heat-product">phosphonate</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.50, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.60</td>
<td>Other</td>

<td>ectoine</td>


<td>Sporosarcina pasteurii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853: 8-3345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.50, order:0.50, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.51, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.57</td>
<td>Other</td>

<td>ectoine</td>


<td>Streptomyces anulatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052: 188-3620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.50, order:0.50, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.50, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.57</td>
<td>Other (Ectoine)</td>

<td>ectoine</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.50, order:0.50, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.57</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium alcaliphilum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.49, order:0.50, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.56</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium kenyense</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.49, order:0.50, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.56</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylophaga thalassica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000904: 116-14341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000904/index.html#r1c1" target="_new">BGC0000904</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.48, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.48, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.47</td>
<td>Other</td>

<td>FR-900098</td>


<td>Streptomyces rubellomurinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001859: 6609-19144">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001859/index.html#r1c1" target="_new">BGC0001859</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.51, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.51, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.46</td>
<td>Other (Phosphonate)</td>

<td>fosfomycin</td>


<td>Pseudomonas syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854: 838-6879">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.49, order:0.43, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.49, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.46</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium alcaliphilum 20Z</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860: 710-5134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.50, order:0.46, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.50, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.45</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylarcula marina</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r19c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylobacter marinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>ectoine</td>
<td>Sporosarcina pasteurii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853: 8-3345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.50, order:0.50, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Other</td>
<td>ectoine</td>
<td>Streptomyces anulatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.50, order:0.50, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium alcaliphilum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052: 188-3620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.50, order:0.50, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Other (Ectoine)</td>
<td>ectoine</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.49, order:0.50, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium kenyense</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.49, order:0.50, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga thalassica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860: 710-5134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.50, order:0.46, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylarcula marina</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854: 838-6879">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.49, order:0.44, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium alcaliphilum 20Z</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857: 300-4871">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.49, order:0.46, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga alcalica</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r19c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-79"></div> <span class="help-tooltip" id="cb-general-help-79">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r19c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r19c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r19c1_1.svg">
NZ_CP031560 (1532770-1573664): Dickeya dianthicola strain ME23 chromosome, co... (18% of genes show similarity), phosphonate
</option>

<option value = "svg/clusterblast_r19c1_2.svg">
NZ_CP053581 (3657705-3735679): Salmonella enterica strain 94-0093 chromosome,... (9% of genes show similarity), NRPS,phosphonate
</option>

<option value = "svg/clusterblast_r19c1_3.svg">
NZ_CP054715 (1979426-2070037): Salmonella enterica strain 85-0120 chromosome,... (9% of genes show similarity), NRPS,T1PKS,phosphonate
</option>

<option value = "svg/clusterblast_r19c1_4.svg">
NC_020209 (407330-448200): Pseudomonas poae RE*1-1-14, complete sequence (20% of genes show similarity), phosphonate
</option>

<option value = "svg/clusterblast_r19c1_5.svg">
NZ_JAAQRN010000003 (442269-483139): Serratia plymuthica strain IV-11-34 scaff... (22% of genes show similarity), phosphonate
</option>

<option value = "svg/clusterblast_r19c1_6.svg">
NZ_ARWD01000001 (3430377-3471247): Serratia plymuthica RVH1 C786 chromosome1.... (22% of genes show similarity), phosphonate
</option>

<option value = "svg/clusterblast_r19c1_7.svg">
NZ_CP012096 (4272979-4313849): Serratia plymuthica strain 3Rp8 chromosome, co... (24% of genes show similarity), phosphonate
</option>

<option value = "svg/clusterblast_r19c1_8.svg">
NC_021591 (3347014-3387884): Serratia plymuthica 4Rx13, complete genome (25% of genes show similarity), phosphonate
</option>

<option value = "svg/clusterblast_r19c1_9.svg">
NC_021659 (3416146-3457016): Serratia plymuthica S13, complete sequence (22% of genes show similarity), phosphonate
</option>

<option value = "svg/clusterblast_r19c1_10.svg">
NZ_CP012097 (3466086-3506956): Serratia plymuthica strain 3Re4-18 chromosome,... (24% of genes show similarity), phosphonate
</option>

</select>
<button id= "clusterblast-r19c1-download">Download graphic</button>
</div>
<div id="clusterblast-r19c1-svg"></div>

</div>
</div>

<div class="body-details-section r19c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-80"></div> <span class="help-tooltip" id="cb-known-help-80">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r19c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r19c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r19c1_1.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r19c1_2.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r19c1_3.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r19c1_4.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r19c1_5.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r19c1_6.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r19c1_7.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r19c1_8.svg">
ectoine
</option>

</select>
<button id="knownclusterblast-r19c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r19c1-svg"></div>

</div>
</div>

<div class="body-details-section r19c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-81"></div> <span class="help-tooltip" id="cb-sub-help-81">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r19c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-83"></div> <span class="help-tooltip" id="tfbs-body-help-83">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 21.70 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r19c1-75192-ZuR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
OsdR
<div class="collapser collapser-target-OsdR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Development and stress management regulator<br>
<span>Score</span>: 22.61 out of 30.12
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r19c1-81634-OsdR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r19c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-85"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-85">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r19c1-pfam-details-svg" data-filename="NODE_19_length_113190_cov_33.580932_r19c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r19c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r19c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-86"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-86">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r19c1-tigrfam-details-svg" data-filename="NODE_19_length_113190_cov_33.580932_r19c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r19c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r19c1-tfbs-finder" data-name="r19c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r19c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-82"></div> <span class="help-tooltip" id="tfbs-side-help-82">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/OsdR" target="_blank">OsdR</a></td>
<td>Development and stress management regulator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r25c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_25_length_82593_cov_31.697574 - Region 1 - NAPAA <div class="help-container"> <div class="help-icon" data-id="region_svg-help-184"></div> <span class="help-tooltip" id="region_svg-help-184">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_25_length_82593_cov_31.697574.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r25c1-svg" data-filename="NODE_25_length_82593_cov_31.697574_r25c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 21,024 nt. (total: 21,024 nt)

<a class="cluster-rules-header" id="r25c1-rules-header" href="#r25c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r25c1-rules">NAPAA: (AMP-binding and TIGR02353)</div>

<div class="region-svg-container">
<div id='r25c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-185"></div> <span class="help-tooltip" id="focus-panel-help-185">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r25c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r25c1-gene-table-details" data-name="r25c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r25c1-nrps_pks" data-name="r25c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r25c1-MIBiG-cluster-compare" data-name="r25c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r25c1-clusterblast" data-name="r25c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r25c1-knownclusterblast" data-name="r25c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r25c1-subclusterblast" data-name="r25c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r25c1-tfbs-finder" data-name="r25c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r25c1-nrps-pks-bubbles" data-name="r25c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r25c1-pfam-details" data-name="r25c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r25c1-tigrfam-details" data-name="r25c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r25c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-97"></div> <span class="help-tooltip" id="gt-help-tooltip-help-97">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r25c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r25c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r25c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r25c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r25c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-87"></div> <span class="help-tooltip" id="nrps-domain-help-87">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r25c1-details-svg" data-filename="NODE_25_length_82593_cov_31.697574_r25c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r25c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r25c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-88"></div> <span class="help-tooltip" id="MIBiG-help-88">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r25c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r25c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r25c1"></div>
<div id="comparison-MIBiG-r25c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NAPAA</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002174: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002174/index.html#r1c1" target="_new">BGC0002174</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>-Poly-L-lysine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002535: 1708-15164">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002535/index.html#r1c1" target="_new">BGC0002535</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>-poly-L-2,4-diaminobutyric acid</td>


<td>Streptomyces celluloflavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>RiPP</td>

<td>SCO-2138</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>RiPP</td>

<td>SLI-2138</td>


<td>Streptomyces lividans TK24</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000131: 0-6280">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000131/index.html#r1c1" target="_new">BGC0000131</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>pyrrolomycin</td>


<td>Streptomyces sp. UC 11065</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002217: 0-33547">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002217/index.html#r1c1" target="_new">BGC0002217</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>HEx-tc3 terpene</td>


<td>Botrytis cinerea T4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778: 0-16475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Other</td>

<td>showdomycin</td>


<td>Streptomyces showdoensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>SWA-2138</td>


<td>Streptomyces sp. e14</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r25c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002174: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002174/index.html#r1c1" target="_new">BGC0002174</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>-Poly-L-lysine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002535: 1708-15164">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002535/index.html#r1c1" target="_new">BGC0002535</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>-poly-L-2,4-diaminobutyric acid</td>
<td>Streptomyces celluloflavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>RiPP</td>
<td>SCO-2138</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>RiPP</td>
<td>SLI-2138</td>
<td>Streptomyces lividans TK24</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000131: 0-6280">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000131/index.html#r1c1" target="_new">BGC0000131</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>pyrrolomycin</td>
<td>Streptomyces sp. UC 11065</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002217: 0-33547">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002217/index.html#r1c1" target="_new">BGC0002217</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>HEx-tc3 terpene</td>
<td>Botrytis cinerea T4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778: 0-16475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Other</td>
<td>showdomycin</td>
<td>Streptomyces showdoensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>RiPP</td>
<td>SWA-2138</td>
<td>Streptomyces sp. e14</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r25c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-89"></div> <span class="help-tooltip" id="cb-general-help-89">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r25c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r25c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r25c1_1.svg">
NZ_CP010945 (5203106-5245112): Pseudomonas fluorescens NCIMB 11764 chromosome... (18% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r25c1_2.svg">
NZ_KN639176 (637429-711933): Pseudomonas putida strain PA14H7 Circular chromo... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r25c1_3.svg">
NZ_CP035952 (3923879-4005887): Pseudomonas sp. SNU WT1 chromosome, complete g... (8% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r25c1_4.svg">
NZ_KI547171 (11165-64037): Gamma proteobacterium L18 scaffold00008, whole gen... (9% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r25c1_5.svg">
NZ_RBOI01000041 (87050-131091): Pseudomonas syringae pv. aptata strain G733 P... (5% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r25c1_6.svg">
NZ_CP005970 (3160121-3201723): Pseudomonas syringae UMAF0158 chromosome, comp... (5% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r25c1_7.svg">
NZ_CP020351 (5442210-5484940): Pseudomonas amygdali pv. lachrymans strain NM0... (5% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r25c1_8.svg">
NZ_CP031225 (377402-419624): Pseudomonas amygdali pv. lachrymans str. M301315... (5% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r25c1_9.svg">
NZ_LT629769 (4147937-4189287): Pseudomonas syringae strain 31R1 chromosome I (5% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r25c1_10.svg">
NZ_LT222319 (521801-564397): Pseudomonas cerasi isolate Sour cherry (Prunus c... (5% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r25c1-download">Download graphic</button>
</div>
<div id="clusterblast-r25c1-svg"></div>

</div>
</div>

<div class="body-details-section r25c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-90"></div> <span class="help-tooltip" id="cb-known-help-90">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r25c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-91"></div> <span class="help-tooltip" id="cb-sub-help-91">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r25c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-95"></div> <span class="help-tooltip" id="tfbs-body-help-95">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (3):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
HypR
<div class="collapser collapser-target-HypR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: L-hydroxyproline utilization repressor<br>
<span>Score</span>: 18.94 out of 22.40
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r25c1-11671-HypR"></div>

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 21.53 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r25c1-17667-NrtR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 19.32 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r25c1-20998-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r25c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-96"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-96">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r25c1-domain-bubble-svg" data-filename="NODE_25_length_82593_cov_31.697574_r25c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r25c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (0 - 21024): single NAPAA
</option>


</select>

<div class="details-svg" id="r25c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r25c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r25c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-98"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-98">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r25c1-pfam-details-svg" data-filename="NODE_25_length_82593_cov_31.697574_r25c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r25c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r25c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-99"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-99">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r25c1-tigrfam-details-svg" data-filename="NODE_25_length_82593_cov_31.697574_r25c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r25c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r25c1-nrps_pks_products" data-name="r25c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r25c1-nrps_pks_monomers" data-name="r25c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r25c1-tfbs-finder" data-name="r25c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r25c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-92"></div> <span class="help-tooltip" id="nrps-structures-help-92">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 0 - 21024:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r25c1-1" class="smiles-canvas" data-smiles="NC(CCCCN)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Lys)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r25c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CCCCN)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r25c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-93"></div> <span class="help-tooltip" id="nrps-monomers-help-93">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">ctg25_3</span></strong>: Lys
<div class="collapser collapser-target-ctg25_3 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (37..434)</strong>: Lys
<div class="collapser collapser-target-ctg25_3 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Lys
<div class="collapser collapser-target-ctg25_3-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DMEHNGTVSK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Lys <span class="serif">DAEDNGTVSK</span> (59% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r25c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-94"></div> <span class="help-tooltip" id="tfbs-side-help-94">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/HypR" target="_blank">HypR</a></td>
<td>L-hydroxyproline utilization repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>




<div class="page" id='r27c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_27_length_76213_cov_33.724217 - Region 1 - NRP-metallophore,NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-186"></div> <span class="help-tooltip" id="region_svg-help-186">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_27_length_76213_cov_33.724217.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r27c1-svg" data-filename="NODE_27_length_76213_cov_33.724217_r27c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 69,682 nt. (total: 69,682 nt)

<a class="cluster-rules-header" id="r27c1-rules-header" href="#r27c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r27c1-rules">NRP-metallophore: (VibH_like or Cy_tandem or (cds(Condensation and AMP-binding) and ((IBH_Asp and not SBH_Asp) or IBH_His or TBH_Asp or CyanoBH_Asp1 or CyanoBH_Asp2 or IPL or SalSyn or (EntA and EntC) or (GrbD and GrbE) or (FbnL and FbnM) or PvdO or PvdP or (Orn_monoox and not (KtzT or MetRS-like)) or Lys_monoox or VbsL)))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r27c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-187"></div> <span class="help-tooltip" id="focus-panel-help-187">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r27c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r27c1-gene-table-details" data-name="r27c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r27c1-nrps_pks" data-name="r27c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r27c1-MIBiG-cluster-compare" data-name="r27c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r27c1-clusterblast" data-name="r27c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r27c1-knownclusterblast" data-name="r27c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r27c1-subclusterblast" data-name="r27c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r27c1-tfbs-finder" data-name="r27c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r27c1-nrps-pks-bubbles" data-name="r27c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r27c1-pfam-details" data-name="r27c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r27c1-tigrfam-details" data-name="r27c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r27c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-110"></div> <span class="help-tooltip" id="gt-help-tooltip-help-110">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r27c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r27c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r27c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r27c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r27c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-100"></div> <span class="help-tooltip" id="nrps-domain-help-100">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r27c1-details-svg" data-filename="NODE_27_length_76213_cov_33.724217_r27c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r27c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r27c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-101"></div> <span class="help-tooltip" id="MIBiG-help-101">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r27c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r27c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r27c1"></div>
<div id="comparison-MIBiG-r27c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRP-metallophore</span></th>

<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050: 0-19730">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050</a></td>





<td class="cc-heat-cell" title="0.72 (id:0.44, order:1.00, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>






<td class="cc-heat-cell" title="0.72 (id:0.44, order:1.00, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>


<td>0.80</td>
<td>NRP, Polyketide</td>

<td>thalassospiramide A</td>


<td>Tistrella bauzanensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>





<td class="cc-heat-cell" title="0.71 (id:0.44, order:1.00, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>






<td class="cc-heat-cell" title="0.71 (id:0.44, order:1.00, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>


<td>0.78</td>
<td>NRP</td>

<td>taxlllaid A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.46, order:1.00, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>






<td class="cc-heat-cell" title="0.70 (id:0.46, order:1.00, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.76</td>
<td>NRP</td>

<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>


<td>Pseudomonas sp. QS1027</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001629: 209-63961">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001629/index.html#r1c1" target="_new">BGC0001629</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.45, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.45, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.71</td>
<td>NRP</td>

<td>jessenipeptin</td>


<td>Pseudomonas sp. QS1027</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000438: 137-73725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000438/index.html#r1c1" target="_new">BGC0000438</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.45, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.45, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.70</td>
<td>NRP</td>

<td>syringopeptin 25A</td>


<td>Pseudomonas syringae pv. syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.67</td>
<td>NRP</td>

<td>thermoactinoamide A</td>


<td>Thermoactinomyces sp. AS95</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.65</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.62</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.45, order:0.75, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.45, order:0.75, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.61</td>
<td>NRP</td>

<td>sevadicin</td>


<td>Paenibacillus larvae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002476: 0-10330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002476/index.html#r1c1" target="_new">BGC0002476</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.61</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Escherichia coli str. K-12 substr. MG1655</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r27c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050: 0-19730">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050</a></td>


<td class="cc-heat-cell" title="0.72 (id:0.44, order:1.00, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
<td>0.72</td>
<td>NRP, Polyketide</td>
<td>thalassospiramide A</td>
<td>Tistrella bauzanensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>


<td class="cc-heat-cell" title="0.71 (id:0.44, order:1.00, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
<td>0.71</td>
<td>NRP</td>
<td>taxlllaid A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.46, order:1.00, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP</td>
<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>
<td>Pseudomonas sp. QS1027</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001629: 209-63961">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001629/index.html#r1c1" target="_new">BGC0001629</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.45, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>jessenipeptin</td>
<td>Pseudomonas sp. QS1027</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000438: 137-73725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000438/index.html#r1c1" target="_new">BGC0000438</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.45, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>syringopeptin 25A</td>
<td>Pseudomonas syringae pv. syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>thermoactinoamide A</td>
<td>Thermoactinomyces sp. AS95</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.45, order:0.75, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>sevadicin</td>
<td>Paenibacillus larvae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002476: 0-10330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002476/index.html#r1c1" target="_new">BGC0002476</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Escherichia coli str. K-12 substr. MG1655</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r27c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-102"></div> <span class="help-tooltip" id="cb-general-help-102">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r27c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r27c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r27c1_1.svg">
NZ_LT629796 (51952-114136): Pseudomonas mandelii strain LMG 21607 chromosome I (68% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r27c1_2.svg">
NZ_CP053929 (6646944-6709674): Pseudomonas sp. B14-6 chromosome, complete genome (68% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r27c1_3.svg">
NZ_CP005960 (1579045-1641236): Pseudomonas mandelii JR-1 chromosome, complete... (68% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r27c1_4.svg">
NZ_VIUJ01000012 (8175-70917): Pseudomonas sp. SJZ077 Ga0314243 112, whole gen... (68% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r27c1_5.svg">
NZ_RHGD01000001 (5141385-5203575): Pseudomonas mandelii strain B2/38-Q-III Sc... (68% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r27c1_6.svg">
NZ_CM001561 (2053576-2115971): Pseudomonas fluorescens R124 chromosome, whole... (67% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r27c1_7.svg">
NZ_NEHJ01000010 (92528-156102): Pseudomonas sp. R43(2017) R43 10, whole genom... (65% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r27c1_8.svg">
NZ_CP029608 (2143026-2202957): Pseudomonas kribbensis strain 46-2 chromosome,... (69% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r27c1_9.svg">
NZ_FOYE01000003 (423858-497811): Pseudomonas sp. NFIX49, whole genome shotgun... (51% of genes show similarity), NRPS,terpene
</option>

<option value = "svg/clusterblast_r27c1_10.svg">
NZ_CP048408 (2355111-2426762): Pseudomonas fluorescens strain DR397 chromosom... (51% of genes show similarity), NRPS,terpene
</option>

</select>
<button id= "clusterblast-r27c1-download">Download graphic</button>
</div>
<div id="clusterblast-r27c1-svg"></div>

</div>
</div>

<div class="body-details-section r27c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-103"></div> <span class="help-tooltip" id="cb-known-help-103">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r27c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r27c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r27c1_1.svg">
Pf-5 pyoverdine
</option>

<option value = "svg/knownclusterblast_r27c1_2.svg">
pyoverdine SMX-1
</option>

<option value = "svg/knownclusterblast_r27c1_3.svg">
crochelin A
</option>

<option value = "svg/knownclusterblast_r27c1_4.svg">
azotobactin D
</option>

<option value = "svg/knownclusterblast_r27c1_5.svg">
viscosin
</option>

<option value = "svg/knownclusterblast_r27c1_6.svg">
taiwachelin
</option>

<option value = "svg/knownclusterblast_r27c1_7.svg">
pacifibactin
</option>

<option value = "svg/knownclusterblast_r27c1_8.svg">
delftibactin A/delftibactin B
</option>

<option value = "svg/knownclusterblast_r27c1_9.svg">
variobactin A/variobactin B
</option>

<option value = "svg/knownclusterblast_r27c1_10.svg">
variochelin A/variochelin B
</option>

</select>
<button id="knownclusterblast-r27c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r27c1-svg"></div>

</div>
</div>

<div class="body-details-section r27c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-104"></div> <span class="help-tooltip" id="cb-sub-help-104">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r27c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-108"></div> <span class="help-tooltip" id="tfbs-body-help-108">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (3):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
DasR
<div class="collapser collapser-target-DasR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: N-acetylglucosamine dependent repressor<br>
<span>Score</span>: 20.40 out of 27.04
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r27c1-33-DasR"></div>

<div class="tfbs-hit-detail">
DmdR1
<div class="collapser collapser-target-DmdR1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Iron(II)-dependent repressor<br>
<span>Score</span>: 22.08 out of 36.53
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r27c1-2952-DmdR1"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 19.14 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r27c1-66096-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r27c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-109"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-109">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r27c1-domain-bubble-svg" data-filename="NODE_27_length_76213_cov_33.724217_r27c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r27c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (0 - 69682): chemical hybrid NRP-metallophore-NRPS
</option>


</select>

<div class="details-svg" id="r27c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r27c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r27c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-111"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-111">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r27c1-pfam-details-svg" data-filename="NODE_27_length_76213_cov_33.724217_r27c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r27c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r27c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-112"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-112">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r27c1-tigrfam-details-svg" data-filename="NODE_27_length_76213_cov_33.724217_r27c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r27c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r27c1-nrps_pks_products" data-name="r27c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r27c1-nrps_pks_monomers" data-name="r27c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r27c1-tfbs-finder" data-name="r27c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r27c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-105"></div> <span class="help-tooltip" id="nrps-structures-help-105">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 0 - 69682:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r27c1-1" class="smiles-canvas" data-smiles="NC(CCCCN)C(=O)NC(CC(=O)O)C(=O)NC(C)C(=O)NC(C(O)C)C(=O)NC(C)C(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Lys) + (D-Asp - Ala) + (D-Thr - D-Ala - D-X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys&amp;nrps2=d-asp,ala&amp;nrps3=d-thr,d-ala,d-x" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys*&amp;nrps2=d-asp*,ala*&amp;nrps3=d-thr*,d-ala*,d-x*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r27c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CCCCN)C(=O)NC(CC(=O)O)C(=O)NC(C)C(=O)NC(C(O)C)C(=O)NC(C)C(=O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r27c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-106"></div> <span class="help-tooltip" id="nrps-monomers-help-106">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">ctg27_18</span></strong>: Thr - Ala - X
<div class="collapser collapser-target-ctg27_18 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr,ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*,ala*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (511..913)</strong>: Thr
<div class="collapser collapser-target-ctg27_18 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Thr
<div class="collapser collapser-target-ctg27_18-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>thr (Thr)</dd>
<dt>Single AA prediction:</dt>
<dd>thr (Thr)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DFWNVGMVHK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Thr <span class="serif">DFWNVGMVHK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (2021..2411)</strong>: Ala
<div class="collapser collapser-target-ctg27_18 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala
<div class="collapser collapser-target-ctg27_18-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Tiny, hydrophilic, transition to aliphatic (Gly, Ala)</dd>
<dt>Single AA prediction:</dt>
<dd>ala (Ala)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLWNNALTYK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ala <span class="serif">DLWNNALTYK</span> (94% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (3498..3897)</strong>: X
<div class="collapser collapser-target-ctg27_18 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-ctg27_18-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Long positively charged side chain (Orn, Lys, Arg)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>lys (Lys)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DGECCGGVTK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>OH-Orn <span class="serif">DGEACGGVTK</span> (82% 8 match)</dd>
<dd>Fo-OH-Orn <span class="serif">DGEVCGGVTK</span> (76% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

<strong><span class="serif">ctg27_19</span></strong>: Asp - Ala
<div class="collapser collapser-target-ctg27_19 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=asp,ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=asp*,ala*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (505..903)</strong>: Asp
<div class="collapser collapser-target-ctg27_19 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Asp
<div class="collapser collapser-target-ctg27_19-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>asp,asn (Asp, Asn)</dd>
<dt>Single AA prediction:</dt>
<dd>asp (Asp)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLTKIGHVGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Asp <span class="serif">DLTKIGHVGK</span> (94% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (2008..2398)</strong>: Ala
<div class="collapser collapser-target-ctg27_19 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala
<div class="collapser collapser-target-ctg27_19-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Tiny, hydrophilic, transition to aliphatic (Gly, Ala)</dd>
<dt>Single AA prediction:</dt>
<dd>ala (Ala)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLWNNALTYK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ala <span class="serif">DLWNNALTYK</span> (97% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">ctg27_20</span></strong>: Lys
<div class="collapser collapser-target-ctg27_20 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (519..926)</strong>: Lys
<div class="collapser collapser-target-ctg27_20 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Lys
<div class="collapser collapser-target-ctg27_20-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DGEDHGTVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Lys <span class="serif">DGEDHGTVVK</span> (88% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r27c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-107"></div> <span class="help-tooltip" id="tfbs-side-help-107">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/DasR" target="_blank">DasR</a></td>
<td>N-acetylglucosamine dependent repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/DmdR1" target="_blank">DmdR1</a></td>
<td>Iron(II)-dependent repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>




<div class="page" id='r29c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_29_length_74090_cov_33.425885 - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-188"></div> <span class="help-tooltip" id="region_svg-help-188">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_29_length_74090_cov_33.425885.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r29c1-svg" data-filename="NODE_29_length_74090_cov_33.425885_r29c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 16,216 - 28,423 nt. (total: 12,208 nt)

<a class="cluster-rules-header" id="r29c1-rules-header" href="#r29c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r29c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193 or (YcaO or TIGR03604))</div>

<div class="region-svg-container">
<div id='r29c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-189"></div> <span class="help-tooltip" id="focus-panel-help-189">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r29c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r29c1-gene-table-details" data-name="r29c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r29c1-MIBiG-cluster-compare" data-name="r29c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r29c1-clusterblast" data-name="r29c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r29c1-knownclusterblast" data-name="r29c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r29c1-subclusterblast" data-name="r29c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r29c1-tfbs-finder" data-name="r29c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r29c1-pfam-details" data-name="r29c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r29c1-tigrfam-details" data-name="r29c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r29c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-119"></div> <span class="help-tooltip" id="gt-help-tooltip-help-119">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r29c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r29c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r29c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r29c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r29c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-113"></div> <span class="help-tooltip" id="MIBiG-help-113">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r29c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r29c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r29c1"></div>
<div id="comparison-MIBiG-r29c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001285: 8-22479">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001285/index.html#r1c1" target="_new">BGC0001285</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.65, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Other (Fatty acid)</td>

<td>pseudopyronine A, pseudopyronine B</td>


<td>Pseudomonas putida</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000944: 0-6093">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000944/index.html#r1c1" target="_new">BGC0000944</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Other (Non-NRP siderophore)</td>

<td>staphyloferrin A</td>


<td>Staphylococcus aureus subsp. aureus NCTC 8325</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000937: 0-23647">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000937/index.html#r1c1" target="_new">BGC0000937</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.43, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>Other</td>

<td>fosfazinomycin A, fosfazinomycin B</td>


<td>Streptomyces sp. WM6372</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002017: 0-12514">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002017/index.html#r1c1" target="_new">BGC0002017</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.44, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>NRP</td>

<td>potensibactin</td>


<td>Nocardiopsis potens DSM 45234</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000236: 0-36887">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000236/index.html#r1c1" target="_new">BGC0000236</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.47, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>kinamycin</td>


<td>Streptomyces murayamaensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002016: 0-28409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002016/index.html#r1c1" target="_new">BGC0002016</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.47, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>lugdunomycin</td>


<td>Streptomyces sp. QL37</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001596: 0-46688">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001596/index.html#r1c1" target="_new">BGC0001596</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.43, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>fluostatins M-Q</td>


<td>Streptomyces albus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001904: 2-34103">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001904/index.html#r1c1" target="_new">BGC0001904</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.43, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>fluostatin</td>


<td>Micromonospora rosaria</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001442: 0-32297">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001442/index.html#r1c1" target="_new">BGC0001442</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.43, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP</td>

<td>actinonin</td>


<td>Streptomyces sp. strain ATCC 14903</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000679: 0-47916">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000679/index.html#r1c1" target="_new">BGC0000679</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.43, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>Terpene</td>

<td>diazepinomicin</td>


<td>Micromonospora sp. M42</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r29c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001285: 8-22479">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001285/index.html#r1c1" target="_new">BGC0001285</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.65, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Other (Fatty acid)</td>
<td>pseudopyronine A, pseudopyronine B</td>
<td>Pseudomonas putida</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000944: 0-6093">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000944/index.html#r1c1" target="_new">BGC0000944</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Other (Non-NRP siderophore)</td>
<td>staphyloferrin A</td>
<td>Staphylococcus aureus subsp. aureus NCTC 8325</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000937: 0-23647">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000937/index.html#r1c1" target="_new">BGC0000937</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.43, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>Other</td>
<td>fosfazinomycin A, fosfazinomycin B</td>
<td>Streptomyces sp. WM6372</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002017: 0-12514">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002017/index.html#r1c1" target="_new">BGC0002017</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.44, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>NRP</td>
<td>potensibactin</td>
<td>Nocardiopsis potens DSM 45234</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000236: 0-36887">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000236/index.html#r1c1" target="_new">BGC0000236</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.47, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Polyketide</td>
<td>kinamycin</td>
<td>Streptomyces murayamaensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002016: 0-28409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002016/index.html#r1c1" target="_new">BGC0002016</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.47, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>Polyketide</td>
<td>lugdunomycin</td>
<td>Streptomyces sp. QL37</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001596: 0-46688">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001596/index.html#r1c1" target="_new">BGC0001596</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.43, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>Polyketide</td>
<td>fluostatins M-Q</td>
<td>Streptomyces albus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001904: 2-34103">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001904/index.html#r1c1" target="_new">BGC0001904</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.43, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Polyketide</td>
<td>fluostatin</td>
<td>Micromonospora rosaria</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001442: 0-32297">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001442/index.html#r1c1" target="_new">BGC0001442</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.43, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP</td>
<td>actinonin</td>
<td>Streptomyces sp. strain ATCC 14903</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000679: 0-47916">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000679/index.html#r1c1" target="_new">BGC0000679</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.43, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>Terpene</td>
<td>diazepinomicin</td>
<td>Micromonospora sp. M42</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r29c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-114"></div> <span class="help-tooltip" id="cb-general-help-114">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r29c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r29c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r29c1_1.svg">
NZ_SOCT01000001 (370617-397885): Pseudomonas sp. LP 7 YM Ga0304827 101, whole... (6% of genes show similarity), thiopeptide
</option>

<option value = "svg/clusterblast_r29c1_2.svg">
NC_021237 (3114320-3167070): Pseudomonas protegens CHA0, complete genome (5% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r29c1_3.svg">
NZ_PPEF01000017 (45756-98506): Pseudomonas protegens strain AS15 scaffold 17,... (5% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r29c1_4.svg">
NZ_FPKJ01000001 (1054786-1107536): Pseudomonas sp. NFPP16, whole genome shotg... (5% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r29c1_5.svg">
NC_004129 (3074355-3127105): Pseudomonas protegens Pf-5, complete sequence (5% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r29c1_6.svg">
NZ_MRUC01000003 (1858971-1911721): Pseudomonas protegens strain PA25 NODE 4 l... (5% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r29c1_7.svg">
NZ_CP013184 (3217566-3270316): Pseudomonas protegens strain H78, complete genome (5% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r29c1_8.svg">
NZ_RJUX01000001 (2309519-2362269): Pseudomonas protegens strain JUb28 Ga03047... (5% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r29c1_9.svg">
NZ_AP014522 (3089356-3142106): Pseudomonas protegens Cab57 (5% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r29c1_10.svg">
NZ_QJRN01000020 (54607-95273): Pseudomonas protegens strain MB-090624 NODE 20... (5% of genes show similarity), bacteriocin,thiopeptide
</option>

</select>
<button id= "clusterblast-r29c1-download">Download graphic</button>
</div>
<div id="clusterblast-r29c1-svg"></div>

</div>
</div>

<div class="body-details-section r29c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-115"></div> <span class="help-tooltip" id="cb-known-help-115">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r29c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-116"></div> <span class="help-tooltip" id="cb-sub-help-116">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r29c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-118"></div> <span class="help-tooltip" id="tfbs-body-help-118">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 19.08 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r29c1-21216-ZuR"></div>

<div class="tfbs-hit-detail">
DmdR1
<div class="collapser collapser-target-DmdR1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Iron(II)-dependent repressor<br>
<span>Score</span>: 22.09 out of 36.53
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r29c1-26848-DmdR1"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r29c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-120"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-120">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r29c1-pfam-details-svg" data-filename="NODE_29_length_74090_cov_33.425885_r29c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r29c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r29c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-121"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-121">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r29c1-tigrfam-details-svg" data-filename="NODE_29_length_74090_cov_33.425885_r29c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r29c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r29c1-tfbs-finder" data-name="r29c1-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r29c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-117"></div> <span class="help-tooltip" id="tfbs-side-help-117">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/DmdR1" target="_blank">DmdR1</a></td>
<td>Iron(II)-dependent repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r31c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_31_length_58202_cov_34.183880 - Region 1 - NAGGN <div class="help-container"> <div class="help-icon" data-id="region_svg-help-190"></div> <span class="help-tooltip" id="region_svg-help-190">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_31_length_58202_cov_34.183880.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r31c1-svg" data-filename="NODE_31_length_58202_cov_34.183880_r31c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 20,612 - 35,337 nt. (total: 14,726 nt)

<a class="cluster-rules-header" id="r31c1-rules-header" href="#r31c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r31c1-rules">NAGGN: ((TIGR03104 or cds(GATase_7 and Asn_synthase)) and TIGR03103 and (TIGR03106 or Peptidase_M42))</div>

<div class="region-svg-container">
<div id='r31c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-191"></div> <span class="help-tooltip" id="focus-panel-help-191">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r31c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r31c1-gene-table-details" data-name="r31c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r31c1-MIBiG-cluster-compare" data-name="r31c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r31c1-clusterblast" data-name="r31c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r31c1-knownclusterblast" data-name="r31c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r31c1-subclusterblast" data-name="r31c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r31c1-pfam-details" data-name="r31c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r31c1-tigrfam-details" data-name="r31c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r31c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-126"></div> <span class="help-tooltip" id="gt-help-tooltip-help-126">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r31c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r31c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r31c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r31c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r31c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-122"></div> <span class="help-tooltip" id="MIBiG-help-122">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r31c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r31c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r31c1"></div>
<div id="comparison-MIBiG-r31c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NAGGN</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001880: 0-8263">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001880/index.html#r1c1" target="_new">BGC0001880</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>RiPP</td>

<td>scytodecamide</td>


<td>Scytonema sp. UIC 10036</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000846: 192-22933">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000846/index.html#r1c1" target="_new">BGC0000846</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Other</td>

<td>tabtoxin</td>


<td>Pseudomonas syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888: 0-4342">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Other</td>

<td>bacilysin</td>


<td>Bacillus sp. CS93</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000885: 0-16706">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000885/index.html#r1c1" target="_new">BGC0000885</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.44, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Other</td>

<td>4-hydroxy-3-nitrosobenzamide</td>


<td>Streptomyces murayamaensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231: 189-5010">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Polyketide</td>

<td>griseusin A, griseusin B</td>


<td>Streptomyces griseus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000937: 0-23647">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000937/index.html#r1c1" target="_new">BGC0000937</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Other</td>

<td>fosfazinomycin A, fosfazinomycin B</td>


<td>Streptomyces sp. WM6372</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002429: 0-14296">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002429/index.html#r1c1" target="_new">BGC0002429</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Terpene, Polyketide</td>

<td>higginsianin B</td>


<td>Colletotrichum higginsianum IMI 349063</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Other</td>

<td>citrinin</td>


<td>Monascus aurantiacus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000866: 467-5165">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000866/index.html#r1c1" target="_new">BGC0000866</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Other</td>

<td>polyhydroxyalkanoate</td>


<td>Burkholderia sp. DSM 9242</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000867: 21-5822">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000867/index.html#r1c1" target="_new">BGC0000867</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Other</td>

<td>polyhydroxyalkanoic acids</td>


<td>Ectothiorhodospira shaposhnikovii</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r31c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001880: 0-8263">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001880/index.html#r1c1" target="_new">BGC0001880</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>RiPP</td>
<td>scytodecamide</td>
<td>Scytonema sp. UIC 10036</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000846: 192-22933">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000846/index.html#r1c1" target="_new">BGC0000846</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Other</td>
<td>tabtoxin</td>
<td>Pseudomonas syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888: 0-4342">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Other</td>
<td>bacilysin</td>
<td>Bacillus sp. CS93</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000885: 0-16706">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000885/index.html#r1c1" target="_new">BGC0000885</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.44, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Other</td>
<td>4-hydroxy-3-nitrosobenzamide</td>
<td>Streptomyces murayamaensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231: 189-5010">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Polyketide</td>
<td>griseusin A, griseusin B</td>
<td>Streptomyces griseus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000937: 0-23647">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000937/index.html#r1c1" target="_new">BGC0000937</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Other</td>
<td>fosfazinomycin A, fosfazinomycin B</td>
<td>Streptomyces sp. WM6372</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002429: 0-14296">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002429/index.html#r1c1" target="_new">BGC0002429</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Terpene, Polyketide</td>
<td>higginsianin B</td>
<td>Colletotrichum higginsianum IMI 349063</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Other</td>
<td>citrinin</td>
<td>Monascus aurantiacus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000866: 467-5165">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000866/index.html#r1c1" target="_new">BGC0000866</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Other</td>
<td>polyhydroxyalkanoate</td>
<td>Burkholderia sp. DSM 9242</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000867: 21-5822">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000867/index.html#r1c1" target="_new">BGC0000867</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Other</td>
<td>polyhydroxyalkanoic acids</td>
<td>Ectothiorhodospira shaposhnikovii</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r31c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-123"></div> <span class="help-tooltip" id="cb-general-help-123">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r31c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r31c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r31c1_1.svg">
NZ_QRAV01000002 (400736-415614): Pseudomonas jessenii strain E2333 Ga0222391 ... (72% of genes show similarity), NAGGN
</option>

<option value = "svg/clusterblast_r31c1_2.svg">
NZ_CP019397 (4541155-4555940): Pseudomonas sp. S19 chromosome, complete genome (62% of genes show similarity), NAGGN
</option>

<option value = "svg/clusterblast_r31c1_3.svg">
NZ_CP027710 (5255067-5269896): Pseudomonas chlororaphis subsp. piscium strain... (60% of genes show similarity), NAGGN
</option>

<option value = "svg/clusterblast_r31c1_4.svg">
NZ_CP059162 (4952191-4967028): Pseudomonas chlororaphis subsp. aurantiaca str... (56% of genes show similarity), NAGGN
</option>

<option value = "svg/clusterblast_r31c1_5.svg">
NZ_CP027719 (5321447-5336281): Pseudomonas chlororaphis subsp. aureofaciens s... (75% of genes show similarity), NAGGN
</option>

<option value = "svg/clusterblast_r31c1_6.svg">
NZ_JACHKI010000001 (216085-230898): Pseudomonas rhodesiae strain S00002 Ga043... (52% of genes show similarity), NAGGN
</option>

<option value = "svg/clusterblast_r31c1_7.svg">
NZ_LMIY01000012 (212884-227745): Pseudomonas sp. Root9 contig 2, whole genome... (52% of genes show similarity), NAGGN
</option>

<option value = "svg/clusterblast_r31c1_8.svg">
NZ_CM001490 (4993232-5008066): Pseudomonas chlororaphis O6 chromosome, whole ... (75% of genes show similarity), NAGGN
</option>

<option value = "svg/clusterblast_r31c1_9.svg">
NZ_JACAPG010000011 (84985-99704): Pseudomonas reactans strain C5002 run0228 C... (64% of genes show similarity), NAGGN
</option>

<option value = "svg/clusterblast_r31c1_10.svg">
NZ_SOEK01000036 (1927-16805): Pseudomonas sp. OV341 Ga0215802 136, whole geno... (81% of genes show similarity), NAGGN
</option>

</select>
<button id= "clusterblast-r31c1-download">Download graphic</button>
</div>
<div id="clusterblast-r31c1-svg"></div>

</div>
</div>

<div class="body-details-section r31c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-124"></div> <span class="help-tooltip" id="cb-known-help-124">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r31c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-125"></div> <span class="help-tooltip" id="cb-sub-help-125">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r31c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-127"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-127">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r31c1-pfam-details-svg" data-filename="NODE_31_length_58202_cov_34.183880_r31c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r31c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r31c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-128"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-128">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r31c1-tigrfam-details-svg" data-filename="NODE_31_length_58202_cov_34.183880_r31c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r31c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>




<div class="page" id='r36c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_36_length_46894_cov_34.610291 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-192"></div> <span class="help-tooltip" id="region_svg-help-192">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_36_length_46894_cov_34.610291.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r36c1-svg" data-filename="NODE_36_length_46894_cov_34.610291_r36c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 35,294 nt. (total: 35,294 nt)

<a class="cluster-rules-header" id="r36c1-rules-header" href="#r36c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r36c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r36c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-193"></div> <span class="help-tooltip" id="focus-panel-help-193">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r36c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r36c1-gene-table-details" data-name="r36c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r36c1-nrps_pks" data-name="r36c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r36c1-MIBiG-cluster-compare" data-name="r36c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r36c1-clusterblast" data-name="r36c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r36c1-knownclusterblast" data-name="r36c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r36c1-subclusterblast" data-name="r36c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r36c1-tfbs-finder" data-name="r36c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r36c1-nrps-pks-bubbles" data-name="r36c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r36c1-pfam-details" data-name="r36c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r36c1-tigrfam-details" data-name="r36c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r36c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-139"></div> <span class="help-tooltip" id="gt-help-tooltip-help-139">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r36c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r36c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r36c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r36c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r36c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-129"></div> <span class="help-tooltip" id="nrps-domain-help-129">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r36c1-details-svg" data-filename="NODE_36_length_46894_cov_34.610291_r36c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r36c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r36c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-130"></div> <span class="help-tooltip" id="MIBiG-help-130">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r36c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r36c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r36c1"></div>
<div id="comparison-MIBiG-r36c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>





<td class="cc-heat-cell" title="0.77 (id:0.54, order:1.00, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>


<td>0.47</td>
<td>NRP</td>

<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>


<td>Pseudomonas sp. QS1027</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>





<td class="cc-heat-cell" title="0.73 (id:0.45, order:1.00, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>


<td>0.41</td>
<td>NRP</td>

<td>taxlllaid A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.72 (id:0.50, order:0.75, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>


<td>0.41</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000435: 0-32188">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000435/index.html#r1c1" target="_new">BGC0000435</a></td>





<td class="cc-heat-cell" title="0.72 (id:0.54, order:0.83, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>syringafactin</td>


<td>Pseudomonas syringae pv. tomato str. DC3000</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000323: 0-32510">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000323/index.html#r1c1" target="_new">BGC0000323</a></td>





<td class="cc-heat-cell" title="0.72 (id:0.53, order:0.83, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>cichofactin A, cichofactin B</td>


<td>Pseudomonas cichorii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.38</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.50, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.49, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.49, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r36c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>


<td class="cc-heat-cell" title="0.77 (id:0.54, order:1.00, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
<td>0.77</td>
<td>NRP</td>
<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>
<td>Pseudomonas sp. QS1027</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>


<td class="cc-heat-cell" title="0.73 (id:0.45, order:1.00, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
<td>0.73</td>
<td>NRP</td>
<td>taxlllaid A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.72 (id:0.50, order:0.75, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
<td>0.72</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000435: 0-32188">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000435/index.html#r1c1" target="_new">BGC0000435</a></td>


<td class="cc-heat-cell" title="0.72 (id:0.54, order:0.83, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
<td>0.72</td>
<td>NRP</td>
<td>syringafactin</td>
<td>Pseudomonas syringae pv. tomato str. DC3000</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000323: 0-32510">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000323/index.html#r1c1" target="_new">BGC0000323</a></td>


<td class="cc-heat-cell" title="0.72 (id:0.53, order:0.83, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
<td>0.72</td>
<td>NRP</td>
<td>cichofactin A, cichofactin B</td>
<td>Pseudomonas cichorii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.50, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.49, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.49, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r36c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-131"></div> <span class="help-tooltip" id="cb-general-help-131">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r36c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r36c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r36c1_1.svg">
NZ_WHPA01000022 (881007-935095): Pseudomonas viridiflava strain BAV 2572 NODE... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r36c1_2.svg">
NZ_LT963402 (2777436-2858944): Pseudomonas syringae pv. avii isolate CFBP3846... (10% of genes show similarity), NRPS,siderophore
</option>

<option value = "svg/clusterblast_r36c1_3.svg">
NZ_FYDI01000007 (137372-314945): Pseudomonas sp. Irchel 3A18 isolate Pseudomo... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r36c1_4.svg">
NZ_QPDS01000001 (87715-265429): Pseudomonas cichorii strain 473 Contig 1, who... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r36c1_5.svg">
NZ_CP007039 (3008321-3185141): Pseudomonas cichorii JBC1 chromosome, complete... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r36c1_6.svg">
NZ_CP041189 (5996702-6101184): Pseudomonas sp. NIBRBAC000502773 chromosome, c... (18% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r36c1_7.svg">
NZ_CP060201 (6624206-6724422): Pseudomonas protegens strain H1F5C chromosome,... (20% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r36c1_8.svg">
NZ_CP031641 (2752355-2861769): Pseudomonas parafulva strain JBCS1880 chromoso... (18% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r36c1_9.svg">
NZ_CP009747 (2049309-2158911): Pseudomonas parafulva strain CRS01-1 chromosom... (18% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r36c1_10.svg">
NZ_CM001514 (3644222-3743043): Pseudomonas synxantha BG33R chromosome, whole ... (10% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r36c1-download">Download graphic</button>
</div>
<div id="clusterblast-r36c1-svg"></div>

</div>
</div>

<div class="body-details-section r36c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-132"></div> <span class="help-tooltip" id="cb-known-help-132">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r36c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r36c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r36c1_1.svg">
gacamide A
</option>

<option value = "svg/knownclusterblast_r36c1_2.svg">
orfamide A/orfamide C
</option>

<option value = "svg/knownclusterblast_r36c1_3.svg">
orfamide B
</option>

<option value = "svg/knownclusterblast_r36c1_4.svg">
putisolvin
</option>

<option value = "svg/knownclusterblast_r36c1_5.svg">
putisolvin III/putisolvin VI/putisolvin V
</option>

<option value = "svg/knownclusterblast_r36c1_6.svg">
corpeptin A/corpeptin B
</option>

<option value = "svg/knownclusterblast_r36c1_7.svg">
cichopeptin
</option>

<option value = "svg/knownclusterblast_r36c1_8.svg">
sessilin A
</option>

<option value = "svg/knownclusterblast_r36c1_9.svg">
viscosinamide A/pseudodesmin A
</option>

<option value = "svg/knownclusterblast_r36c1_10.svg">
bananamide 1/bananamide 2/bananamide 3
</option>

</select>
<button id="knownclusterblast-r36c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r36c1-svg"></div>

</div>
</div>

<div class="body-details-section r36c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-133"></div> <span class="help-tooltip" id="cb-sub-help-133">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r36c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-137"></div> <span class="help-tooltip" id="tfbs-body-help-137">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.93 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r36c1-20075-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r36c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-138"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-138">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r36c1-domain-bubble-svg" data-filename="NODE_36_length_46894_cov_34.610291_r36c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r36c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (0 - 35294): single NRPS
</option>


</select>

<div class="details-svg" id="r36c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r36c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r36c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-140"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-140">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r36c1-pfam-details-svg" data-filename="NODE_36_length_46894_cov_34.610291_r36c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r36c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r36c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-141"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-141">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r36c1-tigrfam-details-svg" data-filename="NODE_36_length_46894_cov_34.610291_r36c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r36c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r36c1-nrps_pks_products" data-name="r36c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r36c1-nrps_pks_monomers" data-name="r36c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r36c1-tfbs-finder" data-name="r36c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r36c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-134"></div> <span class="help-tooltip" id="nrps-structures-help-134">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 0 - 35294:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r36c1-1" class="smiles-canvas" data-smiles="NC(CC(C)C)C(=O)NC([*])C(=O)NC(CC(C)C)C(=O)NC(C(C)CC)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Leu - X) + (Leu - Ile)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu&amp;nrps2=leu,ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*,x&amp;nrps2=leu*,ile*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r36c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CC(C)C)C(=O)NC([*])C(=O)NC(CC(C)C)C(=O)NC(C(C)CC)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r36c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-135"></div> <span class="help-tooltip" id="nrps-monomers-help-135">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">ctg36_1</span></strong>: Leu - X
<div class="collapser collapser-target-ctg36_1 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*" target="_new">relaxed</a>
</dd>
<br>







<dd><strong>AMP-binding (694..1082)</strong>: Leu
<div class="collapser collapser-target-ctg36_1 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Leu
<div class="collapser collapser-target-ctg36_1-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>leu (Leu)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAWFLGNVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Leu <span class="serif">DAWFLGNVVK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1754..2153)</strong>: X
<div class="collapser collapser-target-ctg36_1 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-ctg36_1-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>asp,asn (Asp, Asn)</dd>
<dt>Single AA prediction:</dt>
<dd>asp (Asp)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DGWKVGVVDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Glu <span class="serif">DGWKFGVVDK</span> (94% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">ctg36_2</span></strong>: Leu - Ile
<div class="collapser collapser-target-ctg36_2 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu,ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*,ile*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (568..957)</strong>: Leu
<div class="collapser collapser-target-ctg36_2 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Leu
<div class="collapser collapser-target-ctg36_2-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>leu (Leu)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAWFLGNVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Leu <span class="serif">DAWFLGNVVK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1614..2009)</strong>: Ile
<div class="collapser collapser-target-ctg36_2 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ile
<div class="collapser collapser-target-ctg36_2-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>ile (Ile)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAMFLGCTYK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ile <span class="serif">DAMFLGCTYK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r36c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-136"></div> <span class="help-tooltip" id="tfbs-side-help-136">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>




<div class="page" id='r38c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_38_length_43995_cov_34.355731 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-194"></div> <span class="help-tooltip" id="region_svg-help-194">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_38_length_43995_cov_34.355731.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r38c1-svg" data-filename="NODE_38_length_43995_cov_34.355731_r38c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 41,734 nt. (total: 41,734 nt)

<a class="cluster-rules-header" id="r38c1-rules-header" href="#r38c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r38c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r38c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-195"></div> <span class="help-tooltip" id="focus-panel-help-195">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r38c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r38c1-gene-table-details" data-name="r38c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r38c1-nrps_pks" data-name="r38c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r38c1-MIBiG-cluster-compare" data-name="r38c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r38c1-clusterblast" data-name="r38c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r38c1-knownclusterblast" data-name="r38c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r38c1-subclusterblast" data-name="r38c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r38c1-tfbs-finder" data-name="r38c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r38c1-nrps-pks-bubbles" data-name="r38c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r38c1-pfam-details" data-name="r38c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r38c1-tigrfam-details" data-name="r38c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r38c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-152"></div> <span class="help-tooltip" id="gt-help-tooltip-help-152">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r38c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r38c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r38c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r38c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r38c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-142"></div> <span class="help-tooltip" id="nrps-domain-help-142">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r38c1-details-svg" data-filename="NODE_38_length_43995_cov_34.355731_r38c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r38c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r38c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-143"></div> <span class="help-tooltip" id="MIBiG-help-143">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r38c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r38c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r38c1"></div>
<div id="comparison-MIBiG-r38c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>





<td class="cc-heat-cell" title="0.76 (id:0.51, order:1.00, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>


<td>0.46</td>
<td>NRP</td>

<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>


<td>Pseudomonas sp. QS1027</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.73 (id:0.51, order:0.75, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>


<td>0.41</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>





<td class="cc-heat-cell" title="0.72 (id:0.44, order:1.00, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>taxlllaid A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050: 0-19730">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.44, order:1.00, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.38</td>
<td>NRP, Polyketide</td>

<td>thalassospiramide A</td>


<td>Tistrella bauzanensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.50, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>NRP</td>

<td>rhizomide A, rhizomide B, rhizomide C</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.50, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000396: 320-14288">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000396/index.html#r1c1" target="_new">BGC0000396</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.51, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>NRP</td>

<td>nodularin</td>


<td>Nostoc sp. 73.1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.44, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>sevadicin</td>


<td>Paenibacillus larvae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r38c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>


<td class="cc-heat-cell" title="0.76 (id:0.51, order:1.00, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
<td>0.76</td>
<td>NRP</td>
<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>
<td>Pseudomonas sp. QS1027</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.73 (id:0.51, order:0.75, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
<td>0.73</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>


<td class="cc-heat-cell" title="0.72 (id:0.44, order:1.00, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
<td>0.72</td>
<td>NRP</td>
<td>taxlllaid A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050: 0-19730">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.44, order:1.00, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP, Polyketide</td>
<td>thalassospiramide A</td>
<td>Tistrella bauzanensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.50, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>NRP</td>
<td>rhizomide A, rhizomide B, rhizomide C</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.50, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000396: 320-14288">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000396/index.html#r1c1" target="_new">BGC0000396</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.51, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>nodularin</td>
<td>Nostoc sp. 73.1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>sevadicin</td>
<td>Paenibacillus larvae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r38c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-144"></div> <span class="help-tooltip" id="cb-general-help-144">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r38c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r38c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r38c1_1.svg">
NZ_AP022671 (1045570-1134078): Nitrospira sp. KM1 (13% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r38c1_2.svg">
NC_008027 (3531163-3618247): Pseudomonas entomophila L48, complete genome (4% of genes show similarity), NRPS,NRPS-like,bacteriocin
</option>

<option value = "svg/clusterblast_r38c1_3.svg">
NZ_QKZD01000004 (171899-218241): Pseudomonas sp. 2848 Ga0255711 104, whole ge... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r38c1_4.svg">
NZ_QJRO01000011 (24336-70678): Pseudomonas soli strain LBp-160603 NODE 11 len... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r38c1_5.svg">
NZ_CP023299 (432052-476478): Pseudomonas mosselii strain BS011 chromosome, co... (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r38c1_6.svg">
NZ_QJRQ01000004 (5895-52237): Pseudomonas soli strain 57B-090714 NODE 4 lengt... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r38c1_7.svg">
NZ_CP014343 (2215414-2259842): Pseudomonas putida strain 1A00316, complete ge... (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r38c1_8.svg">
NZ_LSLE01000008 (27449-73788): Pseudomonas mosselii strain BW11M1 BW11M1 cont... (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r38c1_9.svg">
NZ_JHYW01000028 (28667-75006): Pseudomonas mosselii DSM 17497 Q380DRAFT scaff... (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r38c1_10.svg">
NZ_FOEQ01000005 (305039-351381): Pseudomonas soli strain LMG 27941, whole gen... (11% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r38c1-download">Download graphic</button>
</div>
<div id="clusterblast-r38c1-svg"></div>

</div>
</div>

<div class="body-details-section r38c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-145"></div> <span class="help-tooltip" id="cb-known-help-145">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r38c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r38c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r38c1_1.svg">
lokisin
</option>

<option value = "svg/knownclusterblast_r38c1_2.svg">
anikasin
</option>

<option value = "svg/knownclusterblast_r38c1_3.svg">
viscosinamide A/pseudodesmin A
</option>

<option value = "svg/knownclusterblast_r38c1_4.svg">
MA026
</option>

<option value = "svg/knownclusterblast_r38c1_5.svg">
putisolvin III/putisolvin VI/putisolvin V
</option>

<option value = "svg/knownclusterblast_r38c1_6.svg">
putisolvin
</option>

<option value = "svg/knownclusterblast_r38c1_7.svg">
bananamide 1/bananamide 2/bananamide 3
</option>

<option value = "svg/knownclusterblast_r38c1_8.svg">
bananamide F/bananamide D/bananamide G/bananamide E
</option>

<option value = "svg/knownclusterblast_r38c1_9.svg">
tolaasin I/tolaasin F
</option>

<option value = "svg/knownclusterblast_r38c1_10.svg">
syringomycin
</option>

</select>
<button id="knownclusterblast-r38c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r38c1-svg"></div>

</div>
</div>

<div class="body-details-section r38c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-146"></div> <span class="help-tooltip" id="cb-sub-help-146">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r38c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-150"></div> <span class="help-tooltip" id="tfbs-body-help-150">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (4):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.60 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r38c1-22028-ZuR"></div>

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 20.94 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r38c1-24687-NrtR"></div>

<div class="tfbs-hit-detail">
NrdR
<div class="collapser collapser-target-NrdR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Represses ribonucleotide reductase encoding genes<br>
<span>Score</span>: 21.36 out of 27.18
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r38c1-32053-NrdR"></div>

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 20.20 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r38c1-39718-NrtR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r38c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-151"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-151">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r38c1-domain-bubble-svg" data-filename="NODE_38_length_43995_cov_34.355731_r38c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r38c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (0 - 41734): single NRPS
</option>


</select>

<div class="details-svg" id="r38c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r38c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r38c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-153"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-153">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r38c1-pfam-details-svg" data-filename="NODE_38_length_43995_cov_34.355731_r38c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r38c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r38c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-154"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-154">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r38c1-tigrfam-details-svg" data-filename="NODE_38_length_43995_cov_34.355731_r38c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r38c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r38c1-nrps_pks_products" data-name="r38c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r38c1-nrps_pks_monomers" data-name="r38c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r38c1-tfbs-finder" data-name="r38c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r38c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-147"></div> <span class="help-tooltip" id="nrps-structures-help-147">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 0 - 41734:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r38c1-1" class="smiles-canvas" data-smiles="NC(CO)C(=O)NC(CO)C(=O)NC(CC(C)C)C(=O)NC(C(C)CC)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Ser - Ser - Leu - Ile - X) + (Thr)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,ser,leu,ile&amp;nrps2=thr" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,ser*,leu*,ile*,x&amp;nrps2=thr*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r38c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CO)C(=O)NC(CO)C(=O)NC(CC(C)C)C(=O)NC(C(C)CC)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r38c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-148"></div> <span class="help-tooltip" id="nrps-monomers-help-148">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">ctg38_1</span></strong>: Thr
<div class="collapser collapser-target-ctg38_1 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (554..958)</strong>: Thr
<div class="collapser collapser-target-ctg38_1 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Thr
<div class="collapser collapser-target-ctg38_1-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>thr (Thr)</dd>
<dt>Single AA prediction:</dt>
<dd>thr (Thr)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DFWNIGMVHK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Thr <span class="serif">DFWNIGMVHK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

<strong><span class="serif">ctg38_2</span></strong>: Ser - Ser - Leu - Ile - X
<div class="collapser collapser-target-ctg38_2 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,ser,leu,ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,ser*,leu*,ile*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (486..886)</strong>: Ser
<div class="collapser collapser-target-ctg38_2 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ser
<div class="collapser collapser-target-ctg38_2-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>ser (Ser)</dd>
<dt>Single AA prediction:</dt>
<dd>ser (Ser)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVWHMSLVDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ser <span class="serif">DVWHMSLVDK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1566..1968)</strong>: Ser
<div class="collapser collapser-target-ctg38_2 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ser
<div class="collapser collapser-target-ctg38_2-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>ser (Ser)</dd>
<dt>Single AA prediction:</dt>
<dd>ser (Ser)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVWHMSLVDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ser <span class="serif">DVWHMSLVDK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (2648..3035)</strong>: Leu
<div class="collapser collapser-target-ctg38_2 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Leu
<div class="collapser collapser-target-ctg38_2-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>leu (Leu)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAWFLGNVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Leu <span class="serif">DAWFLGNVVK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (3684..4079)</strong>: Ile
<div class="collapser collapser-target-ctg38_2 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ile
<div class="collapser collapser-target-ctg38_2-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>ile (Ile)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAMFLGCTYK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ile <span class="serif">DAMFLGCTYK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (4757..5157)</strong>: X
<div class="collapser collapser-target-ctg38_2 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-ctg38_2-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>asp,asn (Asp, Asn)</dd>
<dt>Single AA prediction:</dt>
<dd>asp (Asp)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DGWKVGVVDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Glu <span class="serif">DGWKFGVVDK</span> (94% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r38c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-149"></div> <span class="help-tooltip" id="tfbs-side-help-149">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrdR" target="_blank">NrdR</a></td>
<td>Represses ribonucleotide reductase encoding genes</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>




<div class="page" id='r39c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_39_length_43244_cov_33.315727 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-196"></div> <span class="help-tooltip" id="region_svg-help-196">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_39_length_43244_cov_33.315727.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r39c1-svg" data-filename="NODE_39_length_43244_cov_33.315727_r39c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 43,244 nt. (total: 43,244 nt)

<a class="cluster-rules-header" id="r39c1-rules-header" href="#r39c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r39c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r39c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-197"></div> <span class="help-tooltip" id="focus-panel-help-197">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r39c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r39c1-gene-table-details" data-name="r39c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r39c1-nrps_pks" data-name="r39c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r39c1-MIBiG-cluster-compare" data-name="r39c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r39c1-clusterblast" data-name="r39c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r39c1-knownclusterblast" data-name="r39c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r39c1-subclusterblast" data-name="r39c1-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r39c1-nrps-pks-bubbles" data-name="r39c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r39c1-pfam-details" data-name="r39c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r39c1-tigrfam-details" data-name="r39c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r39c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-163"></div> <span class="help-tooltip" id="gt-help-tooltip-help-163">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r39c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r39c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r39c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r39c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r39c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-155"></div> <span class="help-tooltip" id="nrps-domain-help-155">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r39c1-details-svg" data-filename="NODE_39_length_43244_cov_33.315727_r39c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r39c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r39c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-156"></div> <span class="help-tooltip" id="MIBiG-help-156">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r39c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r39c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r39c1"></div>
<div id="comparison-MIBiG-r39c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050: 0-19730">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.44, order:1.00, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.38</td>
<td>NRP, Polyketide</td>

<td>thalassospiramide A</td>


<td>Tistrella bauzanensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.45, order:1.00, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.38</td>
<td>NRP</td>

<td>taxlllaid A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.44, order:1.00, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>NRP</td>

<td>sevadicin</td>


<td>Paenibacillus larvae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.46, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>NRP</td>

<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>


<td>Pseudomonas sp. QS1027</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185: 0-11721">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>NRP</td>

<td>bacillibactin</td>


<td>Bacillus velezensis FZB42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001841: 6-14286">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001841/index.html#r1c1" target="_new">BGC0001841</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.49, order:0.66, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>NRP, Polyketide</td>

<td>detoxin P1, detoxin P2, detoxin P3</td>


<td>Amycolatopsis jejuensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401: 0-12882">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.47, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>paenibactin</td>


<td>Paenibacillus elgii B69</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Other</td>

<td>citrinin</td>


<td>Monascus aurantiacus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000438: 137-73725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000438/index.html#r1c1" target="_new">BGC0000438</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>syringopeptin 25A</td>


<td>Pseudomonas syringae pv. syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465: 0-10043">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>xenortide A, xenortide B, xenortide C, xenortide D</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r39c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050: 0-19730">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.44, order:1.00, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP, Polyketide</td>
<td>thalassospiramide A</td>
<td>Tistrella bauzanensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.45, order:1.00, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP</td>
<td>taxlllaid A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.44, order:1.00, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP</td>
<td>sevadicin</td>
<td>Paenibacillus larvae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.46, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>NRP</td>
<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>
<td>Pseudomonas sp. QS1027</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185: 0-11721">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>bacillibactin</td>
<td>Bacillus velezensis FZB42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001841: 6-14286">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001841/index.html#r1c1" target="_new">BGC0001841</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.49, order:0.66, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP, Polyketide</td>
<td>detoxin P1, detoxin P2, detoxin P3</td>
<td>Amycolatopsis jejuensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401: 0-12882">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.47, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>paenibactin</td>
<td>Paenibacillus elgii B69</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>citrinin</td>
<td>Monascus aurantiacus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000438: 137-73725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000438/index.html#r1c1" target="_new">BGC0000438</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>syringopeptin 25A</td>
<td>Pseudomonas syringae pv. syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465: 0-10043">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>xenortide A, xenortide B, xenortide C, xenortide D</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r39c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-157"></div> <span class="help-tooltip" id="cb-general-help-157">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r39c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r39c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r39c1_1.svg">
NZ_CP012831 (6938138-7005512): Pseudomonas fluorescens strain FW300-N2C3 chro... (15% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r39c1_2.svg">
NZ_VCNG01000003 (73704-193211): Pseudomonas sp. ICMP22404 contig00003, whole ... (15% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r39c1_3.svg">
NZ_LKEH01000043 (280413-343337): Pseudomonas viridiflava strain ICMP 2848 sca... (14% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r39c1_4.svg">
NZ_CABVHA010000011 (97272-160304): Pseudomonas fluorescens isolate PS634, who... (16% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r39c1_5.svg">
NZ_CABVIA010000015 (51901-114933): Pseudomonas fluorescens isolate PS689, who... (14% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r39c1_6.svg">
NZ_LT855380 (4004763-4067795): Pseudomonas viridiflava strain CFBP 1590 isola... (14% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r39c1_7.svg">
NZ_MBPF01000033 (278130-341153): Pseudomonas viridiflava strain CDRTc14 NODE ... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r39c1_8.svg">
NZ_FOYE01000007 (365302-427512): Pseudomonas sp. NFIX49, whole genome shotgun... (17% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r39c1_9.svg">
NZ_CP048408 (2878085-2936999): Pseudomonas fluorescens strain DR397 chromosom... (17% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r39c1_10.svg">
NZ_ARVC01000011 (688735-750948): Pseudomonas sp. LAIL14HWK12:I2 F638DRAFT sca... (17% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r39c1-download">Download graphic</button>
</div>
<div id="clusterblast-r39c1-svg"></div>

</div>
</div>

<div class="body-details-section r39c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-158"></div> <span class="help-tooltip" id="cb-known-help-158">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r39c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r39c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r39c1_1.svg">
chitinimide B/chitinimide F/chitinimide D/chitinimide A/chitinimide C/chitinimide E/chitinimide G
</option>

<option value = "svg/knownclusterblast_r39c1_2.svg">
detoxin N2/detoxin N3
</option>

<option value = "svg/knownclusterblast_r39c1_3.svg">
pseudovibriamide A1/pseudovibriamide B1/pseudovibriamide A2/pseudovibriamide A3/pseudovibriamide A4/pseudovibriamide B2/pseudovibriamide A5/pseudovibriamide A6/pseudovibriamide B3/pseudovibriamide B4/pseudovibriamide B6/pseudovibriamide B5
</option>

<option value = "svg/knownclusterblast_r39c1_4.svg">
rimosamide
</option>

<option value = "svg/knownclusterblast_r39c1_5.svg">
detoxin S1
</option>

<option value = "svg/knownclusterblast_r39c1_6.svg">
detoxin P1/detoxin P2/detoxin P3
</option>

<option value = "svg/knownclusterblast_r39c1_7.svg">
icosalide A/icosalide B
</option>

</select>
<button id="knownclusterblast-r39c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r39c1-svg"></div>

</div>
</div>

<div class="body-details-section r39c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-159"></div> <span class="help-tooltip" id="cb-sub-help-159">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r39c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-162"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-162">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r39c1-domain-bubble-svg" data-filename="NODE_39_length_43244_cov_33.315727_r39c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r39c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (0 - 43244): single NRPS
</option>


</select>

<div class="details-svg" id="r39c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r39c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r39c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-164"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-164">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r39c1-pfam-details-svg" data-filename="NODE_39_length_43244_cov_33.315727_r39c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r39c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r39c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-165"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-165">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r39c1-tigrfam-details-svg" data-filename="NODE_39_length_43244_cov_33.315727_r39c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r39c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r39c1-nrps_pks_products" data-name="r39c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r39c1-nrps_pks_monomers" data-name="r39c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>








</div>










<div class="sidepanel-details-section r39c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-160"></div> <span class="help-tooltip" id="nrps-structures-help-160">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 0 - 43244:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r39c1-1" class="smiles-canvas" data-smiles="NC(CCCNC(N)=N)C(=O)N1C(CCC1)C(=O)NC(CS)C(=O)NC([*])C(=O)NC(C(C)CC)C(=O)N1C(CCC1)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Arg - Pro - Cys) + (X) + (Ile - Pro)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=arg,pro,cys&amp;nrps3=ile,pro" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=arg*,pro*,cys*&amp;nrps2=x&amp;nrps3=ile*,pro*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r39c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CCCNC(N)=N)C(=O)N1C(CCC1)C(=O)NC(CS)C(=O)NC([*])C(=O)NC(C(C)CC)C(=O)N1C(CCC1)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r39c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-161"></div> <span class="help-tooltip" id="nrps-monomers-help-161">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">ctg39_12</span></strong>: Arg - Pro - Cys
<div class="collapser collapser-target-ctg39_12 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=arg,pro,cys" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=arg*,pro*,cys*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (36..447)</strong>: Arg
<div class="collapser collapser-target-ctg39_12 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Arg
<div class="collapser collapser-target-ctg39_12-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Long positively charged side chain (Orn, Lys, Arg)</dd>
<dt>Small clusters prediction:</dt>
<dd>arg (Arg)</dd>
<dt>Single AA prediction:</dt>
<dd>arg (Arg)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVADVGAIDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Arg <span class="serif">DVADVGAIDK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1104..1510)</strong>: Pro
<div class="collapser collapser-target-ctg39_12 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Pro
<div class="collapser collapser-target-ctg39_12-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Cyclic aliphatic chain (polar NH2 group) (Pro, Hpr)</dd>
<dt>Small clusters prediction:</dt>
<dd>pro (Pro)</dd>
<dt>Single AA prediction:</dt>
<dd>pro (Pro)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVQFVAQVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Pro <span class="serif">DVQFVAQVVK</span> (91% 8 match)</dd>
<dd>Hpr <span class="serif">DVQFVAQVVK</span> (74% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (2164..2562)</strong>: Cys
<div class="collapser collapser-target-ctg39_12 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Cys
<div class="collapser collapser-target-ctg39_12-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Polar, uncharged (aliphatic with -SH) (Cys)</dd>
<dt>Small clusters prediction:</dt>
<dd>Polar, uncharged (aliphatic with -SH) (Cys)</dd>
<dt>Single AA prediction:</dt>
<dd>Polar, uncharged (aliphatic with -SH) (Cys)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLYNMSLIWK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Cys <span class="serif">DLYNMSLIWK</span> (97% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">ctg39_13</span></strong>: X
<div class="collapser collapser-target-ctg39_13 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>AMP-binding (565..972)</strong>: X
<div class="collapser collapser-target-ctg39_13 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-ctg39_13-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAWVFGVVIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Gln <span class="serif">DAWQFGVVDK</span> (76% 8 match)</dd>
<dd>aIle <span class="serif">DALVFGVVLK</span> (56% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

<strong><span class="serif">ctg39_14</span></strong>: Ile - Pro
<div class="collapser collapser-target-ctg39_14 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile,pro" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile*,pro*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (17..415)</strong>: Ile
<div class="collapser collapser-target-ctg39_14 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ile
<div class="collapser collapser-target-ctg39_14-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>val (Val)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAFFLGVTFK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ile <span class="serif">DAFFLGVTFK</span> (65% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1059..1467)</strong>: Pro
<div class="collapser collapser-target-ctg39_14 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Pro
<div class="collapser collapser-target-ctg39_14-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Cyclic aliphatic chain (polar NH2 group) (Pro, Hpr)</dd>
<dt>Small clusters prediction:</dt>
<dd>pro (Pro)</dd>
<dt>Single AA prediction:</dt>
<dd>pro (Pro)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVQFVAHVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Pro <span class="serif">DVQFVAHVVK</span> (91% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>












</div>
<br>

</dl>
</div>
</div>








</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>