
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>CP8C_scaffolds - 13 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="CP8C_scaffolds - 10 record(s) - 13 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-ecc99105-1e6a-46e5-9ce6-3da55b7b07cd/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.0.1
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="CP8C_scaffolds.zip">Download all results</a></li>

<li><a href="CP8C_scaffolds.gbk">Download GenBank summary file</a></li>

<li><a href="bacteria-ecc99105-1e6a-46e5-9ce6-3da55b7b07cd.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton hybrid r1c1">
<a href="#r1c1">1.1</a>
</div>

<div class="regbutton other betalactone r1c2">
<a href="#r1c2">1.2</a>
</div>

<div class="regbutton NRPS r1c3">
<a href="#r1c3">1.3</a>
</div>



<div class="regbutton RiPP RiPP-like r2c1">
<a href="#r2c1">2.1</a>
</div>



<div class="regbutton NRPS NRPS r4c1">
<a href="#r4c1">4.1</a>
</div>

<div class="regbutton other NI-siderophore r4c2">
<a href="#r4c2">4.2</a>
</div>



<div class="regbutton other arylpolyene r6c1">
<a href="#r6c1">6.1</a>
</div>



<div class="regbutton RiPP redox-cofactor r13c1">
<a href="#r13c1">13.1</a>
</div>



<div class="regbutton RiPP RiPP-like r15c1">
<a href="#r15c1">15.1</a>
</div>



<div class="regbutton other NAGGN r19c1">
<a href="#r19c1">19.1</a>
</div>



<div class="regbutton NRPS NRPS r30c1">
<a href="#r30c1">30.1</a>
</div>



<div class="regbutton RiPP RiPP-like r39c1">
<a href="#r39c1">39.1</a>
</div>



<div class="regbutton NRPS NRPS-like r52c1">
<a href="#r52c1">52.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NODE_1_length_641826_cov_19.882008</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-1" data-filename="NODE_1_length_641826_cov_19.882008_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-1" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r1c1">
<td class="regbutton hybrid r1c1">
<a href="#r1c1">Region&nbsp1.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lap" target="_blank">LAP</a>
</td>
<td class="digits">86,564</td>
<td class="digits table-split-left">130,527</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002379/1" target="_blank">MA026</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 14%, #ffffff00 14%)">14%</td>

</tr>
<tr class="linked-row even" data-anchor="#r1c2">
<td class="regbutton other betalactone r1c2">
<a href="#r1c2">Region&nbsp1.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#betalactone" target="_blank">betalactone</a>
</td>
<td class="digits">239,920</td>
<td class="digits table-split-left">268,400</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001095/1" target="_blank">fengycin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c3">
<td class="regbutton NRPS r1c3">
<a href="#r1c3">Region&nbsp1.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">530,421</td>
<td class="digits table-split-left">607,164</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002693/1" target="_blank">pyoverdine SMX-1</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 38%, #ffffff00 38%)">38%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_2_length_547678_cov_19.474303</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-2" data-filename="NODE_2_length_547678_cov_19.474303_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-2" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton RiPP RiPP-like r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
</td>
<td class="digits">46,666</td>
<td class="digits table-split-left">57,511</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_4_length_376817_cov_19.716587</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-4" data-filename="NODE_4_length_376817_cov_19.716587_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-4" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r4c1">
<td class="regbutton NRPS NRPS r4c1">
<a href="#r4c1">Region&nbsp4.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">33,478</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002433/1" target="_blank">azotobactin D</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>

</tr>
<tr class="linked-row even" data-anchor="#r4c2">
<td class="regbutton other NI-siderophore r4c2">
<a href="#r4c2">Region&nbsp4.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ni-siderophore" target="_blank">NI-siderophore</a>
</td>
<td class="digits">122,693</td>
<td class="digits table-split-left">134,618</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_6_length_246143_cov_19.272524</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-6" data-filename="NODE_6_length_246143_cov_19.272524_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-6" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r6c1">
<td class="regbutton other arylpolyene r6c1">
<a href="#r6c1">Region&nbsp6.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#arylpolyene" target="_blank">arylpolyene</a>
</td>
<td class="digits">77,116</td>
<td class="digits table-split-left">120,691</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000837/1" target="_blank">APE Vf</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_13_length_148291_cov_19.867833</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-13" data-filename="NODE_13_length_148291_cov_19.867833_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-13" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r13c1">
<td class="regbutton RiPP redox-cofactor r13c1">
<a href="#r13c1">Region&nbsp13.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#redox-cofactor" target="_blank">redox-cofactor</a>
</td>
<td class="digits">32,410</td>
<td class="digits table-split-left">54,557</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001100/1" target="_blank">lankacidin C</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_15_length_143283_cov_19.608976</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-15" data-filename="NODE_15_length_143283_cov_19.608976_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-15" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r15c1">
<td class="regbutton RiPP RiPP-like r15c1">
<a href="#r15c1">Region&nbsp15.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
</td>
<td class="digits">34,073</td>
<td class="digits table-split-left">44,954</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_19_length_106372_cov_20.016360</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-19" data-filename="NODE_19_length_106372_cov_20.016360_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-19" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r19c1">
<td class="regbutton other NAGGN r19c1">
<a href="#r19c1">Region&nbsp19.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#naggn" target="_blank">NAGGN</a>
</td>
<td class="digits">32,875</td>
<td class="digits table-split-left">47,755</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_30_length_62891_cov_19.830261</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-30" data-filename="NODE_30_length_62891_cov_19.830261_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-30" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r30c1">
<td class="regbutton NRPS NRPS r30c1">
<a href="#r30c1">Region&nbsp30.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">11,142</td>
<td class="digits table-split-left">62,891</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001312/1" target="_blank">viscosin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 68%, #ffffff00 68%)">68%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_39_length_36248_cov_19.215200</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-39" data-filename="NODE_39_length_36248_cov_19.215200_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-39" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r39c1">
<td class="regbutton RiPP RiPP-like r39c1">
<a href="#r39c1">Region&nbsp39.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">7,497</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_52_length_18213_cov_21.895732</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-52" data-filename="NODE_52_length_18213_cov_21.895732_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-52" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r52c1">
<td class="regbutton NRPS NRPS-like r52c1">
<a href="#r52c1">Region&nbsp52.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">18,213</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r1c1">
<td class="regbutton hybrid r1c1">
<a href="#r1c1">Region&nbsp1.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lap" target="_blank">LAP</a>
</td>
<td class="digits">86,564</td>
<td class="digits table-split-left">130,527</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002379/1" target="_blank">MA026</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 14%, #ffffff00 14%)">14%</td>

</tr>
<tr class="linked-row even" data-anchor="#r1c2">
<td class="regbutton other betalactone r1c2">
<a href="#r1c2">Region&nbsp1.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#betalactone" target="_blank">betalactone</a>
</td>
<td class="digits">239,920</td>
<td class="digits table-split-left">268,400</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001095/1" target="_blank">fengycin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c3">
<td class="regbutton NRPS r1c3">
<a href="#r1c3">Region&nbsp1.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">530,421</td>
<td class="digits table-split-left">607,164</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002693/1" target="_blank">pyoverdine SMX-1</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 38%, #ffffff00 38%)">38%</td>

</tr>


<tr class="linked-row even" data-anchor="#r2c1">
<td class="regbutton RiPP RiPP-like r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
</td>
<td class="digits">46,666</td>
<td class="digits table-split-left">57,511</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r4c1">
<td class="regbutton NRPS NRPS r4c1">
<a href="#r4c1">Region&nbsp4.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">33,478</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002433/1" target="_blank">azotobactin D</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>

</tr>
<tr class="linked-row even" data-anchor="#r4c2">
<td class="regbutton other NI-siderophore r4c2">
<a href="#r4c2">Region&nbsp4.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ni-siderophore" target="_blank">NI-siderophore</a>
</td>
<td class="digits">122,693</td>
<td class="digits table-split-left">134,618</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r6c1">
<td class="regbutton other arylpolyene r6c1">
<a href="#r6c1">Region&nbsp6.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#arylpolyene" target="_blank">arylpolyene</a>
</td>
<td class="digits">77,116</td>
<td class="digits table-split-left">120,691</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000837/1" target="_blank">APE Vf</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>


<tr class="linked-row even" data-anchor="#r13c1">
<td class="regbutton RiPP redox-cofactor r13c1">
<a href="#r13c1">Region&nbsp13.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#redox-cofactor" target="_blank">redox-cofactor</a>
</td>
<td class="digits">32,410</td>
<td class="digits table-split-left">54,557</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001100/1" target="_blank">lankacidin C</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r15c1">
<td class="regbutton RiPP RiPP-like r15c1">
<a href="#r15c1">Region&nbsp15.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
</td>
<td class="digits">34,073</td>
<td class="digits table-split-left">44,954</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r19c1">
<td class="regbutton other NAGGN r19c1">
<a href="#r19c1">Region&nbsp19.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#naggn" target="_blank">NAGGN</a>
</td>
<td class="digits">32,875</td>
<td class="digits table-split-left">47,755</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r30c1">
<td class="regbutton NRPS NRPS r30c1">
<a href="#r30c1">Region&nbsp30.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">11,142</td>
<td class="digits table-split-left">62,891</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001312/1" target="_blank">viscosin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 68%, #ffffff00 68%)">68%</td>

</tr>


<tr class="linked-row even" data-anchor="#r39c1">
<td class="regbutton RiPP RiPP-like r39c1">
<a href="#r39c1">Region&nbsp39.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">7,497</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r52c1">
<td class="regbutton NRPS NRPS-like r52c1">
<a href="#r52c1">Region&nbsp52.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">18,213</td>

<td colspan="3"></td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NODE_3_length_522668_cov_20.515797</strong></dd>

<dd><strong>NODE_5_length_314290_cov_19.989160</strong></dd>

<dd><strong>NODE_7_length_204241_cov_19.529437</strong></dd>

<dd><strong>NODE_8_length_195925_cov_19.576299</strong></dd>

<dd><strong>NODE_9_length_195540_cov_19.564634</strong></dd>

<dd><strong>NODE_10_length_167872_cov_19.582342</strong></dd>

<dd><strong>NODE_11_length_163602_cov_19.421520</strong></dd>

<dd><strong>NODE_12_length_152387_cov_19.334193</strong></dd>

<dd><strong>NODE_14_length_146324_cov_19.422039</strong></dd>

<dd><strong>NODE_16_length_122784_cov_19.831835</strong></dd>

<dd><strong>NODE_17_length_118428_cov_19.506747</strong></dd>

<dd><strong>NODE_18_length_106958_cov_19.577306</strong></dd>

<dd><strong>NODE_20_length_105378_cov_19.471116</strong></dd>

<dd><strong>NODE_21_length_97455_cov_20.684313</strong></dd>

<dd><strong>NODE_22_length_95359_cov_19.062782</strong></dd>

<dd><strong>NODE_23_length_89019_cov_20.043680</strong></dd>

<dd><strong>NODE_24_length_84006_cov_19.800236</strong></dd>

<dd><strong>NODE_25_length_78895_cov_19.688333</strong></dd>

<dd><strong>NODE_26_length_76436_cov_19.085805</strong></dd>

<dd><strong>NODE_27_length_75940_cov_20.024281</strong></dd>

<dd><strong>NODE_28_length_74660_cov_18.869903</strong></dd>

<dd><strong>NODE_29_length_66081_cov_18.642749</strong></dd>

<dd><strong>NODE_31_length_62506_cov_18.014368</strong></dd>

<dd><strong>NODE_32_length_62504_cov_20.534544</strong></dd>

<dd><strong>NODE_33_length_57679_cov_19.632270</strong></dd>

<dd><strong>NODE_34_length_51800_cov_19.813816</strong></dd>

<dd><strong>NODE_35_length_47636_cov_19.361257</strong></dd>

<dd><strong>NODE_36_length_41560_cov_35.424198</strong></dd>

<dd><strong>NODE_37_length_38156_cov_19.121957</strong></dd>

<dd><strong>NODE_38_length_36861_cov_19.300049</strong></dd>

<dd><strong>NODE_40_length_33185_cov_19.219886</strong></dd>

<dd><strong>NODE_41_length_32569_cov_19.689277</strong></dd>

<dd><strong>NODE_42_length_31094_cov_19.393429</strong></dd>

<dd><strong>NODE_43_length_27943_cov_18.571700</strong></dd>

<dd><strong>NODE_44_length_26589_cov_19.299789</strong></dd>

<dd><strong>NODE_45_length_26006_cov_20.203363</strong></dd>

<dd><strong>NODE_46_length_23416_cov_18.968465</strong></dd>

<dd><strong>NODE_47_length_22243_cov_20.453623</strong></dd>

<dd><strong>NODE_48_length_21807_cov_19.917948</strong></dd>

<dd><strong>NODE_49_length_21156_cov_19.140993</strong></dd>

<dd><strong>NODE_50_length_19418_cov_19.676956</strong></dd>

<dd><strong>NODE_51_length_19139_cov_18.545641</strong></dd>

<dd><strong>NODE_53_length_10639_cov_19.149972</strong></dd>

<dd><strong>NODE_54_length_8251_cov_17.188402</strong></dd>

<dd><strong>NODE_55_length_6257_cov_19.543689</strong></dd>

<dd><strong>NODE_56_length_5210_cov_133.637444</strong></dd>

<dd><strong>NODE_57_length_3857_cov_51.151852</strong></dd>

<dd><strong>NODE_58_length_2031_cov_16.865404</strong></dd>

<dd><strong>NODE_59_length_1671_cov_35.026976</strong></dd>

<dd><strong>NODE_60_length_1290_cov_27.713108</strong></dd>

<dd><strong>NODE_61_length_1251_cov_110.251278</strong></dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>







<div class="page" id='r1c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_1_length_641826_cov_19.882008 - Region 1 - LAP,NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-136"></div> <span class="help-tooltip" id="region_svg-help-136">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_1_length_641826_cov_19.882008.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c1-svg" data-filename="NODE_1_length_641826_cov_19.882008_r1c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 86,564 - 130,527 nt. (total: 43,964 nt)

<a class="cluster-rules-header" id="r1c1-rules-header" href="#r1c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>LAP: ((((goadsporin_like or PF00881 or TIGR03605) and (YcaO or TIGR03604)) or TIGR03882 or TIGR03603 or TIGR03602) and not cyanobactin_synth)</div>

<div class="region-svg-container">
<div id='r1c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-137"></div> <span class="help-tooltip" id="focus-panel-help-137">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c1-gene-table-details" data-name="r1c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c1-nrps_pks" data-name="r1c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c1-MIBiG-cluster-compare" data-name="r1c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c1-clusterblast" data-name="r1c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c1-knownclusterblast" data-name="r1c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c1-subclusterblast" data-name="r1c1-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r1c1-tfbs-finder" data-name="r1c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r1c1-nrps-pks-bubbles" data-name="r1c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c1-pfam-details" data-name="r1c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c1-tigrfam-details" data-name="r1c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-11"></div> <span class="help-tooltip" id="gt-help-tooltip-help-11">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-1"></div> <span class="help-tooltip" id="nrps-domain-help-1">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-details-svg" data-filename="NODE_1_length_641826_cov_19.882008_r1c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-2"></div> <span class="help-tooltip" id="MIBiG-help-2">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c1"></div>
<div id="comparison-MIBiG-r1c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">LAP</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.77 (id:0.51, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>






<td class="cc-heat-cell" title="0.77 (id:0.51, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>


<td>0.95</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002144: 0-5515">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002144/index.html#r1c1" target="_new">BGC0002144</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.64</td>
<td>RiPP</td>

<td>phazolicin</td>


<td>Rhizobium sp. Pop5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000568: 0-5145">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000568/index.html#r1c1" target="_new">BGC0000568</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.44, order:0.62, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.44, order:0.62, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.63</td>
<td>RiPP</td>

<td>microcin B17</td>


<td>Escherichia coli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.63</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.59</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.59</td>
<td>NRP</td>

<td>vicibactin</td>


<td>Rhizobium etli CFN 42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.58</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.57</td>
<td>NRP</td>

<td>thermoactinoamide A</td>


<td>Thermoactinomyces sp. AS95</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.56</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002226: 138-12428">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002226/index.html#r1c1" target="_new">BGC0002226</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.55</td>
<td>NRP</td>

<td>acu-dioxomorpholine B, acu-dioxomorpholine A</td>


<td>Aspergillus aculeatus ATCC 16872</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.77 (id:0.51, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
<td>0.77</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002474: 103-7254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002474/index.html#r1c1" target="_new">BGC0002474</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>agrobactin</td>
<td>Agrobacterium tumefaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002493: 1205-6076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002493/index.html#r1c1" target="_new">BGC0002493</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>parabactin</td>
<td>Paracoccus denitrificans PD1222</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>vicibactin</td>
<td>Rhizobium etli CFN 42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002144: 0-5515">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002144/index.html#r1c1" target="_new">BGC0002144</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>RiPP</td>
<td>phazolicin</td>
<td>Rhizobium sp. Pop5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000568: 0-5145">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000568/index.html#r1c1" target="_new">BGC0000568</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.44, order:0.62, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>RiPP</td>
<td>microcin B17</td>
<td>Escherichia coli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>thermoactinoamide A</td>
<td>Thermoactinomyces sp. AS95</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-3"></div> <span class="help-tooltip" id="cb-general-help-3">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c1_1.svg">
NZ_CP015600 (4078713-4122677): Pseudomonas antarctica strain PAMC 27494 chrom... (100% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c1_2.svg">
NZ_LT629702 (2387155-2431779): Pseudomonas azotoformans strain LMG 21611 chro... (97% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c1_3.svg">
NZ_CP027755 (4041030-4085866): Pseudomonas synxantha strain 2-79 chromosome, ... (87% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c1_4.svg">
NZ_CP014546 (5820070-5863843): Pseudomonas azotoformans strain S4 chromosome,... (84% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c1_5.svg">
NZ_JACAQO010000012 (32891-79101): Pseudomonas reactans strain IPO3750 IPO 375... (83% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c1_6.svg">
NZ_JACAQM010000030 (30927-77137): Pseudomonas reactans strain IPO3748 IPO 374... (83% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c1_7.svg">
NZ_AYTD01000013 (501962-546190): Pseudomonas canadensis strain Feb-92 scaffol... (86% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c1_8.svg">
NZ_RXYG01000015 (56986-103196): Pseudomonas sp. C 49-2 NODE 15 length 127589 ... (83% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c1_9.svg">
NZ_MOAZ01000010 (57497-103722): Pseudomonas canadensis strain 36C8 36C8 conti... (83% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c1_10.svg">
NZ_FOQS01000005 (23302-69512): Pseudomonas sp. NFPP04, whole genome shotgun s... (83% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r1c1-download">Download graphic</button>
</div>
<div id="clusterblast-r1c1-svg"></div>

</div>
</div>

<div class="body-details-section r1c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-4"></div> <span class="help-tooltip" id="cb-known-help-4">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r1c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r1c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r1c1_1.svg">
MA026
</option>

<option value = "svg/knownclusterblast_r1c1_2.svg">
viscosinamide A/pseudodesmin A
</option>

<option value = "svg/knownclusterblast_r1c1_3.svg">
bananamide 1/bananamide 2/bananamide 3
</option>

<option value = "svg/knownclusterblast_r1c1_4.svg">
bananamide F/bananamide D/bananamide G/bananamide E
</option>

<option value = "svg/knownclusterblast_r1c1_5.svg">
lokisin
</option>

<option value = "svg/knownclusterblast_r1c1_6.svg">
anikasin
</option>

<option value = "svg/knownclusterblast_r1c1_7.svg">
putisolvin
</option>

<option value = "svg/knownclusterblast_r1c1_8.svg">
putisolvin III/putisolvin VI/putisolvin V
</option>

<option value = "svg/knownclusterblast_r1c1_9.svg">
tolaasin I/tolaasin F
</option>

<option value = "svg/knownclusterblast_r1c1_10.svg">
sessilin A
</option>

</select>
<button id="knownclusterblast-r1c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r1c1-svg"></div>

</div>
</div>

<div class="body-details-section r1c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-5"></div> <span class="help-tooltip" id="cb-sub-help-5">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r1c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-9"></div> <span class="help-tooltip" id="tfbs-body-help-9">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NrdR
<div class="collapser collapser-target-NrdR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Represses ribonucleotide reductase encoding genes<br>
<span>Score</span>: 22.14 out of 27.18
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r1c1-112163-NrdR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 22.10 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c1-129532-NuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r1c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-10"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-10">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-domain-bubble-svg" data-filename="NODE_1_length_641826_cov_19.882008_r1c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (86563 - 130527): neighbouring NRPS-LAP
</option>


</select>

<div class="details-svg" id="r1c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-12"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-12">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-pfam-details-svg" data-filename="NODE_1_length_641826_cov_19.882008_r1c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-13"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-13">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-tigrfam-details-svg" data-filename="NODE_1_length_641826_cov_19.882008_r1c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c1-nrps_pks_products" data-name="r1c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c1-nrps_pks_monomers" data-name="r1c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>





<div class="sidepanel-details-header r1c1-tfbs-finder" data-name="r1c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r1c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-6"></div> <span class="help-tooltip" id="nrps-structures-help-6">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 86563 - 130527:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c1-1" class="smiles-canvas" data-smiles="NC(CC(C)C)C(=O)NC(CCC(=O)O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Leu - Glu)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu,glu" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*,glu*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CC(C)C)C(=O)NC(CCC(=O)O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-7"></div> <span class="help-tooltip" id="nrps-monomers-help-7">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">ctg1_107</span></strong>: Leu - Glu
<div class="collapser collapser-target-ctg1_107 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu,glu" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*,glu*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (466..849)</strong>: Leu
<div class="collapser collapser-target-ctg1_107 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Leu
<div class="collapser collapser-target-ctg1_107-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>leu (Leu)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAWFLGNVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Leu <span class="serif">DAWFLGNVVK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1498..1898)</strong>: Glu
<div class="collapser collapser-target-ctg1_107 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Glu
<div class="collapser collapser-target-ctg1_107-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DGWQLGVVDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Glu <span class="serif">DGWKLGVVDK</span> (91% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>





<div class="sidepanel-details-section r1c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-8"></div> <span class="help-tooltip" id="tfbs-side-help-8">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrdR" target="_blank">NrdR</a></td>
<td>Represses ribonucleotide reductase encoding genes</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r1c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_1_length_641826_cov_19.882008 - Region 2 - betalactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-138"></div> <span class="help-tooltip" id="region_svg-help-138">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_1_length_641826_cov_19.882008.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c2-svg" data-filename="NODE_1_length_641826_cov_19.882008_r1c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 239,920 - 268,400 nt. (total: 28,481 nt)

<a class="cluster-rules-header" id="r1c2-rules-header" href="#r1c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c2-rules">betalactone: (HMGL-like and AMP-binding)</div>

<div class="region-svg-container">
<div id='r1c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-139"></div> <span class="help-tooltip" id="focus-panel-help-139">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c2-gene-table-details" data-name="r1c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c2-MIBiG-cluster-compare" data-name="r1c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c2-clusterblast" data-name="r1c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c2-knownclusterblast" data-name="r1c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c2-subclusterblast" data-name="r1c2-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r1c2-tfbs-finder" data-name="r1c2-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r1c2-pfam-details" data-name="r1c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c2-tigrfam-details" data-name="r1c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-20"></div> <span class="help-tooltip" id="gt-help-tooltip-help-20">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-14"></div> <span class="help-tooltip" id="MIBiG-help-14">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c2"></div>
<div id="comparison-MIBiG-r1c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">betalactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000131: 0-6280">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000131/index.html#r1c1" target="_new">BGC0000131</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>pyrrolomycin</td>


<td>Streptomyces sp. UC 11065</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002284: 0-17900">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002284/index.html#r1c1" target="_new">BGC0002284</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.52, order:0.36, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>corynecin III, corynecin I, corynecin II</td>


<td>Rhodococcus sp. H-CA8f</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>RiPP</td>

<td>SCO-2138</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234: 152-6978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.47, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>jadomycin</td>


<td>Streptomyces venezuelae ATCC 10712</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000891: 0-11263">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000891/index.html#r1c1" target="_new">BGC0000891</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Other (Aminocoumarin)</td>

<td>pentabromopseudilin</td>


<td>Pseudoalteromonas luteoviolacea 2ta16</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001465: 0-12220">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001465/index.html#r1c1" target="_new">BGC0001465</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Other</td>

<td>bromophene, bistribromopyrrole, pentabromopseudilin</td>


<td>Marinomonas mediterranea MMB-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000890: 0-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000890/index.html#r1c1" target="_new">BGC0000890</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Other (Aminocoumarin)</td>

<td>pentabromopseudilin</td>


<td>Pseudoalteromonas phenolica O-BC30</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000312: 0-41205">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000312/index.html#r1c1" target="_new">BGC0000312</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>bassianolide</td>


<td>Beauveria bassiana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000922: 0-13136">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000922/index.html#r1c1" target="_new">BGC0000922</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Other</td>

<td>pseudomonas quinolone signal (PQS)</td>


<td>Pseudomonas aeruginosa PAO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001458: 100-11195">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001458/index.html#r1c1" target="_new">BGC0001458</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Other (Aminocoumarin)</td>

<td>salinipostin G</td>


<td>Salinispora tropica CNB-440</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000131: 0-6280">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000131/index.html#r1c1" target="_new">BGC0000131</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>pyrrolomycin</td>
<td>Streptomyces sp. UC 11065</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002284: 0-17900">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002284/index.html#r1c1" target="_new">BGC0002284</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.52, order:0.36, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>corynecin III, corynecin I, corynecin II</td>
<td>Rhodococcus sp. H-CA8f</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>RiPP</td>
<td>SCO-2138</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234: 152-6978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.47, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>jadomycin</td>
<td>Streptomyces venezuelae ATCC 10712</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000891: 0-11263">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000891/index.html#r1c1" target="_new">BGC0000891</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other (Aminocoumarin)</td>
<td>pentabromopseudilin</td>
<td>Pseudoalteromonas luteoviolacea 2ta16</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001465: 0-12220">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001465/index.html#r1c1" target="_new">BGC0001465</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>bromophene, bistribromopyrrole, pentabromopseudilin</td>
<td>Marinomonas mediterranea MMB-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000890: 0-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000890/index.html#r1c1" target="_new">BGC0000890</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Other (Aminocoumarin)</td>
<td>pentabromopseudilin</td>
<td>Pseudoalteromonas phenolica O-BC30</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000312: 0-41205">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000312/index.html#r1c1" target="_new">BGC0000312</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>bassianolide</td>
<td>Beauveria bassiana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000922: 0-13136">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000922/index.html#r1c1" target="_new">BGC0000922</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Other</td>
<td>pseudomonas quinolone signal (PQS)</td>
<td>Pseudomonas aeruginosa PAO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001458: 100-11195">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001458/index.html#r1c1" target="_new">BGC0001458</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Other (Aminocoumarin)</td>
<td>salinipostin G</td>
<td>Salinispora tropica CNB-440</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-15"></div> <span class="help-tooltip" id="cb-general-help-15">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c2_1.svg">
NZ_CP015600 (3940024-3968496): Pseudomonas antarctica strain PAMC 27494 chrom... (100% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r1c2_2.svg">
NZ_CP027755 (3898890-3927365): Pseudomonas synxantha strain 2-79 chromosome, ... (100% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r1c2_3.svg">
NZ_JACHCZ010000001 (1178886-1207223): Pseudomonas sp. JAI120 Ga0401626 01, wh... (100% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r1c2_4.svg">
NZ_CP019856 (4157722-4186036): Pseudomonas azotoformans strain F77 chromosome... (100% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r1c2_5.svg">
NZ_FYDK01000002 (151827-180170): Pseudomonas sp. Irchel 3H9 isolate Pseudomon... (100% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r1c2_6.svg">
NZ_CP031450 (4240699-4269025): Pseudomonas fluorescens strain SIK W1 chromoso... (100% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r1c2_7.svg">
NZ_VIUN01000010 (158158-186495): Pseudomonas sp. SJZ103 Ga0314254 110, whole ... (100% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r1c2_8.svg">
NZ_JACAOY010000027 (77016-105337): Pseudomonas sp. B6002 run0228 B6002 S26 L0... (100% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r1c2_9.svg">
NZ_LT629801 (1635911-1664260): Pseudomonas rhodesiae strain LMG 17764 chromos... (100% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r1c2_10.svg">
NZ_CP054205 (3774874-3803223): Pseudomonas rhodesiae strain NL2019 chromosome... (100% of genes show similarity), betalactone
</option>

</select>
<button id= "clusterblast-r1c2-download">Download graphic</button>
</div>
<div id="clusterblast-r1c2-svg"></div>

</div>
</div>

<div class="body-details-section r1c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-16"></div> <span class="help-tooltip" id="cb-known-help-16">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r1c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r1c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r1c2_1.svg">
fengycin
</option>

<option value = "svg/knownclusterblast_r1c2_2.svg">
plipastatin
</option>

<option value = "svg/knownclusterblast_r1c2_3.svg">
mycosubtilin
</option>

<option value = "svg/knownclusterblast_r1c2_4.svg">
corynecin III/corynecin I/corynecin II
</option>

</select>
<button id="knownclusterblast-r1c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r1c2-svg"></div>

</div>
</div>

<div class="body-details-section r1c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-17"></div> <span class="help-tooltip" id="cb-sub-help-17">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r1c2-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-19"></div> <span class="help-tooltip" id="tfbs-body-help-19">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.93 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c2-243050-ZuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 19.89 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c2-263623-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r1c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-21"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-21">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-pfam-details-svg" data-filename="NODE_1_length_641826_cov_19.882008_r1c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-22"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-22">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-tigrfam-details-svg" data-filename="NODE_1_length_641826_cov_19.882008_r1c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r1c2-tfbs-finder" data-name="r1c2-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r1c2-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-18"></div> <span class="help-tooltip" id="tfbs-side-help-18">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r1c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_1_length_641826_cov_19.882008 - Region 3 - NRP-metallophore,NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-140"></div> <span class="help-tooltip" id="region_svg-help-140">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_1_length_641826_cov_19.882008.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c3-svg" data-filename="NODE_1_length_641826_cov_19.882008_r1c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 530,421 - 607,164 nt. (total: 76,744 nt)

<a class="cluster-rules-header" id="r1c3-rules-header" href="#r1c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c3-rules">NRP-metallophore: (VibH_like or Cy_tandem or (cds(Condensation and AMP-binding) and ((IBH_Asp and not SBH_Asp) or IBH_His or TBH_Asp or CyanoBH_Asp1 or CyanoBH_Asp2 or IPL or SalSyn or (EntA and EntC) or (GrbD and GrbE) or (FbnL and FbnM) or PvdO or PvdP or (Orn_monoox and not (KtzT or MetRS-like)) or Lys_monoox or VbsL)))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r1c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-141"></div> <span class="help-tooltip" id="focus-panel-help-141">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c3-gene-table-details" data-name="r1c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c3-nrps_pks" data-name="r1c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c3-MIBiG-cluster-compare" data-name="r1c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c3-clusterblast" data-name="r1c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c3-knownclusterblast" data-name="r1c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c3-subclusterblast" data-name="r1c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c3-tfbs-finder" data-name="r1c3-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r1c3-nrps-pks-bubbles" data-name="r1c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c3-pfam-details" data-name="r1c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c3-tigrfam-details" data-name="r1c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-33"></div> <span class="help-tooltip" id="gt-help-tooltip-help-33">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-23"></div> <span class="help-tooltip" id="nrps-domain-help-23">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c3-details-svg" data-filename="NODE_1_length_641826_cov_19.882008_r1c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-24"></div> <span class="help-tooltip" id="MIBiG-help-24">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c3"></div>
<div id="comparison-MIBiG-r1c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRP-metallophore</span></th>

<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>





<td class="cc-heat-cell" title="0.75 (id:0.44, order:1.00, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>






<td class="cc-heat-cell" title="0.75 (id:0.44, order:1.00, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>


<td>0.88</td>
<td>NRP</td>

<td>taxlllaid A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>





<td class="cc-heat-cell" title="0.72 (id:0.44, order:1.00, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>






<td class="cc-heat-cell" title="0.72 (id:0.44, order:1.00, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>


<td>0.79</td>
<td>NRP</td>

<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>


<td>Pseudomonas sp. QS1027</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.69</td>
<td>NRP</td>

<td>thermoactinoamide A</td>


<td>Thermoactinomyces sp. AS95</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.68</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.66</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417: 0-12076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.65</td>
<td>NRP</td>

<td>rhodochelin</td>


<td>Rhodococcus jostii RHA1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.65</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.65</td>
<td>NRP</td>

<td>holrhizin</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.64</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.64</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>


<td class="cc-heat-cell" title="0.75 (id:0.44, order:1.00, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
<td>0.75</td>
<td>NRP</td>
<td>taxlllaid A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>


<td class="cc-heat-cell" title="0.72 (id:0.44, order:1.00, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
<td>0.72</td>
<td>NRP</td>
<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>
<td>Pseudomonas sp. QS1027</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>thermoactinoamide A</td>
<td>Thermoactinomyces sp. AS95</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002017: 0-12514">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002017/index.html#r1c1" target="_new">BGC0002017</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>potensibactin</td>
<td>Nocardiopsis potens DSM 45234</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417: 0-12076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>rhodochelin</td>
<td>Rhodococcus jostii RHA1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>holrhizin</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-25"></div> <span class="help-tooltip" id="cb-general-help-25">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c3_1.svg">
NZ_CP015600 (3603858-3668451): Pseudomonas antarctica strain PAMC 27494 chrom... (100% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c3_2.svg">
NZ_MSDI01000042 (39682-105124): Pseudomonas fluorescens strain E24 PF E24 con... (100% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c3_3.svg">
NZ_LT907842 (5913901-5979533): Pseudomonas fluorescens strain ATCC 13525 chro... (100% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c3_4.svg">
NZ_CP027762 (3697930-3763242): Pseudomonas sp. LBUM920 chromosome, complete g... (100% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c3_5.svg">
NZ_CP052856 (1473273-1536447): Pseudomonas sp. ADAK22 chromosome, complete ge... (100% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c3_6.svg">
NZ_PYAQ01000012 (137100-203005): Pseudomonas sp. R9.37 NODE 12 length 237118 ... (100% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c3_7.svg">
NZ_JACAQV010000025 (946-70153): Pseudomonas salomonii strain IPO3765 IPO 3765... (64% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c3_8.svg">
NZ_JACAQL010000001 (49386-118604): Pseudomonas sp. IPO3747 IPO 3747 S30 L002 ... (63% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c3_9.svg">
NZ_CP019428 (2841935-2945551): Pseudomonas sp. R76 chromosome, complete genome (53% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c3_10.svg">
NZ_VIUN01000008 (211573-288363): Pseudomonas sp. SJZ103 Ga0314254 108, whole ... (70% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r1c3-download">Download graphic</button>
</div>
<div id="clusterblast-r1c3-svg"></div>

</div>
</div>

<div class="body-details-section r1c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-26"></div> <span class="help-tooltip" id="cb-known-help-26">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r1c3-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r1c3_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r1c3_1.svg">
pyoverdine SMX-1
</option>

<option value = "svg/knownclusterblast_r1c3_2.svg">
Pf-5 pyoverdine
</option>

<option value = "svg/knownclusterblast_r1c3_3.svg">
viscosin
</option>

<option value = "svg/knownclusterblast_r1c3_4.svg">
azotobactin D
</option>

<option value = "svg/knownclusterblast_r1c3_5.svg">
variochelin A/variochelin B
</option>

<option value = "svg/knownclusterblast_r1c3_6.svg">
vacidobactin A/vacidobactin B
</option>

<option value = "svg/knownclusterblast_r1c3_7.svg">
taiwachelin
</option>

<option value = "svg/knownclusterblast_r1c3_8.svg">
acidobactin A/acidobactin B
</option>

<option value = "svg/knownclusterblast_r1c3_9.svg">
variobactin A/variobactin B
</option>

<option value = "svg/knownclusterblast_r1c3_10.svg">
pseudopyronine A/pseudopyronine B
</option>

</select>
<button id="knownclusterblast-r1c3-download">Download graphic</button>
</div>
<div id="knownclusterblast-r1c3-svg"></div>

</div>
</div>

<div class="body-details-section r1c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-27"></div> <span class="help-tooltip" id="cb-sub-help-27">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c3-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-31"></div> <span class="help-tooltip" id="tfbs-body-help-31">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 17.80 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r1c3-531123-AbrC3"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (3):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.72 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c3-531603-ZuR"></div>

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 21.07 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c3-602421-NuR"></div>

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 20.77 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c3-602855-NrtR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r1c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-32"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-32">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c3-domain-bubble-svg" data-filename="NODE_1_length_641826_cov_19.882008_r1c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC4">
Candidate 4 (530420 - 607164): chemical hybrid NRP-metallophore-NRPS
</option>


</select>

<div class="details-svg" id="r1c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-34"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-34">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c3-pfam-details-svg" data-filename="NODE_1_length_641826_cov_19.882008_r1c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-35"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-35">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c3-tigrfam-details-svg" data-filename="NODE_1_length_641826_cov_19.882008_r1c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c3-nrps_pks_products" data-name="r1c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c3-nrps_pks_monomers" data-name="r1c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r1c3-tfbs-finder" data-name="r1c3-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r1c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-28"></div> <span class="help-tooltip" id="nrps-structures-help-28">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 4, location 530420 - 607164:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c3-4" class="smiles-canvas" data-smiles="NC(CO)C(=O)NC(CCCCN)C(=O)NCC(=O)NC(CCCN(O)C=O)C(=O)NC(CCCCN)C(=O)NC(CCCN(O)C=O)C(=O)NC(CO)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(D-Ser - Lys - Gly - Fo-OH-Orn) + (Lys - D-Fo-OH-Orn - Ser)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-ser,lys,gly,fo-oh-orn&amp;nrps2=lys,d-fo-oh-orn,ser" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-ser*,lys*,gly*,fo-oh-orn*&amp;nrps2=lys*,d-fo-oh-orn*,ser*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c3-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CO)C(=O)NC(CCCCN)C(=O)NCC(=O)NC(CCCN(O)C=O)C(=O)NC(CCCCN)C(=O)NC(CCCN(O)C=O)C(=O)NC(CO)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-29"></div> <span class="help-tooltip" id="nrps-monomers-help-29">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">ctg1_495</span></strong>: Ser - Lys - Gly - Fo-OH-Orn
<div class="collapser collapser-target-ctg1_495 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,lys,gly,fo-oh-orn" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,lys*,gly*,fo-oh-orn*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (517..914)</strong>: Ser
<div class="collapser collapser-target-ctg1_495 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ser
<div class="collapser collapser-target-ctg1_495-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>ser (Ser)</dd>
<dt>Single AA prediction:</dt>
<dd>ser (Ser)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVWHVSLIDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ser <span class="serif">DVWHVSLIDK</span> (97% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (1991..2397)</strong>: Lys
<div class="collapser collapser-target-ctg1_495 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Lys
<div class="collapser collapser-target-ctg1_495-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DGEDHGTVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Lys <span class="serif">DGEDHGTVVK</span> (91% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (3042..3433)</strong>: Gly
<div class="collapser collapser-target-ctg1_495 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly
<div class="collapser collapser-target-ctg1_495-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Tiny, hydrophilic, transition to aliphatic (Gly, Ala)</dd>
<dt>Single AA prediction:</dt>
<dd>gly (Gly)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DILQLGLVWK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Gly <span class="serif">DILQLGLVWK</span> (85% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (4083..4483)</strong>: Fo-OH-Orn
<div class="collapser collapser-target-ctg1_495 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Fo-OH-Orn
<div class="collapser collapser-target-ctg1_495-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DGEVCGGVTK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Fo-OH-Orn <span class="serif">DGEVCGGVTK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">ctg1_496</span></strong>: Lys - Fo-OH-Orn - Ser
<div class="collapser collapser-target-ctg1_496 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys,fo-oh-orn,ser" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys*,fo-oh-orn*,ser*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (508..916)</strong>: Lys
<div class="collapser collapser-target-ctg1_496 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Lys
<div class="collapser collapser-target-ctg1_496-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DGEDHGTVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Lys <span class="serif">DGEDHGTVVK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1573..1973)</strong>: Fo-OH-Orn
<div class="collapser collapser-target-ctg1_496 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Fo-OH-Orn
<div class="collapser collapser-target-ctg1_496-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DGEVCGGVTK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Fo-OH-Orn <span class="serif">DGEVCGGVTK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (3092..3490)</strong>: Ser
<div class="collapser collapser-target-ctg1_496 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ser
<div class="collapser collapser-target-ctg1_496-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>ser (Ser)</dd>
<dt>Single AA prediction:</dt>
<dd>ser (Ser)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVWHVSLIDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ser <span class="serif">DVWHVSLIDK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r1c3-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-30"></div> <span class="help-tooltip" id="tfbs-side-help-30">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AbrC3" target="_blank">AbrC3</a></td>
<td>Antibiotic production activator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>




<div class="page" id='r2c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_2_length_547678_cov_19.474303 - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-142"></div> <span class="help-tooltip" id="region_svg-help-142">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_2_length_547678_cov_19.474303.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c1-svg" data-filename="NODE_2_length_547678_cov_19.474303_r2c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 46,666 - 57,511 nt. (total: 10,846 nt)

<a class="cluster-rules-header" id="r2c1-rules-header" href="#r2c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193 or (YcaO or TIGR03604))</div>

<div class="region-svg-container">
<div id='r2c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-143"></div> <span class="help-tooltip" id="focus-panel-help-143">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c1-gene-table-details" data-name="r2c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c1-MIBiG-cluster-compare" data-name="r2c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c1-clusterblast" data-name="r2c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c1-knownclusterblast" data-name="r2c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c1-subclusterblast" data-name="r2c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c1-tfbs-finder" data-name="r2c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r2c1-pfam-details" data-name="r2c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c1-tigrfam-details" data-name="r2c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-42"></div> <span class="help-tooltip" id="gt-help-tooltip-help-42">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-36"></div> <span class="help-tooltip" id="MIBiG-help-36">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c1"></div>
<div id="comparison-MIBiG-r2c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000870: 0-3890">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000870/index.html#r1c1" target="_new">BGC0000870</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>Other</td>

<td>mycolic acid</td>


<td>Mycobacterium tuberculosis H37Rv</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002449: 47-14377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002449/index.html#r1c1" target="_new">BGC0002449</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.41, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>Other</td>

<td>sarasinoside</td>


<td>uncultured bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.46, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.06</td>
<td>Other</td>

<td>5-isoprenylindole-3-carboxylate -D-glycosyl ester</td>


<td>Streptomyces sp. RM-5-8</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002425: 1265-25473">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002425/index.html#r1c1" target="_new">BGC0002425</a></td>





<td class="cc-heat-cell" title="0.22 (id:0.43, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>


<td>0.05</td>
<td>RiPP</td>

<td>stlassin</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000259: 167-24834">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000259/index.html#r1c1" target="_new">BGC0000259</a></td>





<td class="cc-heat-cell" title="0.21 (id:0.43, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>prodigiosin</td>


<td>Serratia marcescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000969: 530-48533">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000969/index.html#r1c1" target="_new">BGC0000969</a></td>





<td class="cc-heat-cell" title="0.20 (id:0.41, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>


<td>0.04</td>
<td>NRP, Polyketide</td>

<td>chondramide A</td>


<td>Chondromyces crocatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002217: 0-33547">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002217/index.html#r1c1" target="_new">BGC0002217</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.45, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>Terpene</td>

<td>HEx-tc3 terpene</td>


<td>Botrytis cinerea T4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000058: 1359-73024">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000058/index.html#r1c1" target="_new">BGC0000058</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.44, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>Polyketide</td>

<td>FD-891</td>


<td>Streptomyces graminofaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000821: 421-26011">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000821/index.html#r1c1" target="_new">BGC0000821</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.44, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>Alkaloid</td>

<td>rebeccamycin</td>


<td>Lechevalieria aerocolonigenes</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002624: 1-29648">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002624/index.html#r1c1" target="_new">BGC0002624</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.42, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>NRP, Polyketide</td>

<td>fischerazole A, fischerazole B, fischerazole C</td>


<td>Fischerella sp. PCC 9431</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000870: 0-3890">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000870/index.html#r1c1" target="_new">BGC0000870</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Other</td>
<td>mycolic acid</td>
<td>Mycobacterium tuberculosis H37Rv</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002449: 47-14377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002449/index.html#r1c1" target="_new">BGC0002449</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.41, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>Other</td>
<td>sarasinoside</td>
<td>uncultured bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.46, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Other</td>
<td>5-isoprenylindole-3-carboxylate -D-glycosyl ester</td>
<td>Streptomyces sp. RM-5-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002425: 1265-25473">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002425/index.html#r1c1" target="_new">BGC0002425</a></td>


<td class="cc-heat-cell" title="0.22 (id:0.43, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
<td>0.22</td>
<td>RiPP</td>
<td>stlassin</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000259: 167-24834">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000259/index.html#r1c1" target="_new">BGC0000259</a></td>


<td class="cc-heat-cell" title="0.21 (id:0.43, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
<td>0.21</td>
<td>Polyketide</td>
<td>prodigiosin</td>
<td>Serratia marcescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000969: 530-48533">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000969/index.html#r1c1" target="_new">BGC0000969</a></td>


<td class="cc-heat-cell" title="0.20 (id:0.41, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>
<td>0.20</td>
<td>NRP, Polyketide</td>
<td>chondramide A</td>
<td>Chondromyces crocatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002217: 0-33547">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002217/index.html#r1c1" target="_new">BGC0002217</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.45, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>Terpene</td>
<td>HEx-tc3 terpene</td>
<td>Botrytis cinerea T4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000058: 1359-73024">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000058/index.html#r1c1" target="_new">BGC0000058</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.44, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>Polyketide</td>
<td>FD-891</td>
<td>Streptomyces graminofaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000821: 421-26011">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000821/index.html#r1c1" target="_new">BGC0000821</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.44, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>Alkaloid</td>
<td>rebeccamycin</td>
<td>Lechevalieria aerocolonigenes</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002624: 1-29648">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002624/index.html#r1c1" target="_new">BGC0002624</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.42, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>NRP, Polyketide</td>
<td>fischerazole A, fischerazole B, fischerazole C</td>
<td>Fischerella sp. PCC 9431</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-37"></div> <span class="help-tooltip" id="cb-general-help-37">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c1_1.svg">
NZ_CP015600 (6391218-6402064): Pseudomonas antarctica strain PAMC 27494 chrom... (100% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r2c1_2.svg">
NZ_LT907842 (2087248-2096625): Pseudomonas fluorescens strain ATCC 13525 chro... (100% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r2c1_3.svg">
NZ_MSDI01000011 (181865-192711): Pseudomonas fluorescens strain E24 PF E24 co... (100% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r2c1_4.svg">
NZ_JACAQV010000030 (77522-88368): Pseudomonas salomonii strain IPO3765 IPO 37... (100% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r2c1_5.svg">
NZ_FNOX01000011 (170961-181807): Pseudomonas salomonii strain ICMP 14252, who... (100% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r2c1_6.svg">
NZ_CP019428 (6737260-6748106): Pseudomonas sp. R76 chromosome, complete genome (100% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r2c1_7.svg">
NZ_JACAOU010000018 (160134-170980): Pseudomonas tolaasii strain B1001 run0228... (100% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r2c1_8.svg">
NZ_PYAQ01000001 (604632-615478): Pseudomonas sp. R9.37 NODE 1 length 780182 c... (100% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r2c1_9.svg">
NZ_CP020369 (3976325-3985777): Pseudomonas tolaasii strain 2192T genome (100% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r2c1_10.svg">
NZ_CP052856 (4296063-4305536): Pseudomonas sp. ADAK22 chromosome, complete ge... (100% of genes show similarity), bacteriocin
</option>

</select>
<button id= "clusterblast-r2c1-download">Download graphic</button>
</div>
<div id="clusterblast-r2c1-svg"></div>

</div>
</div>

<div class="body-details-section r2c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-38"></div> <span class="help-tooltip" id="cb-known-help-38">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-39"></div> <span class="help-tooltip" id="cb-sub-help-39">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-41"></div> <span class="help-tooltip" id="tfbs-body-help-41">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AfsR
<div class="collapser collapser-target-AfsR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Pleiotropic regulatory for antibiotic production<br>
<span>Score</span>: 20.11 out of 23.05
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r2c1-47258-AfsR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.80 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c1-57308-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r2c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-43"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-43">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-pfam-details-svg" data-filename="NODE_2_length_547678_cov_19.474303_r2c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-44"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-44">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-tigrfam-details-svg" data-filename="NODE_2_length_547678_cov_19.474303_r2c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c1-tfbs-finder" data-name="r2c1-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r2c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-40"></div> <span class="help-tooltip" id="tfbs-side-help-40">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AfsR" target="_blank">AfsR</a></td>
<td>Pleiotropic regulatory for antibiotic production</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r4c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_4_length_376817_cov_19.716587 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-144"></div> <span class="help-tooltip" id="region_svg-help-144">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_4_length_376817_cov_19.716587.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c1-svg" data-filename="NODE_4_length_376817_cov_19.716587_r4c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 33,478 nt. (total: 33,478 nt)

<a class="cluster-rules-header" id="r4c1-rules-header" href="#r4c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r4c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r4c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-145"></div> <span class="help-tooltip" id="focus-panel-help-145">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c1-gene-table-details" data-name="r4c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c1-nrps_pks" data-name="r4c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r4c1-MIBiG-cluster-compare" data-name="r4c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c1-clusterblast" data-name="r4c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c1-knownclusterblast" data-name="r4c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c1-subclusterblast" data-name="r4c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c1-tfbs-finder" data-name="r4c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r4c1-nrps-pks-bubbles" data-name="r4c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r4c1-pfam-details" data-name="r4c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r4c1-tigrfam-details" data-name="r4c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r4c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-55"></div> <span class="help-tooltip" id="gt-help-tooltip-help-55">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-45"></div> <span class="help-tooltip" id="nrps-domain-help-45">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-details-svg" data-filename="NODE_4_length_376817_cov_19.716587_r4c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r4c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r4c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-46"></div> <span class="help-tooltip" id="MIBiG-help-46">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c1"></div>
<div id="comparison-MIBiG-r4c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>thermoactinoamide A</td>


<td>Thermoactinomyces sp. AS95</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002493: 1205-6076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002493/index.html#r1c1" target="_new">BGC0002493</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>parabactin</td>


<td>Paracoccus denitrificans PD1222</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.44, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>vicibactin</td>


<td>Rhizobium etli CFN 42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002017: 0-12514">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002017/index.html#r1c1" target="_new">BGC0002017</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>potensibactin</td>


<td>Nocardiopsis potens DSM 45234</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001671: 5106-76272">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001671/index.html#r1c1" target="_new">BGC0001671</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>monobactam</td>


<td>Agrobacterium tumefaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000913: 1-11028">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000913/index.html#r1c1" target="_new">BGC0000913</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>Other</td>

<td>MK-8</td>


<td>Enterobacter cloacae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001464: 0-6705">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001464/index.html#r1c1" target="_new">BGC0001464</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Other</td>

<td>tetrabromopyrrole</td>


<td>Pseudoalteromonas sp. PS5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825: 0-14850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>xenematide</td>


<td>Xenorhabdus nematophila AN6/1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>thermoactinoamide A</td>
<td>Thermoactinomyces sp. AS95</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002493: 1205-6076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002493/index.html#r1c1" target="_new">BGC0002493</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>parabactin</td>
<td>Paracoccus denitrificans PD1222</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.44, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>vicibactin</td>
<td>Rhizobium etli CFN 42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002017: 0-12514">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002017/index.html#r1c1" target="_new">BGC0002017</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>potensibactin</td>
<td>Nocardiopsis potens DSM 45234</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001671: 5106-76272">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001671/index.html#r1c1" target="_new">BGC0001671</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>monobactam</td>
<td>Agrobacterium tumefaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000913: 1-11028">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000913/index.html#r1c1" target="_new">BGC0000913</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Other</td>
<td>MK-8</td>
<td>Enterobacter cloacae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001464: 0-6705">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001464/index.html#r1c1" target="_new">BGC0001464</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Other</td>
<td>tetrabromopyrrole</td>
<td>Pseudoalteromonas sp. PS5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825: 0-14850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>xenematide</td>
<td>Xenorhabdus nematophila AN6/1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-47"></div> <span class="help-tooltip" id="cb-general-help-47">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c1_1.svg">
NZ_CP015600 (4464342-4517239): Pseudomonas antarctica strain PAMC 27494 chrom... (53% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_2.svg">
NZ_MSDI01000001 (680074-732971): Pseudomonas fluorescens strain E24 PF E24 co... (53% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_3.svg">
NZ_LT907842 (208134-259890): Pseudomonas fluorescens strain ATCC 13525 chromo... (53% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_4.svg">
NZ_LT629704 (2781129-2832756): Pseudomonas antarctica strain LMG 22709 chromo... (55% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_5.svg">
NZ_LT629708 (1796042-1846970): Pseudomonas extremorientalis strain LMG 19695 ... (59% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_6.svg">
NZ_FOIH01000005 (238861-291740): Pseudomonas sp. NFR09, whole genome shotgun ... (55% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_7.svg">
NZ_FOQS01000003 (395286-448165): Pseudomonas sp. NFPP04, whole genome shotgun... (55% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_8.svg">
NZ_FOKD01000007 (278172-331051): Pseudomonas sp. NFPP24, whole genome shotgun... (55% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_9.svg">
NZ_QAJM01000014 (588-53473): Pseudomonas sp. KBW05 KBW5 ACAGTG contig 14, who... (56% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_10.svg">
NZ_CP046103 (3497555-3550452): Pseudomonas sp. IB20 chromosome (50% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r4c1-download">Download graphic</button>
</div>
<div id="clusterblast-r4c1-svg"></div>

</div>
</div>

<div class="body-details-section r4c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-48"></div> <span class="help-tooltip" id="cb-known-help-48">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r4c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r4c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r4c1_1.svg">
azotobactin D
</option>

<option value = "svg/knownclusterblast_r4c1_2.svg">
Pf-5 pyoverdine
</option>

<option value = "svg/knownclusterblast_r4c1_3.svg">
crochelin A
</option>

<option value = "svg/knownclusterblast_r4c1_4.svg">
delftibactin A/delftibactin B
</option>

<option value = "svg/knownclusterblast_r4c1_5.svg">
corynecin III/corynecin I/corynecin II
</option>

<option value = "svg/knownclusterblast_r4c1_6.svg">
taiwachelin
</option>

</select>
<button id="knownclusterblast-r4c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r4c1-svg"></div>

</div>
</div>

<div class="body-details-section r4c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-49"></div> <span class="help-tooltip" id="cb-sub-help-49">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-53"></div> <span class="help-tooltip" id="tfbs-body-help-53">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 22.73 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r4c1-523-ZuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 22.98 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r4c1-14484-ZuR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (5):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 20.96 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r4c1-366-NrtR"></div>

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 22.22 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r4c1-28458-NuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.63 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r4c1-28459-ZuR"></div>

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 21.44 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r4c1-28485-LexA"></div>

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 20.58 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r4c1-33118-NrtR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r4c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-54"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-54">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-domain-bubble-svg" data-filename="NODE_4_length_376817_cov_19.716587_r4c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r4c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (0 - 33478): single NRPS
</option>


</select>

<div class="details-svg" id="r4c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r4c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r4c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-56"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-56">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-pfam-details-svg" data-filename="NODE_4_length_376817_cov_19.716587_r4c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r4c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-57"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-57">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-tigrfam-details-svg" data-filename="NODE_4_length_376817_cov_19.716587_r4c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r4c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r4c1-nrps_pks_products" data-name="r4c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r4c1-nrps_pks_monomers" data-name="r4c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r4c1-tfbs-finder" data-name="r4c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r4c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-50"></div> <span class="help-tooltip" id="nrps-structures-help-50">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 0 - 33478:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c1-1" class="smiles-canvas" data-smiles="NC(CCC(=O)O)C(=O)NC(Cc1ccc(O)cc1)C(=O)NC(CCN)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Glu - D-Tyr - Dab)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=glu,d-tyr,dab" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=glu*,d-tyr*,dab*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CCC(=O)O)C(=O)NC(Cc1ccc(O)cc1)C(=O)NC(CCN)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r4c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-51"></div> <span class="help-tooltip" id="nrps-monomers-help-51">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">ctg4_1</span></strong>: pk - Glu - Tyr - Dab
<div class="collapser collapser-target-ctg4_1 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=glu,tyr,dab" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=glu*,tyr*,dab*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>CAL_domain (41..477)</strong>: pk
<div class="collapser collapser-target-ctg4_1 collapser-level-candidate"> </div><div class="collapser-content">

minowa:  NH2
<div class="collapser collapser-target-ctg4_1-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>NH2: 156.4</dt>

<dd></dd><dt>AHBA: 20.6</dt>

<dd></dd><dt>fatty_acid: 15.7</dt>

<dd></dd><dt>Acetyl-CoA: 1.6</dt>

<dd></dd><dt>shikimic_acid: -1.5</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1128..1521)</strong>: Glu
<div class="collapser collapser-target-ctg4_1 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Glu
<div class="collapser collapser-target-ctg4_1-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>glu,gln (Glu, Gln)</dd>
<dt>Single AA prediction:</dt>
<dd>glu (Glu)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVWHFGRINK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Glu <span class="serif">DVWHFGRINK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (2174..2578)</strong>: Tyr
<div class="collapser collapser-target-ctg4_1 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Tyr
<div class="collapser collapser-target-ctg4_1-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAEGIGAVMK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Tyr <span class="serif">DAEGIGAVMK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (3696..4095)</strong>: Dab
<div class="collapser collapser-target-ctg4_1 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Dab
<div class="collapser collapser-target-ctg4_1-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DIWELTADDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Dab <span class="serif">DIWELTADDK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r4c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-52"></div> <span class="help-tooltip" id="tfbs-side-help-52">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r4c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_4_length_376817_cov_19.716587 - Region 2 - NI-siderophore <div class="help-container"> <div class="help-icon" data-id="region_svg-help-146"></div> <span class="help-tooltip" id="region_svg-help-146">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_4_length_376817_cov_19.716587.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c2-svg" data-filename="NODE_4_length_376817_cov_19.716587_r4c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 122,693 - 134,618 nt. (total: 11,926 nt)

<a class="cluster-rules-header" id="r4c2-rules-header" href="#r4c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c2-rules">NI-siderophore: IucA_IucC</div>

<div class="region-svg-container">
<div id='r4c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-147"></div> <span class="help-tooltip" id="focus-panel-help-147">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c2-gene-table-details" data-name="r4c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c2-MIBiG-cluster-compare" data-name="r4c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c2-clusterblast" data-name="r4c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c2-knownclusterblast" data-name="r4c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c2-subclusterblast" data-name="r4c2-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r4c2-tfbs-finder" data-name="r4c2-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r4c2-pfam-details" data-name="r4c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r4c2-tigrfam-details" data-name="r4c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r4c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-64"></div> <span class="help-tooltip" id="gt-help-tooltip-help-64">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-58"></div> <span class="help-tooltip" id="MIBiG-help-58">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c2"></div>
<div id="comparison-MIBiG-r4c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NI-siderophore</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002469: 0-6373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002469/index.html#r1c1" target="_new">BGC0002469</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Other</td>

<td>woodybactin A, woodybactin B, woodybactin C, woodybactin D</td>


<td>Shewanella woodyi ATCC 51908</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001453: 0-7234">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001453/index.html#r1c1" target="_new">BGC0001453</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Other</td>

<td>desferrioxamine</td>


<td>Streptomyces argillaceus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000940: 0-7328">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000940/index.html#r1c1" target="_new">BGC0000940</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Other</td>

<td>desferrioxamin B, desferrioxamine E</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000295: 569-12865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000295/index.html#r1c1" target="_new">BGC0000295</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>acinetoferrin</td>


<td>Acinetobacter haemolyticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713: 164-10986">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Other</td>

<td>ochrobactin</td>


<td>Ochrobactrum anthropi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499: 2267-11578">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Other</td>

<td>aerobactin</td>


<td>Pantoea ananatis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000944: 0-6093">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000944/index.html#r1c1" target="_new">BGC0000944</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Other (Non-NRP siderophore)</td>

<td>staphyloferrin A</td>


<td>Staphylococcus aureus subsp. aureus NCTC 8325</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001498: 1442-13533">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001498/index.html#r1c1" target="_new">BGC0001498</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Other</td>

<td>aerobactin</td>


<td>Xenorhabdus szentirmaii DSM 16338</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002681: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002681/index.html#r1c1" target="_new">BGC0002681</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.43, order:0.45, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Other</td>

<td>rhizobactin</td>


<td>Francisella tularensis subsp. tularensis SCHU S4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002682: 139-12158">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002682/index.html#r1c1" target="_new">BGC0002682</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Other</td>

<td>aerobactin</td>


<td>Vibrio mimicus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002469: 0-6373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002469/index.html#r1c1" target="_new">BGC0002469</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Other</td>
<td>woodybactin A, woodybactin B, woodybactin C, woodybactin D</td>
<td>Shewanella woodyi ATCC 51908</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001453: 0-7234">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001453/index.html#r1c1" target="_new">BGC0001453</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Other</td>
<td>desferrioxamine</td>
<td>Streptomyces argillaceus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000940: 0-7328">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000940/index.html#r1c1" target="_new">BGC0000940</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>desferrioxamin B, desferrioxamine E</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000295: 569-12865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000295/index.html#r1c1" target="_new">BGC0000295</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>acinetoferrin</td>
<td>Acinetobacter haemolyticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713: 164-10986">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Other</td>
<td>ochrobactin</td>
<td>Ochrobactrum anthropi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499: 2267-11578">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Other</td>
<td>aerobactin</td>
<td>Pantoea ananatis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000944: 0-6093">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000944/index.html#r1c1" target="_new">BGC0000944</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Other (Non-NRP siderophore)</td>
<td>staphyloferrin A</td>
<td>Staphylococcus aureus subsp. aureus NCTC 8325</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001498: 1442-13533">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001498/index.html#r1c1" target="_new">BGC0001498</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Other</td>
<td>aerobactin</td>
<td>Xenorhabdus szentirmaii DSM 16338</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002681: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002681/index.html#r1c1" target="_new">BGC0002681</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.43, order:0.45, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Other</td>
<td>rhizobactin</td>
<td>Francisella tularensis subsp. tularensis SCHU S4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002682: 139-12158">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002682/index.html#r1c1" target="_new">BGC0002682</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Other</td>
<td>aerobactin</td>
<td>Vibrio mimicus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-59"></div> <span class="help-tooltip" id="cb-general-help-59">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c2_1.svg">
NZ_CP015600 (4606311-4618237): Pseudomonas antarctica strain PAMC 27494 chrom... (100% of genes show similarity), siderophore
</option>

<option value = "svg/clusterblast_r4c2_2.svg">
NZ_LT907842 (349279-361205): Pseudomonas fluorescens strain ATCC 13525 chromo... (100% of genes show similarity), siderophore
</option>

<option value = "svg/clusterblast_r4c2_3.svg">
NZ_MSDI01000001 (821833-833759): Pseudomonas fluorescens strain E24 PF E24 co... (100% of genes show similarity), siderophore
</option>

<option value = "svg/clusterblast_r4c2_4.svg">
NZ_CP019428 (4908769-4920695): Pseudomonas sp. R76 chromosome, complete genome (100% of genes show similarity), siderophore
</option>

<option value = "svg/clusterblast_r4c2_5.svg">
NZ_CP015637 (4959700-4971626): Pseudomonas fluorescens strain L321 chromosome... (100% of genes show similarity), siderophore
</option>

<option value = "svg/clusterblast_r4c2_6.svg">
NC_023064 (5155803-5167729): Pseudomonas sp. TKP, complete sequence (100% of genes show similarity), siderophore
</option>

<option value = "svg/clusterblast_r4c2_7.svg">
NZ_QKMB01000024 (26324-38250): Pseudomonas sp. OV286 Ga0215801 124, whole gen... (100% of genes show similarity), siderophore
</option>

<option value = "svg/clusterblast_r4c2_8.svg">
NZ_JACAOZ010000014 (275275-287201): Pseudomonas edaphica strain B7002 run0228... (100% of genes show similarity), siderophore
</option>

<option value = "svg/clusterblast_r4c2_9.svg">
NZ_FNOX01000005 (314561-326487): Pseudomonas salomonii strain ICMP 14252, who... (100% of genes show similarity), siderophore
</option>

<option value = "svg/clusterblast_r4c2_10.svg">
NZ_CP027727 (4263227-4275153): Pseudomonas sp. R5-89-07 chromosome, complete ... (100% of genes show similarity), siderophore
</option>

</select>
<button id= "clusterblast-r4c2-download">Download graphic</button>
</div>
<div id="clusterblast-r4c2-svg"></div>

</div>
</div>

<div class="body-details-section r4c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-60"></div> <span class="help-tooltip" id="cb-known-help-60">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r4c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-61"></div> <span class="help-tooltip" id="cb-sub-help-61">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r4c2-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-63"></div> <span class="help-tooltip" id="tfbs-body-help-63">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 19.04 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r4c2-126851-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r4c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-65"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-65">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c2-pfam-details-svg" data-filename="NODE_4_length_376817_cov_19.716587_r4c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r4c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-66"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-66">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c2-tigrfam-details-svg" data-filename="NODE_4_length_376817_cov_19.716587_r4c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r4c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r4c2-tfbs-finder" data-name="r4c2-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r4c2-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-62"></div> <span class="help-tooltip" id="tfbs-side-help-62">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r6c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_6_length_246143_cov_19.272524 - Region 1 - arylpolyene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-148"></div> <span class="help-tooltip" id="region_svg-help-148">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_6_length_246143_cov_19.272524.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c1-svg" data-filename="NODE_6_length_246143_cov_19.272524_r6c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 77,116 - 120,691 nt. (total: 43,576 nt)

<a class="cluster-rules-header" id="r6c1-rules-header" href="#r6c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c1-rules">arylpolyene: (APE_KS1 or APE_KS2)</div>

<div class="region-svg-container">
<div id='r6c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-149"></div> <span class="help-tooltip" id="focus-panel-help-149">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c1-gene-table-details" data-name="r6c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c1-MIBiG-cluster-compare" data-name="r6c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c1-clusterblast" data-name="r6c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c1-knownclusterblast" data-name="r6c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c1-subclusterblast" data-name="r6c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r6c1-tfbs-finder" data-name="r6c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r6c1-pfam-details" data-name="r6c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r6c1-tigrfam-details" data-name="r6c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r6c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-73"></div> <span class="help-tooltip" id="gt-help-tooltip-help-73">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-67"></div> <span class="help-tooltip" id="MIBiG-help-67">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c1"></div>
<div id="comparison-MIBiG-r6c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">arylpolyene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002341: 602-1868">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002341/index.html#r1c1" target="_new">BGC0002341</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Other</td>

<td>3,5-dibromo-p-anisic acid</td>


<td>Planctomycetales bacterium 10988</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Other</td>

<td>citrinin</td>


<td>Monascus aurantiacus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002493: 1205-6076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002493/index.html#r1c1" target="_new">BGC0002493</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>parabactin</td>


<td>Paracoccus denitrificans PD1222</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001366: 10-15268">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001366/index.html#r1c1" target="_new">BGC0001366</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>BE-24566B, zunyimycin A</td>


<td>Streptomyces sp. FJS31-2</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231: 189-5010">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>griseusin A, griseusin B</td>


<td>Streptomyces griseus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002008: 0-15251">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002008/index.html#r1c1" target="_new">BGC0002008</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.49, order:0.36, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>aryl polyenes</td>


<td>Xenorhabdus doucetiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001897: 3-16597">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001897/index.html#r1c1" target="_new">BGC0001897</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>cepacin A</td>


<td>Burkholderia ambifaria IOP40-10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000252: 248-15270">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000252/index.html#r1c1" target="_new">BGC0000252</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>nonactin</td>


<td>Streptomyces griseus subsp. griseus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514: 0-37469">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Other</td>

<td>aristeromycin</td>


<td>Streptomyces citricolor</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001463: 0-6707">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001463/index.html#r1c1" target="_new">BGC0001463</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.22</td>
<td>Other</td>

<td>2,3,4,5-tetrabromopyrrole</td>


<td>Pseudoalteromonas piscicida</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002341: 602-1868">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002341/index.html#r1c1" target="_new">BGC0002341</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>3,5-dibromo-p-anisic acid</td>
<td>Planctomycetales bacterium 10988</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>citrinin</td>
<td>Monascus aurantiacus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002493: 1205-6076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002493/index.html#r1c1" target="_new">BGC0002493</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>parabactin</td>
<td>Paracoccus denitrificans PD1222</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001366: 10-15268">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001366/index.html#r1c1" target="_new">BGC0001366</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>BE-24566B, zunyimycin A</td>
<td>Streptomyces sp. FJS31-2</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231: 189-5010">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>griseusin A, griseusin B</td>
<td>Streptomyces griseus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002008: 0-15251">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002008/index.html#r1c1" target="_new">BGC0002008</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.49, order:0.36, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>aryl polyenes</td>
<td>Xenorhabdus doucetiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001897: 3-16597">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001897/index.html#r1c1" target="_new">BGC0001897</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>cepacin A</td>
<td>Burkholderia ambifaria IOP40-10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000252: 248-15270">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000252/index.html#r1c1" target="_new">BGC0000252</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>nonactin</td>
<td>Streptomyces griseus subsp. griseus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514: 0-37469">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Other</td>
<td>aristeromycin</td>
<td>Streptomyces citricolor</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001463: 0-6707">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001463/index.html#r1c1" target="_new">BGC0001463</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Other</td>
<td>2,3,4,5-tetrabromopyrrole</td>
<td>Pseudoalteromonas piscicida</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-68"></div> <span class="help-tooltip" id="cb-general-help-68">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c1_1.svg">
NZ_CP015600 (486845-530421): Pseudomonas antarctica strain PAMC 27494 chromos... (100% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r6c1_2.svg">
NZ_CP027724 (474291-517867): Pseudomonas orientalis strain L1-3-08 chromosome... (100% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r6c1_3.svg">
NZ_CP027755 (501717-545293): Pseudomonas synxantha strain 2-79 chromosome, co... (97% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r6c1_4.svg">
NC_017911 (452874-496450): Pseudomonas fluorescens A506, complete sequence (100% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r6c1_5.svg">
NZ_FPJJ01000004 (250958-294534): Pseudomonas sp. NFR02, whole genome shotgun ... (100% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r6c1_6.svg">
NZ_CM001513 (488676-532252): Pseudomonas lactis strain SS101 chromosome, whol... (100% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r6c1_7.svg">
NZ_FPJI01000002 (308656-352232): Pseudomonas sp. NFPP02, whole genome shotgun... (100% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r6c1_8.svg">
NZ_CP046103 (409202-452778): Pseudomonas sp. IB20 chromosome (100% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r6c1_9.svg">
NZ_LKEG01000042 (203605-247181): Pseudomonas marginalis strain ICMP 3553 scaf... (95% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r6c1_10.svg">
NZ_JACAQV010000001 (39824-83400): Pseudomonas salomonii strain IPO3765 IPO 37... (100% of genes show similarity), arylpolyene
</option>

</select>
<button id= "clusterblast-r6c1-download">Download graphic</button>
</div>
<div id="clusterblast-r6c1-svg"></div>

</div>
</div>

<div class="body-details-section r6c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-69"></div> <span class="help-tooltip" id="cb-known-help-69">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r6c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r6c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r6c1_1.svg">
APE Vf
</option>

<option value = "svg/knownclusterblast_r6c1_2.svg">
aryl polyenes
</option>

<option value = "svg/knownclusterblast_r6c1_3.svg">
APE Ec
</option>

</select>
<button id="knownclusterblast-r6c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r6c1-svg"></div>

</div>
</div>

<div class="body-details-section r6c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-70"></div> <span class="help-tooltip" id="cb-sub-help-70">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r6c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-72"></div> <span class="help-tooltip" id="tfbs-body-help-72">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 20.58 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r6c1-81483-ZuR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 21.60 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r6c1-113921-NuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 20.16 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r6c1-114144-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r6c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-74"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-74">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-pfam-details-svg" data-filename="NODE_6_length_246143_cov_19.272524_r6c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r6c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-75"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-75">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-tigrfam-details-svg" data-filename="NODE_6_length_246143_cov_19.272524_r6c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r6c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r6c1-tfbs-finder" data-name="r6c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r6c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-71"></div> <span class="help-tooltip" id="tfbs-side-help-71">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r13c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_13_length_148291_cov_19.867833 - Region 1 - redox-cofactor <div class="help-container"> <div class="help-icon" data-id="region_svg-help-150"></div> <span class="help-tooltip" id="region_svg-help-150">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_13_length_148291_cov_19.867833.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r13c1-svg" data-filename="NODE_13_length_148291_cov_19.867833_r13c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 32,410 - 54,557 nt. (total: 22,148 nt)

<a class="cluster-rules-header" id="r13c1-rules-header" href="#r13c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r13c1-rules">redox-cofactor: (cds(PF04055 and (TIGR04085 or SPASM)) and (TIGR02111 or TIGR03966) and minimum(1, [Bottromycin_Methyltransferase_RRE, Cyanobactin_D_RRE, Cytolysin_C_RRE, Goadsporin_CD_RRE, Heterocycloanthracin_C_RRE, Lanthipeptide_LanB_RRE, Lasso_Fused_RRE, Mycofactocin_RRE, NHLP_CD_RRE, Other_Sactipeptide_rSAM_RRE, Pantocin_Microcin_RRE, Plantazolicin_C_RRE, PlpY_RRE, PqqD_RRE, Proteusin_Epimerase_RRE, Proteusin_Methyltransferase_RRE, Quinohemoprotein_rSAM_RRE, Ranthipeptide_rSAM_RRE, Stand_Alone_Lasso_RRE, Streptide_RRE, Subtilosin_rSAM_RRE, Thiaglutamate_B_RRE, Thiaglutamate_G_RRE, Thiaglutamate_I_RRE, Thiopeptide_F_RRE, Thuricin_rSAM_RRE, Thurincin_rSAM_RRE, Trifolitoxin_RRE]))</div>

<div class="region-svg-container">
<div id='r13c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-151"></div> <span class="help-tooltip" id="focus-panel-help-151">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r13c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r13c1-gene-table-details" data-name="r13c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r13c1-MIBiG-cluster-compare" data-name="r13c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r13c1-clusterblast" data-name="r13c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r13c1-knownclusterblast" data-name="r13c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r13c1-subclusterblast" data-name="r13c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r13c1-tfbs-finder" data-name="r13c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r13c1-pfam-details" data-name="r13c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r13c1-tigrfam-details" data-name="r13c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r13c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-83"></div> <span class="help-tooltip" id="gt-help-tooltip-help-83">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r13c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r13c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r13c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r13c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r13c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-76"></div> <span class="help-tooltip" id="MIBiG-help-76">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r13c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r13c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r13c1"></div>
<div id="comparison-MIBiG-r13c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">redox-cofactor</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000919: 746-6062">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000919/index.html#r1c1" target="_new">BGC0000919</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Other</td>

<td>phaseolotoxin</td>


<td>Pseudomonas syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylobacter marinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000905: 1-16359">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000905/index.html#r1c1" target="_new">BGC0000905</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>heme D1</td>


<td>Heliobacillus mobilis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000906: 424-16949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000906/index.html#r1c1" target="_new">BGC0000906</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>heme D1</td>


<td>Heliophilum fasciatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000258: 2-24341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000258/index.html#r1c1" target="_new">BGC0000258</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>prodigiosin</td>


<td>Serratia sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Other</td>

<td>ectoine</td>


<td>Sporosarcina pasteurii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000663: 0-13757">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000663/index.html#r1c1" target="_new">BGC0000663</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.22</td>
<td>Terpene</td>

<td>hopene</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000260: 853-35060">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000260/index.html#r1c1" target="_new">BGC0000260</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>prodigiosin</td>


<td>Hahella chejuensis KCTC 2396</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002675: 0-20254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002675/index.html#r1c1" target="_new">BGC0002675</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>di-pyrrolyl-dipyrromethene prodigiosin</td>


<td>Pseudoalteromonas rubra</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000841: 0-11023">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000841/index.html#r1c1" target="_new">BGC0000841</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Other (Non-NRP beta-lactam)</td>

<td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate, clavulanic acid</td>


<td>Streptomyces clavuligerus ATCC 27064</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r13c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000919: 746-6062">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000919/index.html#r1c1" target="_new">BGC0000919</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Other</td>
<td>phaseolotoxin</td>
<td>Pseudomonas syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylobacter marinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000905: 1-16359">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000905/index.html#r1c1" target="_new">BGC0000905</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>heme D1</td>
<td>Heliobacillus mobilis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000906: 424-16949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000906/index.html#r1c1" target="_new">BGC0000906</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>heme D1</td>
<td>Heliophilum fasciatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000258: 2-24341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000258/index.html#r1c1" target="_new">BGC0000258</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>prodigiosin</td>
<td>Serratia sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Other</td>
<td>ectoine</td>
<td>Sporosarcina pasteurii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000663: 0-13757">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000663/index.html#r1c1" target="_new">BGC0000663</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Terpene</td>
<td>hopene</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000260: 853-35060">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000260/index.html#r1c1" target="_new">BGC0000260</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Polyketide</td>
<td>prodigiosin</td>
<td>Hahella chejuensis KCTC 2396</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002675: 0-20254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002675/index.html#r1c1" target="_new">BGC0002675</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Polyketide</td>
<td>di-pyrrolyl-dipyrromethene prodigiosin</td>
<td>Pseudoalteromonas rubra</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000841: 0-11023">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000841/index.html#r1c1" target="_new">BGC0000841</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Other (Non-NRP beta-lactam)</td>
<td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate, clavulanic acid</td>
<td>Streptomyces clavuligerus ATCC 27064</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r13c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-77"></div> <span class="help-tooltip" id="cb-general-help-77">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r13c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r13c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r13c1_1.svg">
NZ_LT222319 (521801-564397): Pseudomonas cerasi isolate Sour cherry (Prunus c... (35% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r13c1_2.svg">
NZ_LT629769 (4147937-4189287): Pseudomonas syringae strain 31R1 chromosome I (33% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r13c1_3.svg">
NZ_CP006256 (5365585-5405357): Pseudomonas syringae pv. syringae HS191 chromo... (36% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r13c1_4.svg">
NZ_RBOI01000041 (87050-131091): Pseudomonas syringae pv. aptata strain G733 P... (35% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r13c1_5.svg">
NZ_CP005970 (3160121-3201723): Pseudomonas syringae UMAF0158 chromosome, comp... (34% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r13c1_6.svg">
NZ_KB316543 (28005-72046): Pseudomonas syringae BRIP34881 scaffold13, whole g... (29% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r13c1_7.svg">
NZ_CP028490 (4967599-5011640): Pseudomonas syringae pv. atrofaciens strain LM... (30% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r13c1_8.svg">
NC_007005 (5504337-5546191): Pseudomonas syringae pv. syringae B728a, complet... (28% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r13c1_9.svg">
NZ_AP020337 (5391819-5433703): Pseudomonas sp. KUIN-1 (27% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r13c1_10.svg">
NZ_FNJH01000007 (136429-180470): Pseudomonas congelans strain DSM 14939, whol... (35% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r13c1-download">Download graphic</button>
</div>
<div id="clusterblast-r13c1-svg"></div>

</div>
</div>

<div class="body-details-section r13c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-78"></div> <span class="help-tooltip" id="cb-known-help-78">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r13c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r13c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r13c1_1.svg">
lankacidin C
</option>

</select>
<button id="knownclusterblast-r13c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r13c1-svg"></div>

</div>
</div>

<div class="body-details-section r13c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-79"></div> <span class="help-tooltip" id="cb-sub-help-79">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r13c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-82"></div> <span class="help-tooltip" id="tfbs-body-help-82">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 20.83 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r13c1-33259-ZuR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 23.61 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r13c1-50847-NrtR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r13c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-84"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-84">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r13c1-pfam-details-svg" data-filename="NODE_13_length_148291_cov_19.867833_r13c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r13c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r13c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-85"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-85">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r13c1-tigrfam-details-svg" data-filename="NODE_13_length_148291_cov_19.867833_r13c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r13c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r13c1-RREfinder" data-name="r13c1-RREfinder">
RREFinder
</div>



<div class="sidepanel-details-header r13c1-tfbs-finder" data-name="r13c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r13c1-RREfinder">
<div class="more-details">
<div class="heading">
<span>RRE predictions</span>
<div class="help-container"> <div class="help-icon" data-id="rre-side-help-80"></div> <span class="help-tooltip" id="rre-side-help-80">RREfinder results sidepanel.</span></div>
</div>
<dl class = 'prediction-text'>

<span><strong>redox-cofactor protocluster</strong> (32409...54557)</span>
<dt></dt>




<strong>ctg13_40</strong> <div class="collapser collapser-target-ctg13_40 collapser-level-cds"> </div><div class="collapser-content">

<dd>RRE type: PqqD_RRE</dd>
<dd>Location: 1 - 90</dd>
<dd>Score: 113.2</dd>

</div><br>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r13c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-81"></div> <span class="help-tooltip" id="tfbs-side-help-81">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r15c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_15_length_143283_cov_19.608976 - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-152"></div> <span class="help-tooltip" id="region_svg-help-152">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_15_length_143283_cov_19.608976.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r15c1-svg" data-filename="NODE_15_length_143283_cov_19.608976_r15c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 34,073 - 44,954 nt. (total: 10,882 nt)

<a class="cluster-rules-header" id="r15c1-rules-header" href="#r15c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r15c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193 or (YcaO or TIGR03604))</div>

<div class="region-svg-container">
<div id='r15c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-153"></div> <span class="help-tooltip" id="focus-panel-help-153">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r15c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r15c1-gene-table-details" data-name="r15c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r15c1-MIBiG-cluster-compare" data-name="r15c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r15c1-clusterblast" data-name="r15c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r15c1-knownclusterblast" data-name="r15c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r15c1-subclusterblast" data-name="r15c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r15c1-tfbs-finder" data-name="r15c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r15c1-pfam-details" data-name="r15c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r15c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-92"></div> <span class="help-tooltip" id="gt-help-tooltip-help-92">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r15c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r15c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r15c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r15c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r15c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-86"></div> <span class="help-tooltip" id="MIBiG-help-86">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r15c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r15c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r15c1"></div>
<div id="comparison-MIBiG-r15c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001579: 0-8838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001579/index.html#r1c1" target="_new">BGC0001579</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>duramycin</td>


<td>Streptomyces cinnamoneus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563: 185-5330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>venezuelin</td>


<td>Streptomyces venezuelae ATCC 10712</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001660: 0-7887">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001660/index.html#r1c1" target="_new">BGC0001660</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>mathermycin</td>


<td>Marinactinospora thermotolerans</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000486: 1577-4152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000486/index.html#r1c1" target="_new">BGC0000486</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>butyrivibriocin AR10</td>


<td>Butyrivibrio fibrisolvens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001779: 0-6292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001779/index.html#r1c1" target="_new">BGC0001779</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>siamycin</td>


<td>Streptomyces nodosus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551: 0-6728">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>SapB</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001226: 0-6018">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001226/index.html#r1c1" target="_new">BGC0001226</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>streptocollin</td>


<td>Streptomyces collinus Tu 365</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928: 108-8402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Other</td>

<td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>


<td>Streptomyces sp. MA37</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000500: 0-8863">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000500/index.html#r1c1" target="_new">BGC0000500</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>carnolysin A1, carnolysin A2</td>


<td>Carnobacterium maltaromaticum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000504: 606-7471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000504/index.html#r1c1" target="_new">BGC0000504</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>cytolysin ClyLl, cytolysin ClyLs</td>


<td>Plasmid pAD1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r15c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001579: 0-8838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001579/index.html#r1c1" target="_new">BGC0001579</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>duramycin</td>
<td>Streptomyces cinnamoneus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563: 185-5330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>venezuelin</td>
<td>Streptomyces venezuelae ATCC 10712</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001660: 0-7887">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001660/index.html#r1c1" target="_new">BGC0001660</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>mathermycin</td>
<td>Marinactinospora thermotolerans</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000486: 1577-4152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000486/index.html#r1c1" target="_new">BGC0000486</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>butyrivibriocin AR10</td>
<td>Butyrivibrio fibrisolvens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001779: 0-6292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001779/index.html#r1c1" target="_new">BGC0001779</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>siamycin</td>
<td>Streptomyces nodosus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551: 0-6728">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>SapB</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001226: 0-6018">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001226/index.html#r1c1" target="_new">BGC0001226</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>streptocollin</td>
<td>Streptomyces collinus Tu 365</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928: 108-8402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Other</td>
<td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>
<td>Streptomyces sp. MA37</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000500: 0-8863">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000500/index.html#r1c1" target="_new">BGC0000500</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>carnolysin A1, carnolysin A2</td>
<td>Carnobacterium maltaromaticum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000504: 606-7471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000504/index.html#r1c1" target="_new">BGC0000504</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>cytolysin ClyLl, cytolysin ClyLs</td>
<td>Plasmid pAD1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r15c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-87"></div> <span class="help-tooltip" id="cb-general-help-87">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r15c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r15c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r15c1_1.svg">
NZ_CP015600 (1615419-1624506): Pseudomonas antarctica strain PAMC 27494 chrom... (91% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r15c1_2.svg">
NZ_JACAQV010000021 (357358-368246): Pseudomonas salomonii strain IPO3765 IPO ... (90% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r15c1_3.svg">
NZ_CP046103 (1379927-1389196): Pseudomonas sp. IB20 chromosome (83% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r15c1_4.svg">
NZ_FNOX01000001 (822671-833559): Pseudomonas salomonii strain ICMP 14252, who... (90% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r15c1_5.svg">
NZ_CP052856 (5891999-5902878): Pseudomonas sp. ADAK22 chromosome, complete ge... (83% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r15c1_6.svg">
NZ_CP027762 (1531630-1540785): Pseudomonas sp. LBUM920 chromosome, complete g... (83% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r15c1_7.svg">
NZ_QKMB01000013 (112911-123790): Pseudomonas sp. OV286 Ga0215801 113, whole g... (81% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r15c1_8.svg">
NZ_LT629704 (5676721-5687600): Pseudomonas antarctica strain LMG 22709 chromo... (81% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r15c1_9.svg">
NZ_CP007637 (1454750-1463483): Pseudomonas simiae strain WCS417 genome (81% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r15c1_10.svg">
NZ_JRMC01000002 (682345-693224): Pseudomonas simiae strain 2-36 scaffold2.1, ... (81% of genes show similarity), bacteriocin
</option>

</select>
<button id= "clusterblast-r15c1-download">Download graphic</button>
</div>
<div id="clusterblast-r15c1-svg"></div>

</div>
</div>

<div class="body-details-section r15c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-88"></div> <span class="help-tooltip" id="cb-known-help-88">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r15c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-89"></div> <span class="help-tooltip" id="cb-sub-help-89">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r15c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-91"></div> <span class="help-tooltip" id="tfbs-body-help-91">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 18.07 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r15c1-44163-AbrC3"></div>

</div>


</div>
</div>



<div class="body-details-section r15c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-93"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-93">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r15c1-pfam-details-svg" data-filename="NODE_15_length_143283_cov_19.608976_r15c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r15c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r15c1-tfbs-finder" data-name="r15c1-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r15c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-90"></div> <span class="help-tooltip" id="tfbs-side-help-90">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AbrC3" target="_blank">AbrC3</a></td>
<td>Antibiotic production activator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r19c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_19_length_106372_cov_20.016360 - Region 1 - NAGGN <div class="help-container"> <div class="help-icon" data-id="region_svg-help-154"></div> <span class="help-tooltip" id="region_svg-help-154">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_19_length_106372_cov_20.016360.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r19c1-svg" data-filename="NODE_19_length_106372_cov_20.016360_r19c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 32,875 - 47,755 nt. (total: 14,881 nt)

<a class="cluster-rules-header" id="r19c1-rules-header" href="#r19c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r19c1-rules">NAGGN: ((TIGR03104 or cds(GATase_7 and Asn_synthase)) and TIGR03103 and (TIGR03106 or Peptidase_M42))</div>

<div class="region-svg-container">
<div id='r19c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-155"></div> <span class="help-tooltip" id="focus-panel-help-155">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r19c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r19c1-gene-table-details" data-name="r19c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r19c1-MIBiG-cluster-compare" data-name="r19c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r19c1-clusterblast" data-name="r19c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r19c1-knownclusterblast" data-name="r19c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r19c1-subclusterblast" data-name="r19c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r19c1-pfam-details" data-name="r19c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r19c1-tigrfam-details" data-name="r19c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r19c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-98"></div> <span class="help-tooltip" id="gt-help-tooltip-help-98">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r19c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r19c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r19c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r19c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r19c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-94"></div> <span class="help-tooltip" id="MIBiG-help-94">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r19c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r19c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r19c1"></div>
<div id="comparison-MIBiG-r19c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NAGGN</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001880: 0-8263">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001880/index.html#r1c1" target="_new">BGC0001880</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>RiPP</td>

<td>scytodecamide</td>


<td>Scytonema sp. UIC 10036</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000846: 192-22933">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000846/index.html#r1c1" target="_new">BGC0000846</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Other</td>

<td>tabtoxin</td>


<td>Pseudomonas syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.19</td>
<td>Other</td>

<td>citrinin</td>


<td>Monascus aurantiacus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000866: 467-5165">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000866/index.html#r1c1" target="_new">BGC0000866</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Other</td>

<td>polyhydroxyalkanoate</td>


<td>Burkholderia sp. DSM 9242</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000867: 21-5822">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000867/index.html#r1c1" target="_new">BGC0000867</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Other</td>

<td>polyhydroxyalkanoic acids</td>


<td>Ectothiorhodospira shaposhnikovii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000885: 0-16706">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000885/index.html#r1c1" target="_new">BGC0000885</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Other</td>

<td>4-hydroxy-3-nitrosobenzamide</td>


<td>Streptomyces murayamaensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231: 189-5010">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>griseusin A, griseusin B</td>


<td>Streptomyces griseus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888: 0-4342">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Other</td>

<td>bacilysin</td>


<td>Bacillus sp. CS93</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000937: 0-23647">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000937/index.html#r1c1" target="_new">BGC0000937</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.43, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Other</td>

<td>fosfazinomycin A, fosfazinomycin B</td>


<td>Streptomyces sp. WM6372</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001184: 0-7298">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001184/index.html#r1c1" target="_new">BGC0001184</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.15</td>
<td>Other</td>

<td>bacilysin</td>


<td>Bacillus velezensis FZB42</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r19c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001880: 0-8263">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001880/index.html#r1c1" target="_new">BGC0001880</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>RiPP</td>
<td>scytodecamide</td>
<td>Scytonema sp. UIC 10036</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000846: 192-22933">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000846/index.html#r1c1" target="_new">BGC0000846</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Other</td>
<td>tabtoxin</td>
<td>Pseudomonas syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Other</td>
<td>citrinin</td>
<td>Monascus aurantiacus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000866: 467-5165">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000866/index.html#r1c1" target="_new">BGC0000866</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Other</td>
<td>polyhydroxyalkanoate</td>
<td>Burkholderia sp. DSM 9242</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000867: 21-5822">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000867/index.html#r1c1" target="_new">BGC0000867</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Other</td>
<td>polyhydroxyalkanoic acids</td>
<td>Ectothiorhodospira shaposhnikovii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000885: 0-16706">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000885/index.html#r1c1" target="_new">BGC0000885</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Other</td>
<td>4-hydroxy-3-nitrosobenzamide</td>
<td>Streptomyces murayamaensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231: 189-5010">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Polyketide</td>
<td>griseusin A, griseusin B</td>
<td>Streptomyces griseus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888: 0-4342">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Other</td>
<td>bacilysin</td>
<td>Bacillus sp. CS93</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000937: 0-23647">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000937/index.html#r1c1" target="_new">BGC0000937</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.43, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Other</td>
<td>fosfazinomycin A, fosfazinomycin B</td>
<td>Streptomyces sp. WM6372</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001184: 0-7298">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001184/index.html#r1c1" target="_new">BGC0001184</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Other</td>
<td>bacilysin</td>
<td>Bacillus velezensis FZB42</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r19c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-95"></div> <span class="help-tooltip" id="cb-general-help-95">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r19c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r19c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r19c1_1.svg">
NZ_CP015600 (4410420-4425301): Pseudomonas antarctica strain PAMC 27494 chrom... (100% of genes show similarity), NAGGN
</option>

<option value = "svg/clusterblast_r19c1_2.svg">
NZ_LT907842 (156004-170862): Pseudomonas fluorescens strain ATCC 13525 chromo... (100% of genes show similarity), NAGGN
</option>

<option value = "svg/clusterblast_r19c1_3.svg">
NZ_MSDI01000001 (628602-643460): Pseudomonas fluorescens strain E24 PF E24 co... (100% of genes show similarity), NAGGN
</option>

<option value = "svg/clusterblast_r19c1_4.svg">
NZ_CP019428 (4715586-4730429): Pseudomonas sp. R76 chromosome, complete genome (90% of genes show similarity), NAGGN
</option>

<option value = "svg/clusterblast_r19c1_5.svg">
NZ_LT629704 (2866925-2881768): Pseudomonas antarctica strain LMG 22709 chromo... (100% of genes show similarity), NAGGN
</option>

<option value = "svg/clusterblast_r19c1_6.svg">
NZ_JACAOZ010000019 (29750-44622): Pseudomonas edaphica strain B7002 run0228 B... (100% of genes show similarity), NAGGN
</option>

<option value = "svg/clusterblast_r19c1_7.svg">
NZ_CP046103 (3454470-3469351): Pseudomonas sp. IB20 chromosome (90% of genes show similarity), NAGGN
</option>

<option value = "svg/clusterblast_r19c1_8.svg">
NZ_QKMB01000006 (47586-62375): Pseudomonas sp. OV286 Ga0215801 106, whole gen... (100% of genes show similarity), NAGGN
</option>

<option value = "svg/clusterblast_r19c1_9.svg">
NZ_JACAQV010000010 (696913-711756): Pseudomonas salomonii strain IPO3765 IPO ... (90% of genes show similarity), NAGGN
</option>

<option value = "svg/clusterblast_r19c1_10.svg">
NZ_FNOX01000012 (17398-32241): Pseudomonas salomonii strain ICMP 14252, whole... (90% of genes show similarity), NAGGN
</option>

</select>
<button id= "clusterblast-r19c1-download">Download graphic</button>
</div>
<div id="clusterblast-r19c1-svg"></div>

</div>
</div>

<div class="body-details-section r19c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-96"></div> <span class="help-tooltip" id="cb-known-help-96">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r19c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-97"></div> <span class="help-tooltip" id="cb-sub-help-97">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r19c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-99"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-99">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r19c1-pfam-details-svg" data-filename="NODE_19_length_106372_cov_20.016360_r19c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r19c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r19c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-100"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-100">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r19c1-tigrfam-details-svg" data-filename="NODE_19_length_106372_cov_20.016360_r19c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r19c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>




<div class="page" id='r30c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_30_length_62891_cov_19.830261 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-156"></div> <span class="help-tooltip" id="region_svg-help-156">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_30_length_62891_cov_19.830261.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r30c1-svg" data-filename="NODE_30_length_62891_cov_19.830261_r30c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 11,142 - 62,891 nt. (total: 51,750 nt)

<a class="cluster-rules-header" id="r30c1-rules-header" href="#r30c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r30c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r30c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-157"></div> <span class="help-tooltip" id="focus-panel-help-157">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r30c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r30c1-gene-table-details" data-name="r30c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r30c1-nrps_pks" data-name="r30c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r30c1-MIBiG-cluster-compare" data-name="r30c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r30c1-clusterblast" data-name="r30c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r30c1-knownclusterblast" data-name="r30c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r30c1-subclusterblast" data-name="r30c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r30c1-tfbs-finder" data-name="r30c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r30c1-nrps-pks-bubbles" data-name="r30c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r30c1-pfam-details" data-name="r30c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r30c1-tigrfam-details" data-name="r30c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r30c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-111"></div> <span class="help-tooltip" id="gt-help-tooltip-help-111">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r30c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r30c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r30c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r30c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r30c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-101"></div> <span class="help-tooltip" id="nrps-domain-help-101">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r30c1-details-svg" data-filename="NODE_30_length_62891_cov_19.830261_r30c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r30c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r30c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-102"></div> <span class="help-tooltip" id="MIBiG-help-102">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r30c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r30c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r30c1"></div>
<div id="comparison-MIBiG-r30c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>





<td class="cc-heat-cell" title="0.78 (id:0.53, order:1.00, components:0.91)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.78), rgb(0, 0, 0, 0.78) 78%, white 78%)"></td>


<td>0.50</td>
<td>NRP</td>

<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>


<td>Pseudomonas sp. QS1027</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001312: 0-43437">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001312/index.html#r1c1" target="_new">BGC0001312</a></td>





<td class="cc-heat-cell" title="0.75 (id:0.70, order:0.63, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>


<td>0.45</td>
<td>NRP</td>

<td>viscosin</td>


<td>Pseudomonas fluorescens SBW25</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>





<td class="cc-heat-cell" title="0.74 (id:0.45, order:1.00, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>


<td>0.43</td>
<td>NRP</td>

<td>taxlllaid A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000323: 0-32510">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000323/index.html#r1c1" target="_new">BGC0000323</a></td>





<td class="cc-heat-cell" title="0.74 (id:0.53, order:0.83, components:0.91)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>


<td>0.43</td>
<td>NRP</td>

<td>cichofactin A, cichofactin B</td>


<td>Pseudomonas cichorii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000435: 0-32188">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000435/index.html#r1c1" target="_new">BGC0000435</a></td>





<td class="cc-heat-cell" title="0.74 (id:0.53, order:0.83, components:0.91)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>


<td>0.43</td>
<td>NRP</td>

<td>syringafactin</td>


<td>Pseudomonas syringae pv. tomato str. DC3000</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.71 (id:0.48, order:0.75, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>


<td>0.39</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>





<td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>


<td>0.38</td>
<td>NRP</td>

<td>rhizomide A, rhizomide B, rhizomide C</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.38</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.37</td>
<td>NRP</td>

<td>ambactin</td>


<td>Xenorhabdus miraniensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>NRP</td>

<td>holrhizin</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r30c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>


<td class="cc-heat-cell" title="0.78 (id:0.53, order:1.00, components:0.91)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.78), rgb(0, 0, 0, 0.78) 78%, white 78%)"></td>
<td>0.78</td>
<td>NRP</td>
<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>
<td>Pseudomonas sp. QS1027</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001312: 0-43437">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001312/index.html#r1c1" target="_new">BGC0001312</a></td>


<td class="cc-heat-cell" title="0.75 (id:0.70, order:0.63, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
<td>0.75</td>
<td>NRP</td>
<td>viscosin</td>
<td>Pseudomonas fluorescens SBW25</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>


<td class="cc-heat-cell" title="0.74 (id:0.45, order:1.00, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
<td>0.74</td>
<td>NRP</td>
<td>taxlllaid A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000323: 0-32510">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000323/index.html#r1c1" target="_new">BGC0000323</a></td>


<td class="cc-heat-cell" title="0.74 (id:0.53, order:0.83, components:0.91)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
<td>0.74</td>
<td>NRP</td>
<td>cichofactin A, cichofactin B</td>
<td>Pseudomonas cichorii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000435: 0-32188">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000435/index.html#r1c1" target="_new">BGC0000435</a></td>


<td class="cc-heat-cell" title="0.74 (id:0.53, order:0.83, components:0.91)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
<td>0.74</td>
<td>NRP</td>
<td>syringafactin</td>
<td>Pseudomonas syringae pv. tomato str. DC3000</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.71 (id:0.48, order:0.75, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
<td>0.71</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>


<td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
<td>0.71</td>
<td>NRP</td>
<td>rhizomide A, rhizomide B, rhizomide C</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>NRP</td>
<td>ambactin</td>
<td>Xenorhabdus miraniensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>NRP</td>
<td>holrhizin</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r30c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-103"></div> <span class="help-tooltip" id="cb-general-help-103">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r30c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r30c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r30c1_1.svg">
NZ_CP015600 (2619322-2681778): Pseudomonas antarctica strain PAMC 27494 chrom... (76% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r30c1_2.svg">
NZ_MSDI01000033 (35897-100142): Pseudomonas fluorescens strain E24 PF E24 con... (71% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r30c1_3.svg">
NZ_LT907842 (4859474-4921220): Pseudomonas fluorescens strain ATCC 13525 chro... (71% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r30c1_4.svg">
NZ_CP052856 (457315-519910): Pseudomonas sp. ADAK22 chromosome, complete genome (70% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r30c1_5.svg">
NZ_CP027762 (2622687-2685327): Pseudomonas sp. LBUM920 chromosome, complete g... (70% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r30c1_6.svg">
NZ_CP046103 (2205110-&gt;2266602): Pseudomonas sp. IB20 chromosome (64% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r30c1_7.svg">
NZ_LT629704 (3510772-3573303): Pseudomonas antarctica strain LMG 22709 chromo... (66% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r30c1_8.svg">
NZ_MOAZ01000013 (40056-144346): Pseudomonas canadensis strain 36C8 36C8 conti... (42% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r30c1_9.svg">
NZ_CP023272 (2351708-2458342): Pseudomonas lurida strain MYb11 chromosome, co... (45% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r30c1_10.svg">
NZ_CABVGW010000014 (41734-146114): Pseudomonas fluorescens isolate PS682, who... (45% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r30c1-download">Download graphic</button>
</div>
<div id="clusterblast-r30c1-svg"></div>

</div>
</div>

<div class="body-details-section r30c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-104"></div> <span class="help-tooltip" id="cb-known-help-104">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r30c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r30c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r30c1_1.svg">
viscosin
</option>

<option value = "svg/knownclusterblast_r30c1_2.svg">
viscosinamide A/pseudodesmin A
</option>

<option value = "svg/knownclusterblast_r30c1_3.svg">
bananamide F/bananamide D/bananamide G/bananamide E
</option>

<option value = "svg/knownclusterblast_r30c1_4.svg">
bananamide 1/bananamide 2/bananamide 3
</option>

<option value = "svg/knownclusterblast_r30c1_5.svg">
gacamide A
</option>

<option value = "svg/knownclusterblast_r30c1_6.svg">
orfamide A/orfamide C
</option>

<option value = "svg/knownclusterblast_r30c1_7.svg">
orfamide B
</option>

<option value = "svg/knownclusterblast_r30c1_8.svg">
putisolvin
</option>

<option value = "svg/knownclusterblast_r30c1_9.svg">
putisolvin III/putisolvin VI/putisolvin V
</option>

<option value = "svg/knownclusterblast_r30c1_10.svg">
WLIP
</option>

</select>
<button id="knownclusterblast-r30c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r30c1-svg"></div>

</div>
</div>

<div class="body-details-section r30c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-105"></div> <span class="help-tooltip" id="cb-sub-help-105">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r30c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-109"></div> <span class="help-tooltip" id="tfbs-body-help-109">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
OsdR
<div class="collapser collapser-target-OsdR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Development and stress management regulator<br>
<span>Score</span>: 24.85 out of 30.12
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r30c1-62110-OsdR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
HypR
<div class="collapser collapser-target-HypR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: L-hydroxyproline utilization repressor<br>
<span>Score</span>: 18.79 out of 22.40
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r30c1-55549-HypR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r30c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-110"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-110">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r30c1-domain-bubble-svg" data-filename="NODE_30_length_62891_cov_19.830261_r30c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r30c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (11141 - 62891): single NRPS
</option>


</select>

<div class="details-svg" id="r30c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r30c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r30c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-112"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-112">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r30c1-pfam-details-svg" data-filename="NODE_30_length_62891_cov_19.830261_r30c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r30c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r30c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-113"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-113">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r30c1-tigrfam-details-svg" data-filename="NODE_30_length_62891_cov_19.830261_r30c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r30c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r30c1-nrps_pks_products" data-name="r30c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r30c1-nrps_pks_monomers" data-name="r30c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r30c1-tfbs-finder" data-name="r30c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r30c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-106"></div> <span class="help-tooltip" id="nrps-structures-help-106">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 11141 - 62891:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r30c1-1" class="smiles-canvas" data-smiles="NC(C(O)C)C(=O)NC(C(C)C)C(=O)NC(CC(C)C)C(=O)NC(CO)C(=O)NC(CC(C)C)C(=O)NC(CO)C(=O)NC(C(C)CC)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Thr - Val - Leu - Ser) + (Leu - Ser - Ile)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr,val,leu,ser&amp;nrps2=leu,ser,ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*,val*,leu*,ser*&amp;nrps2=leu*,ser*,ile*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r30c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(C(O)C)C(=O)NC(C(C)C)C(=O)NC(CC(C)C)C(=O)NC(CO)C(=O)NC(CC(C)C)C(=O)NC(CO)C(=O)NC(C(C)CC)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r30c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-107"></div> <span class="help-tooltip" id="nrps-monomers-help-107">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">ctg30_26</span></strong>: Leu - Ser - Ile
<div class="collapser collapser-target-ctg30_26 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu,ser,ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*,ser*,ile*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (568..953)</strong>: Leu
<div class="collapser collapser-target-ctg30_26 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Leu
<div class="collapser collapser-target-ctg30_26-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>leu (Leu)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAWFLGNVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Leu <span class="serif">DAWFLGNVVK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1606..1997)</strong>: Ser
<div class="collapser collapser-target-ctg30_26 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ser
<div class="collapser collapser-target-ctg30_26-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>ser (Ser)</dd>
<dt>Single AA prediction:</dt>
<dd>ser (Ser)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVWHMSLVDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ser <span class="serif">DVWHMSLVDK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (2636..3027)</strong>: Ile
<div class="collapser collapser-target-ctg30_26 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ile
<div class="collapser collapser-target-ctg30_26-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>ile (Ile)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAMFLGCTYK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ile <span class="serif">DAMFLGCTYK</span> (97% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

<strong><span class="serif">ctg30_27</span></strong>: Thr - Val - Leu - Ser
<div class="collapser collapser-target-ctg30_27 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr,val,leu,ser" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*,val*,leu*,ser*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (552..948)</strong>: Thr
<div class="collapser collapser-target-ctg30_27 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Thr
<div class="collapser collapser-target-ctg30_27-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>thr (Thr)</dd>
<dt>Single AA prediction:</dt>
<dd>thr (Thr)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DFWNIGMVHK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Thr <span class="serif">DFWNIGMVHK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1624..2001)</strong>: Val
<div class="collapser collapser-target-ctg30_27 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Val
<div class="collapser collapser-target-ctg30_27-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>val (Val)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DALWIGGTFK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Val <span class="serif">DALWIGGTFK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (2680..3065)</strong>: Leu
<div class="collapser collapser-target-ctg30_27 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Leu
<div class="collapser collapser-target-ctg30_27-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>leu (Leu)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAWFLGNVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Leu <span class="serif">DAWFLGNVVK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (3735..4130)</strong>: Ser
<div class="collapser collapser-target-ctg30_27 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ser
<div class="collapser collapser-target-ctg30_27-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>ser (Ser)</dd>
<dt>Single AA prediction:</dt>
<dd>ser (Ser)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVWHMSLVDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ser <span class="serif">DVWHMSLVDK</span> (100% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r30c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-108"></div> <span class="help-tooltip" id="tfbs-side-help-108">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/OsdR" target="_blank">OsdR</a></td>
<td>Development and stress management regulator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/HypR" target="_blank">HypR</a></td>
<td>L-hydroxyproline utilization repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>




<div class="page" id='r39c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_39_length_36248_cov_19.215200 - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-158"></div> <span class="help-tooltip" id="region_svg-help-158">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_39_length_36248_cov_19.215200.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r39c1-svg" data-filename="NODE_39_length_36248_cov_19.215200_r39c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 7,497 nt. (total: 7,497 nt)

<a class="cluster-rules-header" id="r39c1-rules-header" href="#r39c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r39c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193 or (YcaO or TIGR03604))</div>

<div class="region-svg-container">
<div id='r39c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-159"></div> <span class="help-tooltip" id="focus-panel-help-159">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r39c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r39c1-gene-table-details" data-name="r39c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r39c1-MIBiG-cluster-compare" data-name="r39c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r39c1-clusterblast" data-name="r39c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r39c1-knownclusterblast" data-name="r39c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r39c1-subclusterblast" data-name="r39c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r39c1-tfbs-finder" data-name="r39c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r39c1-pfam-details" data-name="r39c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r39c1-tigrfam-details" data-name="r39c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r39c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-120"></div> <span class="help-tooltip" id="gt-help-tooltip-help-120">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r39c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r39c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r39c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r39c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r39c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-114"></div> <span class="help-tooltip" id="MIBiG-help-114">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r39c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r39c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r39c1"></div>
<div id="comparison-MIBiG-r39c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001285: 8-22479">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001285/index.html#r1c1" target="_new">BGC0001285</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.65, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Other (Fatty acid)</td>

<td>pseudopyronine A, pseudopyronine B</td>


<td>Pseudomonas putida</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843: 0-23666">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.43, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Other (Non-NRP beta-lactam)</td>

<td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>


<td>Streptomyces clavuligerus ATCC 27064</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000847: 0-32329">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000847/index.html#r1c1" target="_new">BGC0000847</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.43, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Other (Non-NRP beta-lactam)</td>

<td>thienamycin</td>


<td>Streptomyces cattleya</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001312: 0-43437">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001312/index.html#r1c1" target="_new">BGC0001312</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP</td>

<td>viscosin</td>


<td>Pseudomonas fluorescens SBW25</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002694: 0-16770">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002694/index.html#r1c1" target="_new">BGC0002694</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.43, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>Other</td>

<td>roseobactin</td>


<td>Phaeobacter inhibens 2.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002294: 412-30500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002294/index.html#r1c1" target="_new">BGC0002294</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.44, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>Other</td>

<td>streptozotocin</td>


<td>Streptomyces achromogenes subsp. streptozoticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000159: 346-93961">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000159/index.html#r1c1" target="_new">BGC0000159</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.46, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>tautomycin</td>


<td>Streptomyces spiroverticillatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001194: 0-72530">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001194/index.html#r1c1" target="_new">BGC0001194</a></td>





<td class="cc-heat-cell" title="0.22 (id:0.47, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>hitachimycin</td>


<td>Streptomyces scabrisporus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000066: 0-69644">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000066/index.html#r1c1" target="_new">BGC0000066</a></td>





<td class="cc-heat-cell" title="0.21 (id:0.44, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>


<td>0.04</td>
<td>Polyketide</td>

<td>geldanamycin</td>


<td>Streptomyces hygroscopicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001690: 38-91329">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001690/index.html#r1c1" target="_new">BGC0001690</a></td>





<td class="cc-heat-cell" title="0.20 (id:0.43, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>


<td>0.04</td>
<td>Polyketide</td>

<td>natamycin</td>


<td>Streptomyces gilvosporeus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r39c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001285: 8-22479">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001285/index.html#r1c1" target="_new">BGC0001285</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.65, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Other (Fatty acid)</td>
<td>pseudopyronine A, pseudopyronine B</td>
<td>Pseudomonas putida</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843: 0-23666">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.43, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Other (Non-NRP beta-lactam)</td>
<td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
<td>Streptomyces clavuligerus ATCC 27064</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000847: 0-32329">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000847/index.html#r1c1" target="_new">BGC0000847</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.43, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Other (Non-NRP beta-lactam)</td>
<td>thienamycin</td>
<td>Streptomyces cattleya</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001312: 0-43437">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001312/index.html#r1c1" target="_new">BGC0001312</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP</td>
<td>viscosin</td>
<td>Pseudomonas fluorescens SBW25</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002694: 0-16770">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002694/index.html#r1c1" target="_new">BGC0002694</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.43, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>Other</td>
<td>roseobactin</td>
<td>Phaeobacter inhibens 2.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002294: 412-30500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002294/index.html#r1c1" target="_new">BGC0002294</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.44, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>Other</td>
<td>streptozotocin</td>
<td>Streptomyces achromogenes subsp. streptozoticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000159: 346-93961">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000159/index.html#r1c1" target="_new">BGC0000159</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.46, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Polyketide</td>
<td>tautomycin</td>
<td>Streptomyces spiroverticillatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001194: 0-72530">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001194/index.html#r1c1" target="_new">BGC0001194</a></td>


<td class="cc-heat-cell" title="0.22 (id:0.47, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
<td>0.22</td>
<td>Polyketide</td>
<td>hitachimycin</td>
<td>Streptomyces scabrisporus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000066: 0-69644">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000066/index.html#r1c1" target="_new">BGC0000066</a></td>


<td class="cc-heat-cell" title="0.21 (id:0.44, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
<td>0.21</td>
<td>Polyketide</td>
<td>geldanamycin</td>
<td>Streptomyces hygroscopicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001690: 38-91329">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001690/index.html#r1c1" target="_new">BGC0001690</a></td>


<td class="cc-heat-cell" title="0.20 (id:0.43, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>
<td>0.20</td>
<td>Polyketide</td>
<td>natamycin</td>
<td>Streptomyces gilvosporeus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r39c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-115"></div> <span class="help-tooltip" id="cb-general-help-115">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r39c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r39c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r39c1_1.svg">
NZ_CP046103 (2754535-2781745): Pseudomonas sp. IB20 chromosome (26% of genes show similarity), thiopeptide
</option>

<option value = "svg/clusterblast_r39c1_2.svg">
NZ_CP015600 (3439647-3467711): Pseudomonas antarctica strain PAMC 27494 chrom... (25% of genes show similarity), thiopeptide
</option>

<option value = "svg/clusterblast_r39c1_3.svg">
NZ_CP052856 (1295014-1326035): Pseudomonas sp. ADAK22 chromosome, complete ge... (20% of genes show similarity), thiopeptide
</option>

<option value = "svg/clusterblast_r39c1_4.svg">
NZ_LT907842 (5735876-5765125): Pseudomonas fluorescens strain ATCC 13525 chro... (21% of genes show similarity), thiopeptide
</option>

<option value = "svg/clusterblast_r39c1_5.svg">
NZ_JACARQ010000001 (137842-167833): Pseudomonas tolaasii strain P7756 P7756 S... (20% of genes show similarity), thiopeptide
</option>

<option value = "svg/clusterblast_r39c1_6.svg">
NZ_CABVGW010000001 (75337-103892): Pseudomonas fluorescens isolate PS682, who... (20% of genes show similarity), thiopeptide
</option>

<option value = "svg/clusterblast_r39c1_7.svg">
NZ_CP023272 (3235529-3264084): Pseudomonas lurida strain MYb11 chromosome, co... (21% of genes show similarity), thiopeptide
</option>

<option value = "svg/clusterblast_r39c1_8.svg">
NZ_LT629708 (706141-735650): Pseudomonas extremorientalis strain LMG 19695 ch... (19% of genes show similarity), thiopeptide
</option>

<option value = "svg/clusterblast_r39c1_9.svg">
NZ_CP015639 (2765673-2794228): Pseudomonas lurida strain L228 chromosome, com... (24% of genes show similarity), thiopeptide
</option>

<option value = "svg/clusterblast_r39c1_10.svg">
NZ_CP027762 (3514919-3545802): Pseudomonas sp. LBUM920 chromosome, complete g... (20% of genes show similarity), thiopeptide
</option>

</select>
<button id= "clusterblast-r39c1-download">Download graphic</button>
</div>
<div id="clusterblast-r39c1-svg"></div>

</div>
</div>

<div class="body-details-section r39c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-116"></div> <span class="help-tooltip" id="cb-known-help-116">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r39c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-117"></div> <span class="help-tooltip" id="cb-sub-help-117">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r39c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-119"></div> <span class="help-tooltip" id="tfbs-body-help-119">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AfsR
<div class="collapser collapser-target-AfsR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Pleiotropic regulatory for antibiotic production<br>
<span>Score</span>: 18.76 out of 23.05
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r39c1-5783-AfsR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 19.08 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r39c1-2481-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r39c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-121"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-121">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r39c1-pfam-details-svg" data-filename="NODE_39_length_36248_cov_19.215200_r39c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r39c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r39c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-122"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-122">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r39c1-tigrfam-details-svg" data-filename="NODE_39_length_36248_cov_19.215200_r39c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r39c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r39c1-tfbs-finder" data-name="r39c1-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r39c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-118"></div> <span class="help-tooltip" id="tfbs-side-help-118">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AfsR" target="_blank">AfsR</a></td>
<td>Pleiotropic regulatory for antibiotic production</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r52c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_52_length_18213_cov_21.895732 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-160"></div> <span class="help-tooltip" id="region_svg-help-160">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_52_length_18213_cov_21.895732.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r52c1-svg" data-filename="NODE_52_length_18213_cov_21.895732_r52c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 18,213 nt. (total: 18,213 nt)

<a class="cluster-rules-header" id="r52c1-rules-header" href="#r52c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r52c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r52c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-161"></div> <span class="help-tooltip" id="focus-panel-help-161">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r52c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r52c1-gene-table-details" data-name="r52c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r52c1-nrps_pks" data-name="r52c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r52c1-MIBiG-cluster-compare" data-name="r52c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r52c1-clusterblast" data-name="r52c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r52c1-knownclusterblast" data-name="r52c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r52c1-subclusterblast" data-name="r52c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r52c1-tfbs-finder" data-name="r52c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r52c1-nrps-pks-bubbles" data-name="r52c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r52c1-pfam-details" data-name="r52c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r52c1-tigrfam-details" data-name="r52c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r52c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-133"></div> <span class="help-tooltip" id="gt-help-tooltip-help-133">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r52c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r52c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r52c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r52c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r52c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-123"></div> <span class="help-tooltip" id="nrps-domain-help-123">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r52c1-details-svg" data-filename="NODE_52_length_18213_cov_21.895732_r52c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r52c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r52c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-124"></div> <span class="help-tooltip" id="MIBiG-help-124">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r52c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r52c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r52c1"></div>
<div id="comparison-MIBiG-r52c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>livipeptin</td>


<td>Streptomyces lividans 1326</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001599: 0-9373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001599/index.html#r1c1" target="_new">BGC0001599</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.48, order:0.50, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>fragin</td>


<td>Burkholderia cenocepacia H111</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Alkaloid</td>

<td>aspergillic acid</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001516: 0-14978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001516/index.html#r1c1" target="_new">BGC0001516</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>aspergillic acid</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002582: 0-7207">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002582/index.html#r1c1" target="_new">BGC0002582</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>guanipiperazine A, guanipiperazine B</td>


<td>Streptomyces chrestomyceticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002278: 0-28520">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002278/index.html#r1c1" target="_new">BGC0002278</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Alkaloid, NRP</td>

<td>brasiliamide I, brasiliamide A, brasiliamide B, 2,2-dibenzylpiperazine, brasiliamide D</td>


<td>Penicillium brasilianum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002660: 0-23622">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002660/index.html#r1c1" target="_new">BGC0002660</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>NRP</td>

<td>pentacitidin A, pentacitidin B</td>


<td>Chitinophaga eiseniae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>rhizomide A, rhizomide B, rhizomide C</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r52c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>livipeptin</td>
<td>Streptomyces lividans 1326</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001599: 0-9373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001599/index.html#r1c1" target="_new">BGC0001599</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.48, order:0.50, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>fragin</td>
<td>Burkholderia cenocepacia H111</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Alkaloid</td>
<td>aspergillic acid</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001516: 0-14978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001516/index.html#r1c1" target="_new">BGC0001516</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>aspergillic acid</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002582: 0-7207">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002582/index.html#r1c1" target="_new">BGC0002582</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>guanipiperazine A, guanipiperazine B</td>
<td>Streptomyces chrestomyceticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002278: 0-28520">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002278/index.html#r1c1" target="_new">BGC0002278</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Alkaloid, NRP</td>
<td>brasiliamide I, brasiliamide A, brasiliamide B, 2,2-dibenzylpiperazine, brasiliamide D</td>
<td>Penicillium brasilianum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002660: 0-23622">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002660/index.html#r1c1" target="_new">BGC0002660</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>NRP</td>
<td>pentacitidin A, pentacitidin B</td>
<td>Chitinophaga eiseniae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>rhizomide A, rhizomide B, rhizomide C</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r52c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-125"></div> <span class="help-tooltip" id="cb-general-help-125">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r52c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r52c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r52c1_1.svg">
NZ_LT629704 (644283-687164): Pseudomonas antarctica strain LMG 22709 chromoso... (43% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r52c1_2.svg">
NZ_CP025494 (4035690-4076340): Pseudomonas palleroniana strain MAB3 chromosome (43% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r52c1_3.svg">
NZ_CP020369 (5715296-5757024): Pseudomonas tolaasii strain 2192T genome (27% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r52c1_4.svg">
NZ_CP019428 (159849-202848): Pseudomonas sp. R76 chromosome, complete genome (50% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r52c1_5.svg">
NZ_LT629800 (4999199-5042592): Pseudomonas brenneri strain DSM 15294 chromoso... (47% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r52c1_6.svg">
NZ_LT629802 (4806963-4850353): Pseudomonas mucidolens strain LMG 2223 chromos... (47% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r52c1_7.svg">
NZ_MAYQ01000006 (960115-1001168): Pseudomonas moraviensis strain Devor contig... (45% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r52c1_8.svg">
NZ_RHGD01000001 (3150638-&gt;3193447): Pseudomonas mandelii strain B2/38-Q-III S... (35% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r52c1_9.svg">
NZ_CP027753 (132612-175186): Pseudomonas chlororaphis strain B25 chromosome, ... (50% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r52c1_10.svg">
NZ_CP035088 (168889-210046): Pseudomonas sp. 11K1 chromosome, complete genome (37% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r52c1-download">Download graphic</button>
</div>
<div id="clusterblast-r52c1-svg"></div>

</div>
</div>

<div class="body-details-section r52c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-126"></div> <span class="help-tooltip" id="cb-known-help-126">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r52c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-127"></div> <span class="help-tooltip" id="cb-sub-help-127">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r52c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-131"></div> <span class="help-tooltip" id="tfbs-body-help-131">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 17.89 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r52c1-10389-AbrC3"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 22.10 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r52c1-8312-NuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r52c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-132"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-132">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r52c1-domain-bubble-svg" data-filename="NODE_52_length_18213_cov_21.895732_r52c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r52c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (0 - 18213): single NRPS-like
</option>


</select>

<div class="details-svg" id="r52c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r52c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r52c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-134"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-134">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r52c1-pfam-details-svg" data-filename="NODE_52_length_18213_cov_21.895732_r52c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r52c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r52c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-135"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-135">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r52c1-tigrfam-details-svg" data-filename="NODE_52_length_18213_cov_21.895732_r52c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r52c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r52c1-nrps_pks_products" data-name="r52c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r52c1-nrps_pks_monomers" data-name="r52c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r52c1-tfbs-finder" data-name="r52c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r52c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-128"></div> <span class="help-tooltip" id="nrps-structures-help-128">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 0 - 18213:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r52c1-1" class="smiles-canvas" data-smiles="NC(CC(C)C)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Leu)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r52c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CC(C)C)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r52c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-129"></div> <span class="help-tooltip" id="nrps-monomers-help-129">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">ctg52_3</span></strong>: Leu
<div class="collapser collapser-target-ctg52_3 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (178..561)</strong>: Leu
<div class="collapser collapser-target-ctg52_3 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Leu
<div class="collapser collapser-target-ctg52_3-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>leu (Leu)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DSIFLGFVLK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Leu or Ile <span class="serif">DAIYLGVVLK</span> (59% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r52c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-130"></div> <span class="help-tooltip" id="tfbs-side-help-130">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AbrC3" target="_blank">AbrC3</a></td>
<td>Antibiotic production activator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>